{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import getpass\n",
    "\n",
    "\n",
    "\n",
    "from langchain_community.document_loaders import TextLoader\n",
    "from langchain_community.document_loaders import UnstructuredHTMLLoader\n",
    "\n",
    "from langchain.text_splitter import CharacterTextSplitter\n",
    "from langchain_community.vectorstores import FAISS\n",
    "\n",
    "import urllib.request, urllib.error, urllib.parse\n",
    "\n",
    "url = 'https://doi.org/10.1038/s41586-023-06473-y'\n",
    "doi = 's41586-023-06473-y'\n",
    "response = urllib.request.urlopen(url)\n",
    "webContent = response.read().decode('UTF-8')\n",
    "\n",
    "filename = doi + \".html\"\n",
    "with open(filename, \"w\", encoding=\"utf-8\") as f:\n",
    "    f.write(webContent)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Single-cell brain organoid screening identifies developmental defects in autism | Nature': '<!DOCTYPE html>\\n<html lang=\"en\" class=\"grade-c\">\\n<head>\\n    <title>Single-cell brain organoid screening identifies developmental defects in autism | Nature</title>\\n    \\n        \\n<link rel=\"alternate\" type=\"application/rss+xml\" href=\"https://www.nature.com/nature.rss\"/>\\n\\n\\n    \\n        \\n\\n        <script id=\"save-data-connection-testing\">\\n            function hasConnection() {\\n                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;\\n            }\\n\\n            function createLink(src) {\\n                var preloadLink = document.createElement(\"link\");\\n                preloadLink.rel = \"preload\";\\n                preloadLink.href = src;\\n                preloadLink.as = \"font\";\\n                preloadLink.type = \"font/woff2\";\\n                preloadLink.crossOrigin = \"\";\\n                document.head.insertBefore(preloadLink, document.head.firstChild);\\n            }\\n\\n            var connectionDetail = {\\n                saveDataEnabled: false,\\n                slowConnection: false\\n            };\\n\\n            var connection = hasConnection();\\n            if (connection) {\\n                connectionDetail.saveDataEnabled = connection.saveData;\\n                if (/\\\\slow-2g|2g/.test(connection.effectiveType)) {\\n                    connectionDetail.slowConnection = true;\\n                }\\n            }\\n\\n            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {\\n                createLink(\"/static/fonts/HardingText-Regular-Web-cecd90984f.woff2\");\\n            } else {\\n                document.documentElement.classList.add(\\'save-data\\');\\n            }\\n        </script>\\n    \\n\\n<link rel=\"preconnect\" href=\"https://cmp.nature.com\" crossorigin>\\n\\n<meta http-equiv=\"X-UA-Compatible\" content=\"IE=edge\">\\n<meta name=\"applicable-device\" content=\"pc,mobile\">\\n<meta name=\"viewport\" content=\"width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes\">\\n<meta name=\"360-site-verification\" content=\"5a2dc4ab3fcb9b0393241ffbbb490480\" />\\n\\n<script data-test=\"dataLayer\">\\n    window.dataLayer = [{\"content\":{\"category\":{\"contentType\":\"article\",\"legacy\":{\"webtrendsPrimaryArticleType\":\"research\",\"webtrendsSubjectTerms\":\"development-of-the-nervous-system;stem-cell-biotechnology;stem-cells\",\"webtrendsContentCategory\":null,\"webtrendsContentCollection\":null,\"webtrendsContentGroup\":\"Nature\",\"webtrendsContentGroupType\":null,\"webtrendsContentSubGroup\":\"Article\"}},\"article\":{\"doi\":\"10.1038/s41586-023-06473-y\"},\"attributes\":{\"cms\":null,\"deliveryPlatform\":\"oscar\",\"copyright\":{\"open\":true,\"legacy\":{\"webtrendsLicenceType\":\"http://creativecommons.org/licenses/by/4.0/\"}}},\"contentInfo\":{\"authors\":[\"Chong Li\",\"Jonas Simon Fleck\",\"Catarina Martins-Costa\",\"Thomas R. Burkard\",\"Jan Themann\",\"Marlene Stuempflen\",\"Angela Maria Peer\",\"Ábel Vertesy\",\"Jamie B. Littleboy\",\"Christopher Esk\",\"Ulrich Elling\",\"Gregor Kasprian\",\"Nina S. Corsini\",\"Barbara Treutlein\",\"Juergen A. Knoblich\"],\"publishedAt\":1694563200,\"publishedAtString\":\"2023-09-13\",\"title\":\"Single-cell brain organoid screening identifies developmental defects in autism\",\"legacy\":null,\"publishedAtTime\":null,\"documentType\":\"aplusplus\"},\"journal\":{\"pcode\":\"nature\",\"title\":\"nature\",\"volume\":\"621\",\"issue\":\"7978\"},\"authorization\":{\"status\":true},\"features\":[{\"name\":\"furtherReadingSection\",\"present\":true}],\"collection\":null},\"page\":{\"category\":{\"pageType\":\"article\"},\"attributes\":{\"template\":\"mosaic\",\"featureFlags\":[{\"name\":\"nature-onwards-journey\",\"active\":false},{\"name\":\"getftr-entitled\",\"active\":false},{\"name\":\"inline_recommendations\",\"active\":true}],\"testGroup\":null},\"search\":null},\"privacy\":{},\"version\":\"1.0.0\",\"product\":null,\"session\":null,\"user\":null,\"backHalfContent\":true,\"country\":\"GB\",\"hasBody\":true,\"uneditedManuscript\":false,\"twitterId\":[\"o3xnx\",\"o43y9\",\"o3ef7\"],\"baiduId\":\"d38bce82bcb44717ccc29a90c4b781ea\",\"japan\":false}];\\n    window.dataLayer.push({\\n        ga4MeasurementId: \\'G-ERRNTNZ807\\',\\n        ga360TrackingId: \\'UA-71668177-1\\',\\n        twitterId: [\\'3xnx\\', \\'o43y9\\', \\'o3ef7\\'],\\n        baiduId: \\'d38bce82bcb44717ccc29a90c4b781ea\\',\\n        ga4ServerUrl: \\'https://collect.nature.com\\',\\n        imprint: \\'nature\\'\\n    });\\n</script>\\n\\n<script>\\n    (function(w, d) {\\n        w.config = w.config || {};\\n        w.config.mustardcut = false;\\n\\n        \\n        if (w.matchMedia && w.matchMedia(\\'only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)\\').matches) {\\n            w.config.mustardcut = true;\\n            d.classList.add(\\'js\\');\\n            d.classList.remove(\\'grade-c\\');\\n            d.classList.remove(\\'no-js\\');\\n        }\\n    })(window, document.documentElement);\\n</script>\\n \\n\\n\\n\\n     \\n    \\n    \\n        \\n    \\n    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:\", \"}.c-author-list>li:not(:only-child):last-child:before{content:\" & \"}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:\" ... \"}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:\"\";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url(\"data:image/svg+xml,%3Csvg height=\\'16\\' viewBox=\\'0 0 16 16\\' width=\\'16\\' xmlns=\\'http://www.w3.org/2000/svg\\'%3E%3Cpath d=\\'m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z\\' fill=\\'%23333\\' fill-rule=\\'evenodd\\' transform=\\'matrix(0 1 -1 0 11 3)\\'/%3E%3C/svg%3E\");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:\"\";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>\\n\\n\\n\\n\\n    \\n        <link data-test=\"critical-css-handler\" data-inline-css-source=\"critical-css\" rel=\"stylesheet\" href=\"/static/css/enhanced-article-nature-branded-185f696cd4.css\" media=\"print\" onload=\"this.media=\\'only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)\\';this.onload=null\">\\n    \\n    <noscript>\\n        <link rel=\"stylesheet\" type=\"text/css\" href=\"/static/css/enhanced-article-nature-branded-185f696cd4.css\" media=\"only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)\">\\n    </noscript>\\n\\n<link rel=\"stylesheet\" type=\"text/css\" href=\"/static/css/article-print-122346e276.css\" media=\"print\">\\n    \\n\\n\\n\\n<link rel=\"apple-touch-icon\" sizes=\"180x180\" href=/static/images/favicons/nature/apple-touch-icon-f39cb19454.png>\\n<link rel=\"icon\" type=\"image/png\" sizes=\"32x32\" href=/static/images/favicons/nature/favicon-32x32-3fe59ece92.png>\\n<link rel=\"icon\" type=\"image/png\" sizes=\"16x16\" href=/static/images/favicons/nature/favicon-16x16-951651ab72.png>\\n<link rel=\"manifest\" href=/static/manifest.json crossorigin=\"use-credentials\">\\n<link rel=\"mask-icon\" href=/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg color=\"#000000\">\\n<link rel=\"shortcut icon\" href=/static/images/favicons/nature/favicon.ico>\\n<meta name=\"msapplication-TileColor\" content=\"#000000\">\\n<meta name=\"msapplication-config\" content=/static/browserconfig.xml>\\n<meta name=\"theme-color\" content=\"#000000\">\\n<meta name=\"application-name\" content=\"Nature\">\\n\\n\\n<script>\\n    (function () {\\n        if ( typeof window.CustomEvent === \"function\" ) return false;\\n        function CustomEvent ( event, params ) {\\n            params = params || { bubbles: false, cancelable: false, detail: null };\\n            var evt = document.createEvent( \\'CustomEvent\\' );\\n            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );\\n            return evt;\\n        }\\n\\n        CustomEvent.prototype = window.Event.prototype;\\n\\n        window.CustomEvent = CustomEvent;\\n    })();\\n</script>\\n\\n\\n\\n<!-- Google Tag Manager -->\\n<script data-test=\"gtm-head\">\\n    window.initGTM = function() {\\n        if (window.config.mustardcut) {\\n            (function (w, d, s, l, i) {\\n                w[l] = w[l] || [];\\n                w[l].push({\\'gtm.start\\': new Date().getTime(), event: \\'gtm.js\\'});\\n                var f = d.getElementsByTagName(s)[0],\\n                        j = d.createElement(s),\\n                        dl = l != \\'dataLayer\\' ? \\'&l=\\' + l : \\'\\';\\n                j.async = true;\\n                j.src = \\'https://www.googletagmanager.com/gtm.js?id=\\' + i + dl;\\n                f.parentNode.insertBefore(j, f);\\n            })(window, document, \\'script\\', \\'dataLayer\\', \\'GTM-MRVXSHQ\\');\\n        }\\n    }\\n</script>\\n<!-- End Google Tag Manager -->\\n\\n    <script>\\n    (function(w,d,t) {\\n        function cc() {\\n            var h = w.location.hostname;\\n            if (h.indexOf(\\'preview-www.nature.com\\') > -1) return;\\n\\n            var e = d.createElement(t),\\n                    s = d.getElementsByTagName(t)[0];\\n\\n            if (h.indexOf(\\'nature.com\\') > -1) {\\n                if (h.indexOf(\\'test-www.nature.com\\') > -1) {\\n                    e.src = \\'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js\\';\\n                    e.setAttribute(\\'onload\\', \"initGTM(window,document,\\'script\\',\\'dataLayer\\',\\'GTM-MRVXSHQ\\')\");\\n                } else {\\n                    e.src = \\'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js\\';\\n                    e.setAttribute(\\'onload\\', \"initGTM(window,document,\\'script\\',\\'dataLayer\\',\\'GTM-MRVXSHQ\\')\");\\n                }\\n            } else {\\n                e.src = \\'/static/js/cookie-consent-es5-bundle-2b0f06c1e4.js\\';\\n                e.setAttribute(\\'data-consent\\', h);\\n            }\\n            s.insertAdjacentElement(\\'afterend\\', e);\\n        }\\n\\n        cc();\\n    })(window,document,\\'script\\');\\n</script>\\n\\n\\n<script id=\"js-position0\">\\n    (function(w, d) {\\n        w.idpVerifyPrefix = \\'https://verify.nature.com\\';\\n        w.ra21Host = \\'https://wayf.springernature.com\\';\\n        var moduleSupport = (function() {\\n            return \\'noModule\\' in d.createElement(\\'script\\');\\n        })();\\n\\n        var polyfillsUrl = function() {\\n            var features = {\\n                \\'IntersectionObserver\\': window.IntersectionObserver,\\n                \\'Promise\\': window.Promise,\\n                \\'URLSearchParams\\': window.URLSearchParams,\\n                \\'Symbol.iterator\\': window.Symbol && Symbol.iterator,\\n                \\'Array.from\\': Array.from,\\n                \\'Array.prototype.includes\\': Array.prototype.includes,\\n                \\'Array.prototype.find\\': Array.prototype.find,\\n                \\'Array.prototype.forEach\\': Array.prototype.forEach,\\n                \\'NodeList.prototype.forEach\\': NodeList.prototype.forEach,\\n                \\'Element.prototype.closest\\': Element.prototype.closest,\\n                \\'Element.prototype.prepend\\': Element.prototype.prepend,\\n                \\'Element.prototype.remove\\': Element.prototype.remove,\\n                \\'Object.assign\\': Object.assign\\n            };\\n            var req = [];\\n            for (var feature in features) {\\n                if (Object.prototype.hasOwnProperty.call(features, feature) && !features[feature]) {\\n                    req.push(feature);\\n                }\\n            }\\n            if (req.length) {\\n                return \\'https://polyfill.io/v3/polyfill.min.js?features=\\' + req.join(\\'%2C\\') + \\'&flags=always\\';\\n            }\\n            return null;\\n        };\\n\\n        if (w.config.mustardcut === true) {\\n            w.loader = {\\n                index: 0,\\n                registered: [],\\n                scripts: [\\n                    {src: polyfillsUrl(), test: \\'polyfills-js\\', noinit: true},\\n                    \\n                        {src: \\'/static/js/global-article-es6-bundle-f5393c8c3e.js\\', test: \\'global-article-js\\', module: true},\\n                        {src: \\'/static/js/global-article-es5-bundle-3e27e3c833.js\\', test: \\'global-article-js\\', nomodule: true},\\n                        {src: \\'/static/js/shared-es6-bundle-b4b4f7ba97.js\\', test: \\'shared-js\\', module: true},\\n                        {src: \\'/static/js/shared-es5-bundle-93cdd19b66.js\\', test: \\'shared-js\\', nomodule: true},\\n                        {src: \\'/static/js/header-150-es6-bundle-5bb959eaa1.js\\', test: \\'header-150-js\\', module: true},\\n                        {src: \\'/static/js/header-150-es5-bundle-c634a291c7.js\\', test: \\'header-150-js\\', nomodule: true}\\n                    \\n                ].filter(function (s) {\\n                    if (s.src === null) return false;\\n                    if (moduleSupport && s.nomodule) return false;\\n                    return !(!moduleSupport && s.module);\\n                }),\\n\\n                register: function (value) {\\n                    this.registered.push(value);\\n                },\\n\\n                ready: function () {\\n                    if (this.registered.length === this.scripts.length) {\\n                        this.registered.forEach(function (fn) {\\n                            if (typeof fn === \\'function\\') {\\n                                setTimeout(fn, 0); \\n                            }\\n                        });\\n                        this.ready = function () {};\\n                    }\\n                },\\n\\n                insert: function (s) {\\n                    var t = d.getElementById(\\'js-position\\' + this.index);\\n                    if (t && t.insertAdjacentElement) {\\n                        t.insertAdjacentElement(\\'afterend\\', s);\\n                    } else {\\n                        d.head.appendChild(s);\\n                    }\\n                    ++this.index;\\n                },\\n\\n                createScript: function (script, beforeLoad) {\\n                    var s = d.createElement(\\'script\\');\\n                    s.id = \\'js-position\\' + (this.index + 1);\\n                    s.setAttribute(\\'data-test\\', script.test);\\n                    if (beforeLoad) {\\n                        s.defer = \\'defer\\';\\n                        s.onload = function () {\\n                            if (script.noinit) {\\n                                loader.register(true);\\n                            }\\n                            if (d.readyState === \\'interactive\\' || d.readyState === \\'complete\\') {\\n                                loader.ready();\\n                            }\\n                        };\\n                    } else {\\n                        s.async = \\'async\\';\\n                    }\\n                    s.src = script.src;\\n                    return s;\\n                },\\n\\n                init: function () {\\n                    this.scripts.forEach(function (s) {\\n                        loader.insert(loader.createScript(s, true));\\n                    });\\n\\n                    d.addEventListener(\\'DOMContentLoaded\\', function () {\\n                        loader.ready();\\n                        var conditionalScripts;\\n                        \\n                            conditionalScripts = [\\n                                {match: \\'div[data-pan-container]\\', src: \\'/static/js/pan-zoom-es6-bundle-464a2af269.js\\', test: \\'pan-zoom-js\\',  module: true },\\n                                {match: \\'div[data-pan-container]\\', src: \\'/static/js/pan-zoom-es5-bundle-2626f1bdf8.js\\', test: \\'pan-zoom-js\\',  nomodule: true },\\n                                {match: \\'math,span.mathjax-tex\\', src: \\'/static/js/math-es6-bundle-d8f48a7381.js\\', test: \\'math-js\\', module: true},\\n                                {match: \\'math,span.mathjax-tex\\', src: \\'/static/js/math-es5-bundle-c0c8c3515b.js\\', test: \\'math-js\\', nomodule: true}\\n                            ];\\n                        \\n\\n                        if (conditionalScripts) {\\n                            conditionalScripts.filter(function (script) {\\n                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));\\n                            }).forEach(function (script) {\\n                                loader.insert(loader.createScript(script));\\n                            });\\n                        }\\n                    }, false);\\n                }\\n            };\\n            loader.init();\\n        }\\n    })(window, document);\\n</script>\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n<meta name=\"robots\" content=\"noarchive\">\\n<meta name=\"access\" content=\"Yes\">\\n\\n\\n<link rel=\"search\" href=\"https://www.nature.com/search\">\\n<link rel=\"search\" href=\"https://www.nature.com/opensearch/opensearch.xml\" type=\"application/opensearchdescription+xml\" title=\"nature.com\">\\n<link rel=\"search\" href=\"https://www.nature.com/opensearch/request\" type=\"application/sru+xml\" title=\"nature.com\">\\n\\n\\n\\n\\n\\n    \\n    <script type=\"application/ld+json\">{\"mainEntity\":{\"headline\":\"Single-cell brain organoid screening identifies developmental defects in autism\",\"description\":\"The development of the human brain involves unique processes (not observed in many other species) that can contribute to neurodevelopmental disorders1–4. Cerebral organoids enable the study of neurodevelopmental disorders in a human context. We have developed the CRISPR–human organoids–single-cell RNA sequencing\\xa0(CHOOSE) system, which uses verified pairs of guide RNAs, inducible CRISPR–Cas9-based genetic disruption and single-cell transcriptomics for pooled loss-of-function screening in mosaic organoids. Here we show that\\xa0perturbation of 36 high-risk autism spectrum disorder genes related to transcriptional regulation uncovers their effects on cell fate determination. We find that dorsal intermediate progenitors, ventral progenitors and upper-layer excitatory neurons are among the most vulnerable cell types. We construct a developmental gene regulatory network of cerebral organoids from single-cell transcriptomes and chromatin modalities and identify autism spectrum disorder-associated and perturbation-enriched regulatory modules. Perturbing members of the BRG1/BRM-associated factor (BAF) chromatin remodelling complex leads to enrichment of ventral telencephalon progenitors. Specifically, mutating the BAF subunit ARID1B affects the fate transition of progenitors to oligodendrocyte and interneuron precursor cells, a phenotype that we confirmed in patient-specific induced pluripotent stem cell-derived organoids. Our study paves the way for high-throughput phenotypic characterization of disease susceptibility genes in organoid models with cell state, molecular pathway and gene regulatory network readouts. We develop a high-throughput CRISPR screening system in cerebral organoids and identify vulnerable\\xa0cell types and gene regulatory networks associated\\xa0with\\xa0autism spectrum disorder\\xa0from single-cell transcriptomes and chromatin modalities.\",\"datePublished\":\"2023-09-13T00:00:00Z\",\"dateModified\":\"2023-11-14T00:00:00Z\",\"pageStart\":\"373\",\"pageEnd\":\"380\",\"license\":\"http://creativecommons.org/licenses/by/4.0/\",\"sameAs\":\"https://doi.org/10.1038/s41586-023-06473-y\",\"keywords\":[\"Development of the nervous system\",\"Stem-cell biotechnology\",\"Stem cells\",\"Science\",\"Humanities and Social Sciences\",\"multidisciplinary\"],\"image\":[\"https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig1_HTML.png\",\"https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig2_HTML.png\",\"https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig3_HTML.png\",\"https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig4_HTML.png\"],\"isPartOf\":{\"name\":\"Nature\",\"issn\":[\"1476-4687\",\"0028-0836\"],\"volumeNumber\":\"621\",\"@type\":[\"Periodical\",\"PublicationVolume\"]},\"publisher\":{\"name\":\"Nature Publishing Group UK\",\"logo\":{\"url\":\"https://www.springernature.com/app-sn/public/images/logo-springernature.png\",\"@type\":\"ImageObject\"},\"@type\":\"Organization\"},\"author\":[{\"name\":\"Chong Li\",\"url\":\"http://orcid.org/0000-0003-2823-7360\",\"affiliation\":[{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA)\",\"address\":{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"email\":\"chong.li@imba.oeaw.ac.at\",\"@type\":\"Person\"},{\"name\":\"Jonas Simon Fleck\",\"affiliation\":[{\"name\":\"ETH Zürich\",\"address\":{\"name\":\"Department of Biosystems Science and Engineering, ETH Zürich, Basel, Switzerland\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"@type\":\"Person\"},{\"name\":\"Catarina Martins-Costa\",\"url\":\"http://orcid.org/0000-0003-2018-898X\",\"affiliation\":[{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA)\",\"address\":{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"@type\":\"Person\"},{\"name\":\"Thomas R. Burkard\",\"url\":\"http://orcid.org/0000-0002-0884-2929\",\"affiliation\":[{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA)\",\"address\":{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"@type\":\"Person\"},{\"name\":\"Jan Themann\",\"url\":\"http://orcid.org/0009-0004-0892-2138\",\"affiliation\":[{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA)\",\"address\":{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"@type\":\"Person\"},{\"name\":\"Marlene Stuempflen\",\"url\":\"http://orcid.org/0000-0001-9763-9422\",\"affiliation\":[{\"name\":\"Medical University of Vienna\",\"address\":{\"name\":\"Department of Radiodiagnostics, Medical University of Vienna, Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"@type\":\"Person\"},{\"name\":\"Angela Maria Peer\",\"affiliation\":[{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA)\",\"address\":{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"@type\":\"Person\"},{\"name\":\"Ábel Vertesy\",\"url\":\"http://orcid.org/0000-0001-6075-5702\",\"affiliation\":[{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA)\",\"address\":{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"@type\":\"Person\"},{\"name\":\"Jamie B. Littleboy\",\"url\":\"http://orcid.org/0000-0002-1847-0794\",\"affiliation\":[{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA)\",\"address\":{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"@type\":\"Person\"},{\"name\":\"Christopher Esk\",\"url\":\"http://orcid.org/0000-0003-1833-0801\",\"affiliation\":[{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA)\",\"address\":{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"},{\"name\":\"University of Innsbruck\",\"address\":{\"name\":\"Institute of Molecular Biology, University of Innsbruck, Innsbruck, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"@type\":\"Person\"},{\"name\":\"Ulrich Elling\",\"url\":\"http://orcid.org/0000-0002-4514-2809\",\"affiliation\":[{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA)\",\"address\":{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"@type\":\"Person\"},{\"name\":\"Gregor Kasprian\",\"affiliation\":[{\"name\":\"Medical University of Vienna\",\"address\":{\"name\":\"Department of Radiodiagnostics, Medical University of Vienna, Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"@type\":\"Person\"},{\"name\":\"Nina S. Corsini\",\"url\":\"http://orcid.org/0000-0001-6731-3544\",\"affiliation\":[{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA)\",\"address\":{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"@type\":\"Person\"},{\"name\":\"Barbara Treutlein\",\"url\":\"http://orcid.org/0000-0002-3299-5597\",\"affiliation\":[{\"name\":\"ETH Zürich\",\"address\":{\"name\":\"Department of Biosystems Science and Engineering, ETH Zürich, Basel, Switzerland\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"email\":\"barbara.treutlein@bsse.ethz.ch\",\"@type\":\"Person\"},{\"name\":\"Juergen A. Knoblich\",\"url\":\"http://orcid.org/0000-0002-6751-3404\",\"affiliation\":[{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA)\",\"address\":{\"name\":\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"},{\"name\":\"Medical University of Vienna\",\"address\":{\"name\":\"Department of Neurology, Medical University of Vienna, Vienna, Austria\",\"@type\":\"PostalAddress\"},\"@type\":\"Organization\"}],\"email\":\"juergen.knoblich@imba.oeaw.ac.at\",\"@type\":\"Person\"}],\"isAccessibleForFree\":true,\"@type\":\"ScholarlyArticle\"},\"@context\":\"https://schema.org\",\"@type\":\"WebPage\"}</script>\\n\\n\\n\\n\\n    \\n    \\n    \\n\\n\\n    \\n    <link rel=\"canonical\" href=\"https://www.nature.com/articles/s41586-023-06473-y\">\\n    \\n    \\n    <meta name=\"journal_id\" content=\"41586\"/>\\n    <meta name=\"dc.title\" content=\"Single-cell brain organoid screening identifies developmental defects in autism\"/>\\n    <meta name=\"dc.source\" content=\"Nature 2023 621:7978\"/>\\n    <meta name=\"dc.format\" content=\"text/html\"/>\\n    <meta name=\"dc.publisher\" content=\"Nature Publishing Group\"/>\\n    <meta name=\"dc.date\" content=\"2023-09-13\"/>\\n    <meta name=\"dc.type\" content=\"OriginalPaper\"/>\\n    <meta name=\"dc.language\" content=\"En\"/>\\n    <meta name=\"dc.copyright\" content=\"2023 The Author(s)\"/>\\n    <meta name=\"dc.rights\" content=\"2023 The Author(s)\"/>\\n    <meta name=\"dc.rightsAgent\" content=\"journalpermissions@springernature.com\"/>\\n    <meta name=\"dc.description\" content=\"The development of the human brain involves unique processes (not observed in many other species) that can contribute to neurodevelopmental disorders1&#8211;4. Cerebral organoids enable the study of neurodevelopmental disorders in a human context. We have developed the CRISPR&#8211;human organoids&#8211;single-cell RNA sequencing&amp;nbsp;(CHOOSE) system, which uses verified pairs of guide RNAs, inducible CRISPR&#8211;Cas9-based genetic disruption and single-cell transcriptomics for pooled loss-of-function screening in mosaic organoids. Here we show that&amp;nbsp;perturbation of 36 high-risk autism spectrum disorder genes related to transcriptional regulation uncovers their effects on cell fate determination. We find that dorsal intermediate progenitors, ventral progenitors and upper-layer excitatory neurons are among the most vulnerable cell types. We construct a developmental gene regulatory network of cerebral organoids from single-cell transcriptomes and chromatin modalities and identify autism spectrum disorder-associated and perturbation-enriched regulatory modules. Perturbing members of the BRG1/BRM-associated factor (BAF) chromatin remodelling complex leads to enrichment of ventral telencephalon progenitors. Specifically, mutating the BAF subunit ARID1B affects the fate transition of progenitors to oligodendrocyte and interneuron precursor cells, a phenotype that we confirmed in patient-specific induced pluripotent stem cell-derived organoids. Our study paves the way for high-throughput phenotypic characterization of disease susceptibility genes in organoid models with cell state, molecular pathway and gene regulatory network readouts. We develop a high-throughput CRISPR screening system in cerebral organoids and identify vulnerable&amp;nbsp;cell types and gene regulatory networks associated&amp;nbsp;with&amp;nbsp;autism spectrum disorder&amp;nbsp;from single-cell transcriptomes and chromatin modalities.\"/>\\n    <meta name=\"prism.issn\" content=\"1476-4687\"/>\\n    <meta name=\"prism.publicationName\" content=\"Nature\"/>\\n    <meta name=\"prism.publicationDate\" content=\"2023-09-13\"/>\\n    <meta name=\"prism.volume\" content=\"621\"/>\\n    <meta name=\"prism.number\" content=\"7978\"/>\\n    <meta name=\"prism.section\" content=\"OriginalPaper\"/>\\n    <meta name=\"prism.startingPage\" content=\"373\"/>\\n    <meta name=\"prism.endingPage\" content=\"380\"/>\\n    <meta name=\"prism.copyright\" content=\"2023 The Author(s)\"/>\\n    <meta name=\"prism.rightsAgent\" content=\"journalpermissions@springernature.com\"/>\\n    <meta name=\"prism.url\" content=\"https://www.nature.com/articles/s41586-023-06473-y\"/>\\n    <meta name=\"prism.doi\" content=\"doi:10.1038/s41586-023-06473-y\"/>\\n    <meta name=\"citation_pdf_url\" content=\"https://www.nature.com/articles/s41586-023-06473-y.pdf\"/>\\n    <meta name=\"citation_fulltext_html_url\" content=\"https://www.nature.com/articles/s41586-023-06473-y\"/>\\n    <meta name=\"citation_journal_title\" content=\"Nature\"/>\\n    <meta name=\"citation_journal_abbrev\" content=\"Nature\"/>\\n    <meta name=\"citation_publisher\" content=\"Nature Publishing Group\"/>\\n    <meta name=\"citation_issn\" content=\"1476-4687\"/>\\n    <meta name=\"citation_title\" content=\"Single-cell brain organoid screening identifies developmental defects in autism\"/>\\n    <meta name=\"citation_volume\" content=\"621\"/>\\n    <meta name=\"citation_issue\" content=\"7978\"/>\\n    <meta name=\"citation_publication_date\" content=\"2023/09\"/>\\n    <meta name=\"citation_online_date\" content=\"2023/09/13\"/>\\n    <meta name=\"citation_firstpage\" content=\"373\"/>\\n    <meta name=\"citation_lastpage\" content=\"380\"/>\\n    <meta name=\"citation_article_type\" content=\"Article\"/>\\n    <meta name=\"citation_fulltext_world_readable\" content=\"\"/>\\n    <meta name=\"citation_language\" content=\"en\"/>\\n    <meta name=\"dc.identifier\" content=\"doi:10.1038/s41586-023-06473-y\"/>\\n    <meta name=\"DOI\" content=\"10.1038/s41586-023-06473-y\"/>\\n    <meta name=\"size\" content=\"296529\"/>\\n    <meta name=\"citation_doi\" content=\"10.1038/s41586-023-06473-y\"/>\\n    <meta name=\"citation_springer_api_url\" content=\"http://api.springer.com/xmldata/jats?q=doi:10.1038/s41586-023-06473-y&amp;api_key=\"/>\\n    <meta name=\"description\" content=\"The development of the human brain involves unique processes (not observed in many other species) that can contribute to neurodevelopmental disorders1&#8211;4. Cerebral organoids enable the study of neurodevelopmental disorders in a human context. We have developed the CRISPR&#8211;human organoids&#8211;single-cell RNA sequencing&amp;nbsp;(CHOOSE) system, which uses verified pairs of guide RNAs, inducible CRISPR&#8211;Cas9-based genetic disruption and single-cell transcriptomics for pooled loss-of-function screening in mosaic organoids. Here we show that&amp;nbsp;perturbation of 36 high-risk autism spectrum disorder genes related to transcriptional regulation uncovers their effects on cell fate determination. We find that dorsal intermediate progenitors, ventral progenitors and upper-layer excitatory neurons are among the most vulnerable cell types. We construct a developmental gene regulatory network of cerebral organoids from single-cell transcriptomes and chromatin modalities and identify autism spectrum disorder-associated and perturbation-enriched regulatory modules. Perturbing members of the BRG1/BRM-associated factor (BAF) chromatin remodelling complex leads to enrichment of ventral telencephalon progenitors. Specifically, mutating the BAF subunit ARID1B affects the fate transition of progenitors to oligodendrocyte and interneuron precursor cells, a phenotype that we confirmed in patient-specific induced pluripotent stem cell-derived organoids. Our study paves the way for high-throughput phenotypic characterization of disease susceptibility genes in organoid models with cell state, molecular pathway and gene regulatory network readouts. We develop a high-throughput CRISPR screening system in cerebral organoids and identify vulnerable&amp;nbsp;cell types and gene regulatory networks associated&amp;nbsp;with&amp;nbsp;autism spectrum disorder&amp;nbsp;from single-cell transcriptomes and chromatin modalities.\"/>\\n    <meta name=\"dc.creator\" content=\"Li, Chong\"/>\\n    <meta name=\"dc.creator\" content=\"Fleck, Jonas Simon\"/>\\n    <meta name=\"dc.creator\" content=\"Martins-Costa, Catarina\"/>\\n    <meta name=\"dc.creator\" content=\"Burkard, Thomas R.\"/>\\n    <meta name=\"dc.creator\" content=\"Themann, Jan\"/>\\n    <meta name=\"dc.creator\" content=\"Stuempflen, Marlene\"/>\\n    <meta name=\"dc.creator\" content=\"Peer, Angela Maria\"/>\\n    <meta name=\"dc.creator\" content=\"Vertesy, &#193;bel\"/>\\n    <meta name=\"dc.creator\" content=\"Littleboy, Jamie B.\"/>\\n    <meta name=\"dc.creator\" content=\"Esk, Christopher\"/>\\n    <meta name=\"dc.creator\" content=\"Elling, Ulrich\"/>\\n    <meta name=\"dc.creator\" content=\"Kasprian, Gregor\"/>\\n    <meta name=\"dc.creator\" content=\"Corsini, Nina S.\"/>\\n    <meta name=\"dc.creator\" content=\"Treutlein, Barbara\"/>\\n    <meta name=\"dc.creator\" content=\"Knoblich, Juergen A.\"/>\\n    <meta name=\"dc.subject\" content=\"Development of the nervous system\"/>\\n    <meta name=\"dc.subject\" content=\"Stem-cell biotechnology\"/>\\n    <meta name=\"dc.subject\" content=\"Stem cells\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Genom. Hum. Genet.; citation_title=The diverse genetic landscape of neurodevelopmental disorders; citation_author=WF Hu, MH Chahrour, CA Walsh; citation_volume=15; citation_publication_date=2014; citation_pages=195-213; citation_doi=10.1146/annurev-genom-090413-025600; citation_id=CR1\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Science; citation_title=Mapping the molecular and cellular complexity of cortical malformations; citation_author=E Klingler, F Francis, D Jabaudon, S Cappello; citation_volume=371; citation_publication_date=2021; citation_pages=eaba4517; citation_doi=10.1126/science.aba4517; citation_id=CR2\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Annu. Rev. Genet.; citation_title=Cellular and molecular mechanisms linking human cortical development and evolution; citation_author=B Lib&#233;-Philippot, P Vanderhaeghen; citation_volume=55; citation_publication_date=2021; citation_pages=555-581; citation_doi=10.1146/annurev-genet-071719-020705; citation_id=CR3\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell; citation_title=Development and evolution of the human neocortex; citation_author=JH Lui, DV Hansen, AR Kriegstein; citation_volume=146; citation_publication_date=2011; citation_pages=18-36; citation_doi=10.1016/j.cell.2011.06.030; citation_id=CR4\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell; citation_title=Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism; citation_author=AJ Willsey; citation_volume=155; citation_publication_date=2013; citation_pages=997-1007; citation_doi=10.1016/j.cell.2013.10.020; citation_id=CR5\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell; citation_title=Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism; citation_author=NN Parikshak; citation_volume=155; citation_publication_date=2013; citation_pages=1008-1021; citation_doi=10.1016/j.cell.2013.10.031; citation_id=CR6\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell; citation_title=Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism; citation_author=FK Satterstrom; citation_volume=180; citation_publication_date=2020; citation_pages=568-584.e23; citation_doi=10.1016/j.cell.2019.12.036; citation_id=CR7\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Med.; citation_title=Advancing the understanding of autism disease mechanisms through genetics; citation_author=L Torre-Ubieta, H Won, JL Stein, DH Geschwind; citation_volume=22; citation_publication_date=2016; citation_pages=345-361; citation_doi=10.1038/nm.4071; citation_id=CR8\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nature; citation_title=Cerebral organoids model human brain development and microcephaly; citation_author=MA Lancaster; citation_volume=501; citation_publication_date=2013; citation_pages=373-379; citation_doi=10.1038/nature12517; citation_id=CR9\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell; citation_title=FOXG1-dependent dysregulation of GABA/glutamate neuron differentiation in autism spectrum disorders; citation_author=J Mariani; citation_volume=162; citation_publication_date=2015; citation_pages=375-390; citation_doi=10.1016/j.cell.2015.06.034; citation_id=CR10\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nature; citation_title=Autism genes converge on asynchronous development of shared neuron classes; citation_author=B Paulsen; citation_volume=602; citation_publication_date=2022; citation_pages=268-273; citation_doi=10.1038/s41586-021-04358-6; citation_id=CR11\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Science; citation_title=A human tissue screen identifies a regulator of ER secretion as a brain-size determinant; citation_author=C Esk; citation_volume=370; citation_publication_date=2020; citation_pages=935-941; citation_doi=10.1126/science.abb5390; citation_id=CR12\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell Stem Cell; citation_title=Pooled in vitro and in vivo CRISPR-Cas9 screening identifies tumor suppressors in human colon organoids; citation_author=BE Michels; citation_volume=26; citation_publication_date=2020; citation_pages=782-792.e7; citation_doi=10.1016/j.stem.2020.04.003; citation_id=CR13\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell; citation_title=Perturb-seq: dissecting molecular circuits with scalable single-cell RNA; citation_author=A Dixit; citation_volume=167; citation_publication_date=2016; citation_pages=1853-1866.e17; citation_doi=10.1016/j.cell.2016.11.038; citation_id=CR14\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell; citation_title=Dissecting immune circuits by linking CRISPR-pooled screens with single-cell RNA-seq; citation_author=DA Jaitin; citation_volume=167; citation_publication_date=2016; citation_pages=1883-1896.e15; citation_doi=10.1016/j.cell.2016.11.039; citation_id=CR15\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell; citation_title=A multiplexed single-cell CRISPR screening platform enables systematic dissection of the unfolded protein response; citation_author=B Adamson; citation_volume=167; citation_publication_date=2016; citation_pages=1867-1882.e21; citation_doi=10.1016/j.cell.2016.11.048; citation_id=CR16\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Methods; citation_title=Pooled CRISPR screening with single-cell transcriptome read-out; citation_author=P Datlinger; citation_volume=14; citation_publication_date=2017; citation_pages=297-301; citation_doi=10.1038/nmeth.4177; citation_id=CR17\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Rev. Genet.; citation_title=Am I ready for CRISPR? A user&#8217;s guide to genetic screens; citation_author=JG Doench; citation_volume=19; citation_publication_date=2018; citation_pages=67-80; citation_doi=10.1038/nrg.2017.97; citation_id=CR18\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Science; citation_title=Landmarks of human embryonic development inscribed in somatic mutations; citation_author=S Bizzotto; citation_volume=371; citation_publication_date=2021; citation_pages=1249-1253; citation_doi=10.1126/science.abe1544; citation_id=CR19\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Pathology Mech. Dis.; citation_title=The emerging picture of autism spectrum disorder: genetics and pathology; citation_author=JA Chen, O Pe&#241;agarikano, TG Belgard, V Swarup, DH Geschwind; citation_volume=10; citation_publication_date=2015; citation_pages=111-144; citation_doi=10.1146/annurev-pathol-012414-040405; citation_id=CR20\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nature; citation_title=Synaptic, transcriptional, and chromatin genes disrupted in autism; citation_author=TD Study; citation_volume=515; citation_publication_date=2014; citation_pages=209-215; citation_doi=10.1038/nature13772; citation_id=CR21\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell; citation_title=Chromatin and gene-regulatory dynamics of the developing human cerebral cortex at single-cell resolution; citation_author=AE Trevino; citation_volume=184; citation_publication_date=2021; citation_pages=5053-5069.e23; citation_doi=10.1016/j.cell.2021.07.039; citation_id=CR22\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Biotechnol.; citation_title=Guided self-organization and cortical plate formation in human brain organoids; citation_author=MA Lancaster; citation_volume=35; citation_publication_date=2017; citation_pages=659-666; citation_doi=10.1038/nbt.3906; citation_id=CR23\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Science; citation_title=Amplification of human interneuron progenitors promotes brain tumors and neurological defects; citation_author=OL Eichm&#252;ller; citation_volume=375; citation_publication_date=2022; citation_pages=eabf5546; citation_doi=10.1126/science.abf5546; citation_id=CR24\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell Rep.; citation_title=Transcriptional repression by FEZF2 restricts alternative identities of cortical projection neurons; citation_author=J Tsyporin; citation_volume=35; citation_publication_date=2021; citation_pages=109269-109269; citation_doi=10.1016/j.celrep.2021.109269; citation_id=CR25\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cereb. Cortex; citation_title=Identification of the genes that are expressed in the upper layers of the neocortex; citation_author=Y Zhong; citation_volume=14; citation_publication_date=2004; citation_pages=1144-1152; citation_doi=10.1093/cercor/bhh074; citation_id=CR26\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=eLife; citation_title=Identity of neocortical layer 4 neurons is specified through correct positioning into the cortex; citation_author=K Oishi; citation_volume=5; citation_publication_date=2016; citation_pages=e10907; citation_doi=10.7554/eLife.10907; citation_id=CR27\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Science; citation_title=Mouse and human share conserved transcriptional programs for interneuron development; citation_author=Y Shi; citation_volume=374; citation_publication_date=2021; citation_pages=eabj6641; citation_doi=10.1126/science.abj6641; citation_id=CR28\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nature; citation_title=The development and evolution of inhibitory neurons in primate cerebrum; citation_author=MT Schmitz; citation_volume=603; citation_publication_date=2022; citation_pages=871-877; citation_doi=10.1038/s41586-022-04510-w; citation_id=CR29\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nature; citation_title=RNA velocity of single cells; citation_author=GL Manno; citation_volume=560; citation_publication_date=2018; citation_pages=494-498; citation_doi=10.1038/s41586-018-0414-6; citation_id=CR30\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Biotechnol.; citation_title=Generalizing RNA velocity to transient cell states through dynamical modeling; citation_author=V Bergen, M Lange, S Peidli, FA Wolf, FJ Theis; citation_volume=38; citation_publication_date=2020; citation_pages=1408-1414; citation_doi=10.1038/s41587-020-0591-3; citation_id=CR31\"/>\\n    <meta name=\"citation_reference\" content=\"Braun, E. et al. Comprehensive cell atlas of the first-trimester developing human brain. Preprint at bioRxiv \\n                  https://doi.org/10.1101/2022.10.24.513487\\n                  \\n                 (2022).\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Mol. Psychiatr.; citation_title=The ASD Living Biology: from cell proliferation to clinical phenotype; citation_author=E Courchesne; citation_volume=24; citation_publication_date=2019; citation_pages=88-107; citation_doi=10.1038/s41380-018-0056-y; citation_id=CR33\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Science; citation_title=Single-cell genomics identifies cell type&#8211;specific molecular changes in autism; citation_author=D Velmeshev; citation_volume=364; citation_publication_date=2019; citation_pages=685-689; citation_doi=10.1126/science.aav8130; citation_id=CR34\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell Stem Cell; citation_title=Sliced human cortical organoids for modeling distinct cortical layer formation; citation_author=X Qian; citation_volume=26; citation_publication_date=2020; citation_pages=766-781.e9; citation_doi=10.1016/j.stem.2020.02.002; citation_id=CR35\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=iScience; citation_title=The autism-related protein SETD5 controls neural cell proliferation through epigenetic regulation of rDNA expression; citation_author=T Nakagawa; citation_volume=23; citation_publication_date=2020; citation_pages=101030; citation_doi=10.1016/j.isci.2020.101030; citation_id=CR36\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Mitochondrial dysfunction and oxidative stress contribute to cognitive and motor impairment in FOXP1 syndrome; citation_author=J Wang; citation_volume=119; citation_publication_date=2022; citation_pages=e2112852119; citation_doi=10.1073/pnas.2112852119; citation_id=CR37\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Circ. Res.; citation_title=IRF2BP2 reduces macrophage inflammation and susceptibility to atherosclerosis; citation_author=H-H Chen; citation_volume=117; citation_publication_date=2015; citation_pages=671-683; citation_doi=10.1161/CIRCRESAHA.114.305777; citation_id=CR38\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell Death Dis.; citation_title=P300/CBP-associated factor (PCAF) inhibits the growth of hepatocellular carcinoma by promoting cell autophagy; citation_author=Y-L Jia; citation_volume=7; citation_publication_date=2016; citation_pages=e2400-e2400; citation_doi=10.1038/cddis.2016.247; citation_id=CR39\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=EMBO Mol. Med.; citation_title=MECP2 mutations affect ciliogenesis: a novel perspective for Rett syndrome and related disorders; citation_author=A Frasca; citation_volume=12; citation_publication_date=2020; citation_pages=e10270; citation_doi=10.15252/emmm.201910270; citation_id=CR40\"/>\\n    <meta name=\"citation_reference\" content=\"Fleck, J.S., Jansen, S.M.J., Wollny, D. et al. Inferring and perturbing cell fate regulomes in human brain organoids. Nature \\n                  https://doi.org/10.1038/s41586-022-05279-8\\n                  \\n                 (2022).\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Biotechnol.; citation_title=Dimensionality reduction for visualizing single-cell data using UMAP; citation_author=E Becht; citation_volume=37; citation_publication_date=2019; citation_pages=38-44; citation_doi=10.1038/nbt.4314; citation_id=CR42\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Methods; citation_title=CellRank for directed single-cell fate mapping; citation_author=M Lange; citation_volume=19; citation_publication_date=2022; citation_pages=159-170; citation_doi=10.1038/s41592-021-01346-6; citation_id=CR43\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Neuron; citation_title=Dlx1 and Dlx2 control neuronal versus oligodendroglial cell fate acquisition in the developing forebrain; citation_author=MA Petryniak, GB Potter, DH Rowitch, JLR Rubenstein; citation_volume=55; citation_publication_date=2007; citation_pages=417-433; citation_doi=10.1016/j.neuron.2007.06.036; citation_id=CR44\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Neuron; citation_title=Phosphorylation state of Olig2 regulates proliferation of neural progenitors; citation_author=Y Sun; citation_volume=69; citation_publication_date=2011; citation_pages=906-917; citation_doi=10.1016/j.neuron.2011.02.005; citation_id=CR45\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Mol. Psychiatr.; citation_title=Neurobiology of ARID1B haploinsufficiency related to neurodevelopmental and psychiatric disorders; citation_author=JJ Moffat, AL Smith, E-M Jung, M Ka, W-Y Kim; citation_volume=27; citation_publication_date=2022; citation_pages=476-489; citation_doi=10.1038/s41380-021-01060-x; citation_id=CR46\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Methods; citation_title=Fused cerebral organoids model interactions between brain regions; citation_author=JA Bagley, D Reumann, S Bian, J L&#233;vi-Strauss, JA Knoblich; citation_volume=14; citation_publication_date=2017; citation_pages=743-751; citation_doi=10.1038/nmeth.4304; citation_id=CR47\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=EMBO J.; citation_title=Neurotransmitter signaling regulates distinct phases of multimodal human interneuron migration; citation_author=S Bajaj; citation_volume=40; citation_publication_date=2021; citation_pages=e108714; citation_doi=10.15252/embj.2021108714; citation_id=CR48\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cereb. Cortex; citation_title=The role of intermediate progenitor cells in the evolutionary expansion of the cerebral cortex; citation_author=V Mart&#237;nez-Cerde&#241;o, SC Noctor, AR Kriegstein; citation_volume=16; citation_publication_date=2006; citation_pages=i152-i161; citation_doi=10.1093/cercor/bhk017; citation_id=CR49\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Human intermediate progenitor diversity during cortical development; citation_author=M-P Pebworth, J Ross, M Andrews, A Bhaduri, AR Kriegstein; citation_volume=118; citation_publication_date=2021; citation_pages=e2019415118; citation_doi=10.1073/pnas.2019415118; citation_id=CR50\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Neurosci.; citation_title=Competing waves of oligodendrocytes in the forebrain and postnatal elimination of an embryonic lineage; citation_author=N Kessaris; citation_volume=9; citation_publication_date=2006; citation_pages=173-179; citation_doi=10.1038/nn1620; citation_id=CR51\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Front. Mol. Neurosci.; citation_title=Chromatin remodeling BAF (SWI/SNF) complexes in neural development and disorders; citation_author=G Sokpor, Y Xie, J Rosenbusch, T Tuoc; citation_volume=10; citation_publication_date=2017; citation_pages=243; citation_doi=10.3389/fnmol.2017.00243; citation_id=CR52\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell Rep.; citation_title=CHD8 haploinsufficiency links autism to transient alterations in excitatory and inhibitory trajectories; citation_author=CE Villa; citation_volume=39; citation_publication_date=2022; citation_pages=110615; citation_doi=10.1016/j.celrep.2022.110615; citation_id=CR53\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Methods; citation_title=Multilayered VBC score predicts sgRNAs that efficiently generate loss-of-function alleles; citation_author=G Michlits; citation_volume=17; citation_publication_date=2020; citation_pages=708-716; citation_doi=10.1038/s41592-020-0850-8; citation_id=CR54\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Methods; citation_title=Improved vectors and genome-wide libraries for CRISPR screening; citation_author=NE Sanjana, O Shalem, F Zhang; citation_volume=11; citation_publication_date=2014; citation_pages=783-784; citation_doi=10.1038/nmeth.3047; citation_id=CR55\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=BMC Genom.; citation_title=Optimized PCR conditions minimizing the formation of chimeric DNA molecules from MPRA plasmid libraries; citation_author=ES Omelina, AV Ivankin, AE Letiagina, AV Pindyurin; citation_volume=20; citation_publication_date=2019; citation_doi=10.1186/s12864-019-5847-2; citation_id=CR56\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Methods; citation_title=Amplification of complex gene libraries by emulsion PCR; citation_author=R Williams; citation_volume=3; citation_publication_date=2006; citation_pages=545-550; citation_doi=10.1038/nmeth896; citation_id=CR57\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=EMBnet J; citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads; citation_author=M Martin; citation_volume=17; citation_publication_date=2011; citation_pages=10-12; citation_doi=10.14806/ej.17.1.200; citation_id=CR58\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Commun.; citation_title=Massively parallel digital transcriptional profiling of single cells; citation_author=GXY Zheng; citation_volume=8; citation_publication_date=2017; citation_doi=10.1038/ncomms14049; citation_id=CR59\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Cell; citation_title=Integrated analysis of multimodal single-cell data; citation_author=Y Hao; citation_volume=184; citation_publication_date=2021; citation_pages=3573-3587.e29; citation_doi=10.1016/j.cell.2021.04.048; citation_id=CR60\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Methods; citation_title=Fast, sensitive and accurate integration of single-cell data with Harmony; citation_author=I Korsunsky; citation_volume=16; citation_publication_date=2019; citation_pages=1289-1296; citation_doi=10.1038/s41592-019-0619-0; citation_id=CR61\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=J Statist. Mech. Theory Exp.; citation_title=Fast unfolding of communities in large networks; citation_author=VD Blondel, J-L Guillaume, R Lambiotte, E Lefebvre; citation_volume=2008; citation_publication_date=2008; citation_pages=P10008; citation_doi=10.1088/1742-5468/2008/10/P10008; citation_id=CR62\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Biotechnol.; citation_title=Near-optimal probabilistic RNA-seq quantification; citation_author=NL Bray, H Pimentel, P Melsted, L Pachter; citation_volume=34; citation_publication_date=2016; citation_pages=525-527; citation_doi=10.1038/nbt.3519; citation_id=CR63\"/>\\n    <meta name=\"citation_reference\" content=\"Nikolova, M. T. et al. Fate and state transitions during human blood vessel organoid development. Preprint at bioRxiv \\n                  https://doi.org/10.1101/2022.03.23.485329\\n                  \\n                 (2022).\"/>\\n    <meta name=\"citation_reference\" content=\"citation_title=ggplot2: Elegant Graphics for Data Analysis; citation_publication_date=2016; citation_id=CR65; citation_author=H Wickham,; citation_publisher=Springer,\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Innovation; citation_title=clusterProfiler 4.0: a universal enrichment tool for interpreting omics data; citation_author=T Wu; citation_volume=2; citation_publication_date=2021; citation_pages=100141; citation_id=CR66\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Methods; citation_title=Single-cell chromatin state analysis with Signac; citation_author=T Stuart, A Srivastava, S Madad, CA Lareau, R Satija; citation_volume=18; citation_publication_date=2021; citation_pages=1333-1341; citation_doi=10.1038/s41592-021-01282-5; citation_id=CR67\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Protoc.; citation_title=Identifying ChIP-seq enrichment using MACS; citation_author=J Feng, T Liu, B Qin, Y Zhang, XS Liu; citation_volume=7; citation_publication_date=2012; citation_pages=1728-1740; citation_doi=10.1038/nprot.2012.101; citation_id=CR68\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Genome Res.; citation_title=Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes; citation_author=A Siepel; citation_volume=15; citation_publication_date=2005; citation_pages=1034-1050; citation_doi=10.1101/gr.3715005; citation_id=CR69\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nature; citation_title=Expanded encyclopaedias of DNA elements in the human and mouse genomes; citation_author=TEP Consortium; citation_volume=583; citation_publication_date=2020; citation_pages=699-710; citation_doi=10.1038/s41586-020-2493-4; citation_id=CR70\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Biotechnol.; citation_title=GREAT improves functional interpretation of cis-regulatory regions; citation_author=CY McLean; citation_volume=28; citation_publication_date=2010; citation_pages=495-501; citation_doi=10.1038/nbt.1630; citation_id=CR71\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Biotechnol.; citation_title=Characterization of cell fate probabilities in single-cell data with Palantir; citation_author=M Setty; citation_volume=37; citation_publication_date=2019; citation_pages=451-460; citation_doi=10.1038/s41587-019-0068-4; citation_id=CR72\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=J. Chem. Theory Comput.; citation_title=Generalized Markov state modeling method for nonequilibrium biomolecular dynamics: exemplified on amyloid &#946; conformational dynamics driven by an oscillating electric field; citation_author=B Reuter, M Weber, K Fackeldey, S R&#246;blitz, ME Garcia; citation_volume=14; citation_publication_date=2018; citation_pages=3579-3594; citation_doi=10.1021/acs.jctc.8b00079; citation_id=CR73\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Nat. Cell Biol.; citation_title=Human haematopoietic stem cell lineage commitment is a continuous process; citation_author=L Velten; citation_volume=19; citation_publication_date=2017; citation_pages=271-281; citation_doi=10.1038/ncb3493; citation_id=CR74\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Stem Cell Rep.; citation_title=Successful generation of human induced pluripotent stem cell lines from blood samples held at room temperature for up to 48 hr; citation_author=CA Agu; citation_volume=5; citation_publication_date=2015; citation_pages=660-671; citation_doi=10.1016/j.stemcr.2015.08.012; citation_id=CR75\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Science; citation_title=A programmable dual-RNA&#8211;guided DNA endonuclease in adaptive bacterial immunity; citation_author=M Jinek; citation_volume=337; citation_publication_date=2012; citation_pages=816-821; citation_doi=10.1126/science.1225829; citation_id=CR76\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Sci. Rep.; citation_title=A normative spatiotemporal MRI atlas of the fetal brain for automatic segmentation and analysis of early brain growth; citation_author=A Gholipour; citation_volume=7; citation_publication_date=2017; citation_doi=10.1038/s41598-017-00525-w; citation_id=CR77\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Neuroimage; citation_title=User-guided 3D active contour segmentation of anatomical structures: significantly improved efficiency and reliability; citation_author=PA Yushkevich; citation_volume=31; citation_publication_date=2006; citation_pages=1116-1128; citation_doi=10.1016/j.neuroimage.2006.01.015; citation_id=CR78\"/>\\n    <meta name=\"citation_reference\" content=\"citation_journal_title=Neuroimage; citation_title=An automated framework for localization, segmentation and super-resolution reconstruction of fetal brain MRI; citation_author=M Ebner; citation_volume=206; citation_publication_date=2020; citation_pages=116324; citation_doi=10.1016/j.neuroimage.2019.116324; citation_id=CR79\"/>\\n    <meta name=\"citation_reference\" content=\"citation_title=The Human Brain during the Second Trimester; citation_publication_date=2005; citation_id=CR80; citation_author=SA Bayer,; citation_author=J Altman,; citation_publisher=Taylor &amp;amp;amp;amp;amp;amp;amp;amp;amp;amp; Francis,\"/>\\n    <meta name=\"citation_reference\" content=\"citation_title=Atlas of Human Central Nervous System Development; citation_publication_date=2003; citation_id=CR81; citation_author=SA Bayer,; citation_author=J Altman,; citation_publisher=Taylor &amp;amp;amp;amp;amp;amp;amp;amp;amp; Francis,\"/>\\n    <meta name=\"citation_author\" content=\"Li, Chong\"/>\\n    <meta name=\"citation_author_institution\" content=\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\"/>\\n    <meta name=\"citation_author\" content=\"Fleck, Jonas Simon\"/>\\n    <meta name=\"citation_author_institution\" content=\"Department of Biosystems Science and Engineering, ETH Z&#252;rich, Basel, Switzerland\"/>\\n    <meta name=\"citation_author\" content=\"Martins-Costa, Catarina\"/>\\n    <meta name=\"citation_author_institution\" content=\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\"/>\\n    <meta name=\"citation_author\" content=\"Burkard, Thomas R.\"/>\\n    <meta name=\"citation_author_institution\" content=\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\"/>\\n    <meta name=\"citation_author\" content=\"Themann, Jan\"/>\\n    <meta name=\"citation_author_institution\" content=\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\"/>\\n    <meta name=\"citation_author\" content=\"Stuempflen, Marlene\"/>\\n    <meta name=\"citation_author_institution\" content=\"Department of Radiodiagnostics, Medical University of Vienna, Vienna, Austria\"/>\\n    <meta name=\"citation_author\" content=\"Peer, Angela Maria\"/>\\n    <meta name=\"citation_author_institution\" content=\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\"/>\\n    <meta name=\"citation_author\" content=\"Vertesy, &#193;bel\"/>\\n    <meta name=\"citation_author_institution\" content=\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\"/>\\n    <meta name=\"citation_author\" content=\"Littleboy, Jamie B.\"/>\\n    <meta name=\"citation_author_institution\" content=\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\"/>\\n    <meta name=\"citation_author\" content=\"Esk, Christopher\"/>\\n    <meta name=\"citation_author_institution\" content=\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\"/>\\n    <meta name=\"citation_author_institution\" content=\"Institute of Molecular Biology, University of Innsbruck, Innsbruck, Austria\"/>\\n    <meta name=\"citation_author\" content=\"Elling, Ulrich\"/>\\n    <meta name=\"citation_author_institution\" content=\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\"/>\\n    <meta name=\"citation_author\" content=\"Kasprian, Gregor\"/>\\n    <meta name=\"citation_author_institution\" content=\"Department of Radiodiagnostics, Medical University of Vienna, Vienna, Austria\"/>\\n    <meta name=\"citation_author\" content=\"Corsini, Nina S.\"/>\\n    <meta name=\"citation_author_institution\" content=\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\"/>\\n    <meta name=\"citation_author\" content=\"Treutlein, Barbara\"/>\\n    <meta name=\"citation_author_institution\" content=\"Department of Biosystems Science and Engineering, ETH Z&#252;rich, Basel, Switzerland\"/>\\n    <meta name=\"citation_author\" content=\"Knoblich, Juergen A.\"/>\\n    <meta name=\"citation_author_institution\" content=\"Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria\"/>\\n    <meta name=\"citation_author_institution\" content=\"Department of Neurology, Medical University of Vienna, Vienna, Austria\"/>\\n    <meta name=\"access_endpoint\" content=\"https://www.nature.com/platform/readcube-access\"/>\\n    <meta name=\"twitter:site\" content=\"@nature\"/>\\n    <meta name=\"twitter:card\" content=\"summary_large_image\"/>\\n    <meta name=\"twitter:image:alt\" content=\"Content cover image\"/>\\n    <meta name=\"twitter:title\" content=\"Single-cell brain organoid screening identifies developmental defects in autism\"/>\\n    <meta name=\"twitter:description\" content=\"Nature - We develop a high-throughput CRISPR screening system in cerebral organoids and identify vulnerable cell types and gene regulatory networks associated with autism spectrum disorder from...\"/>\\n    <meta name=\"twitter:image\" content=\"https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig1_HTML.png\"/>\\n    \\n\\n    \\n    \\n    <meta property=\"og:url\" content=\"https://www.nature.com/articles/s41586-023-06473-y\"/>\\n    <meta property=\"og:type\" content=\"article\"/>\\n    <meta property=\"og:site_name\" content=\"Nature\"/>\\n    <meta property=\"og:title\" content=\"Single-cell brain organoid screening identifies developmental defects in autism - Nature\"/>\\n    <meta property=\"og:description\" content=\"We develop a high-throughput CRISPR screening system in cerebral organoids and identify vulnerable&amp;nbsp;cell types and gene regulatory networks associated&amp;nbsp;with&amp;nbsp;autism spectrum disorder&amp;nbsp;from single-cell transcriptomes and chromatin modalities.\"/>\\n    <meta property=\"og:image\" content=\"https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig1_HTML.png\"/>\\n    \\n\\n    <script>\\n        window.eligibleForRa21 = \\'false\\'; \\n    </script>\\n</head>\\n<body class=\"article-page\">\\n\\n<noscript><iframe src=\"https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ\"\\n                  height=\"0\" width=\"0\" style=\"display:none;visibility:hidden\"></iframe></noscript>\\n\\n\\n\\n<div class=\"position-relative cleared z-index-50 background-white\" data-test=\"top-containers\">\\n    <a class=\"c-skip-link\" href=\"#content\">Skip to main content</a>\\n\\n\\n\\n<div class=\"c-grade-c-banner u-hide\">\\n    <div class=\"c-grade-c-banner__container\">\\n        \\n        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain\\n            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in\\n            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles\\n            and JavaScript.</p>\\n\\n    </div>\\n</div>\\n\\n    \\n\\n    <div class=\"u-lazy-ad-wrapper u-mbs-0\">\\n            <div class=\"deferred-placeholder\" data-replace=\"true\"\\n                 data-placeholder=\"/placeholder/v1/institutionalBanner?bpids=[bpids] #institutional-banner-container\"></div>\\n            <aside class=\"c-ad c-ad--728x90\">\\n                <div class=\"c-ad__inner\" data-container-type=\"banner-advert\">\\n                    <p class=\"c-ad__label\">Advertisement</p>\\n                    \\n        \\n            \\n    <div id=\"div-gpt-ad-top-1\"\\n         class=\"div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide\"\\n         data-ad-type=\"top\"\\n         data-test=\"top-ad\"\\n         data-pa11y-ignore\\n         data-gpt\\n         data-gpt-unitpath=\"/285/nature.com/article\"\\n         data-gpt-sizes=\"728x90\"\\n         data-gpt-targeting=\"type=article;pos=top;artid=s41586-023-06473-y;doi=10.1038/s41586-023-06473-y;techmeta=100,107,13,14,31,38,39,44,45,47,51,63;subjmeta=136,2320,2571,378,532,61,631;kwrd=Development+of+the+nervous+system,Stem-cell+biotechnology,Stem+cells\">\\n        <noscript>\\n            <a href=\"//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&amp;sz=728x90&amp;c=-829760560&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41586-023-06473-y%26doi%3D10.1038/s41586-023-06473-y%26techmeta%3D100,107,13,14,31,38,39,44,45,47,51,63%26subjmeta%3D136,2320,2571,378,532,61,631%26kwrd%3DDevelopment+of+the+nervous+system,Stem-cell+biotechnology,Stem+cells\">\\n                <img data-test=\"gpt-advert-fallback-img\"\\n                     src=\"//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&amp;sz=728x90&amp;c=-829760560&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41586-023-06473-y%26doi%3D10.1038/s41586-023-06473-y%26techmeta%3D100,107,13,14,31,38,39,44,45,47,51,63%26subjmeta%3D136,2320,2571,378,532,61,631%26kwrd%3DDevelopment+of+the+nervous+system,Stem-cell+biotechnology,Stem+cells\"\\n                     alt=\"Advertisement\"\\n                     width=\"728\"\\n                     height=\"90\"></a>\\n        </noscript>\\n    </div>\\n\\n        \\n    \\n                </div>\\n            </aside>\\n        </div>\\n    <header class=\"c-header\" id=\"header\" data-header data-track-component=\"nature-150-split-header\" style=\"border-color:#000\">\\n        <div class=\"c-header__row\">\\n            <div class=\"c-header__container\">\\n                <div class=\"c-header__split\">\\n                    \\n                    \\n                    <div class=\"c-header__logo-container\">\\n                        \\n                        <a href=\"/\"\\n                           data-track=\"click\" data-track-action=\"home\" data-track-label=\"image\">\\n                            <picture class=\"c-header__logo\">\\n                                <source srcset=\"https://media.springernature.com/full/nature-cms/uploads/product/nature/header-86f1267ea01eccd46b530284be10585e.svg\" media=\"(min-width: 875px)\">\\n                                <img src=\"https://media.springernature.com/full/nature-cms/uploads/product/nature/header-86f1267ea01eccd46b530284be10585e.svg\" height=\"32\" alt=\"Nature\">\\n                            </picture>\\n                        </a>\\n                    \\n                    </div>\\n                    \\n                    <ul class=\"c-header__menu c-header__menu--global\">\\n                        <li class=\"c-header__item c-header__item--padding c-header__item--hide-md-max\">\\n                            <a class=\"c-header__link\" href=\"https://www.nature.com/siteindex\" data-test=\"siteindex-link\"\\n                               data-track=\"click\" data-track-action=\"open nature research index\" data-track-label=\"link\">\\n                                <span>View all journals</span>\\n                            </a>\\n                        </li>\\n                        <li class=\"c-header__item c-header__item--padding c-header__item--pipe\">\\n                            <a class=\"c-header__link c-header__link--search\"\\n                                href=\"#search-menu\"\\n                                data-header-expander\\n                                data-test=\"search-link\" data-track=\"click\" data-track-action=\"open search tray\" data-track-label=\"button\">\\n                                <svg role=\"img\" aria-hidden=\"true\" focusable=\"false\" height=\"22\" width=\"22\" viewBox=\"0 0 18 18\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z\"/></svg><span>Search</span>\\n                            </a>\\n                        </li>\\n                        <li class=\"c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe\">\\n                            \\n                                <a class=\"c-header__link eds-c-header__link\" id=\"identity-account-widget\" href=\\'https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41586-023-06473-y\\'><span class=\"eds-c-header__widget-fragment-title\">Log in</span></a>\\n                            \\n                        </li>\\n                    </ul>\\n                </div>\\n            </div>\\n        </div>\\n        \\n            <div class=\"c-header__row\">\\n                <div class=\"c-header__container\" data-test=\"navigation-row\">\\n                    <div class=\"c-header__split\">\\n                        <ul class=\"c-header__menu c-header__menu--journal\">\\n                            \\n                                <li class=\"c-header__item c-header__item--dropdown-menu\" data-test=\"explore-content-button\">\\n                                    <a href=\"#explore\"\\n                                       class=\"c-header__link\"\\n                                       data-header-expander\\n                                       data-test=\"menu-button--explore\"\\n                                       data-track=\"click\" data-track-action=\"open explore expander\" data-track-label=\"button\">\\n                                        <span><span class=\"c-header__show-text\">Explore</span> content</span><svg role=\"img\" aria-hidden=\"true\" focusable=\"false\" height=\"16\" viewBox=\"0 0 16 16\" width=\"16\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z\" transform=\"matrix(0 1 -1 0 11 3)\"/></svg>\\n                                    </a>\\n                                </li>\\n                            \\n                            \\n                                <li class=\"c-header__item c-header__item--dropdown-menu\">\\n                                    <a href=\"#about-the-journal\"\\n                                       class=\"c-header__link\"\\n                                       data-header-expander\\n                                       data-test=\"menu-button--about-the-journal\"\\n                                       data-track=\"click\" data-track-action=\"open about the journal expander\" data-track-label=\"button\">\\n                                        <span>About <span class=\"c-header__show-text\">the journal</span></span><svg role=\"img\" aria-hidden=\"true\" focusable=\"false\" height=\"16\" viewBox=\"0 0 16 16\" width=\"16\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z\" transform=\"matrix(0 1 -1 0 11 3)\"/></svg>\\n                                    </a>\\n                                </li>\\n                                \\n                                    <li class=\"c-header__item c-header__item--dropdown-menu\" data-test=\"publish-with-us-button\">\\n                                        <a href=\"#publish-with-us\"\\n                                           class=\"c-header__link c-header__link--dropdown-menu\"\\n                                           data-header-expander\\n                                           data-test=\"menu-button--publish\"\\n                                           data-track=\"click\" data-track-action=\"open publish with us expander\" data-track-label=\"button\">\\n                                            <span>Publish <span class=\"c-header__show-text\">with us</span></span><svg role=\"img\" aria-hidden=\"true\" focusable=\"false\" height=\"16\" viewBox=\"0 0 16 16\" width=\"16\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z\" transform=\"matrix(0 1 -1 0 11 3)\"/></svg>\\n                                        </a>\\n                                    </li>\\n                                \\n                            \\n                            \\n                        </ul>\\n                        <ul class=\"c-header__menu c-header__menu--hide-lg-max\">\\n                            \\n                                <li class=\"c-header__item\">\\n                                    <a class=\"c-header__link\"\\n                                       href=\"https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;1\"\\n                                       rel=\"nofollow\"\\n                                       data-track=\"click\"\\n                                       data-track-action=\"Sign up for alerts\"\\n                                       data-track-label=\"link (desktop site header)\"\\n                                       data-track-external>\\n                                        <span>Sign up for alerts</span><svg role=\"img\" aria-hidden=\"true\" focusable=\"false\" height=\"18\" viewBox=\"0 0 18 18\" width=\"18\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z\" fill=\"#222\"/></svg>\\n                                    </a>\\n                                </li>\\n                            \\n                            \\n                                <li class=\"c-header__item c-header__item--pipe\">\\n                                    <a class=\"c-header__link\"\\n                                       href=\"https://www.nature.com/nature.rss\"\\n                                       data-track=\"click\"\\n                                       data-track-action=\"rss feed\"\\n                                       data-track-label=\"link\">\\n                                            <span>RSS feed</span>\\n                                    </a>\\n                                </li>\\n                            \\n                        </ul>\\n                    </div>\\n                </div>\\n            </div>\\n        \\n    </header>\\n\\n\\n    \\n    \\n        <nav class=\"u-mb-16\" aria-label=\"breadcrumbs\">\\n            <div class=\"u-container\">\\n                <ol class=\"c-breadcrumbs\" itemscope itemtype=\"https://schema.org/BreadcrumbList\">\\n                    <li class=\"c-breadcrumbs__item\" id=\"breadcrumb0\" itemprop=\"itemListElement\" itemscope itemtype=\"https://schema.org/ListItem\"><a class=\"c-breadcrumbs__link\"\\n                               href=\"/\" itemprop=\"item\"\\n                               data-track=\"click\" data-track-action=\"breadcrumb\" data-track-category=\"header\" data-track-label=\"link:nature\"><span itemprop=\"name\">nature</span></a><meta itemprop=\"position\" content=\"1\">\\n                                    <svg class=\"c-breadcrumbs__chevron\" role=\"img\" aria-hidden=\"true\" focusable=\"false\" height=\"10\" viewBox=\"0 0 10 10\" width=\"10\"\\n                                         xmlns=\"http://www.w3.org/2000/svg\">\\n                                        <path d=\"m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z\"\\n                                              fill=\"#666\" fill-rule=\"evenodd\" transform=\"matrix(0 -1 1 0 0 10)\"/>\\n                                    </svg>\\n                                </li><li class=\"c-breadcrumbs__item\" id=\"breadcrumb1\" itemprop=\"itemListElement\" itemscope itemtype=\"https://schema.org/ListItem\"><a class=\"c-breadcrumbs__link\"\\n                               href=\"/nature/articles?type&#x3D;article\" itemprop=\"item\"\\n                               data-track=\"click\" data-track-action=\"breadcrumb\" data-track-category=\"header\" data-track-label=\"link:articles\"><span itemprop=\"name\">articles</span></a><meta itemprop=\"position\" content=\"2\">\\n                                    <svg class=\"c-breadcrumbs__chevron\" role=\"img\" aria-hidden=\"true\" focusable=\"false\" height=\"10\" viewBox=\"0 0 10 10\" width=\"10\"\\n                                         xmlns=\"http://www.w3.org/2000/svg\">\\n                                        <path d=\"m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z\"\\n                                              fill=\"#666\" fill-rule=\"evenodd\" transform=\"matrix(0 -1 1 0 0 10)\"/>\\n                                    </svg>\\n                                </li><li class=\"c-breadcrumbs__item\" id=\"breadcrumb2\" itemprop=\"itemListElement\" itemscope itemtype=\"https://schema.org/ListItem\">\\n                                    <span itemprop=\"name\">article</span><meta itemprop=\"position\" content=\"3\"></li>\\n                </ol>\\n            </div>\\n        </nav>\\n    \\n\\n\\n\\n    \\n\\n</div>\\n\\n\\n<div class=\"u-container u-mt-32 u-mb-32 u-clearfix\" id=\"content\" data-component=\"article-container\"  data-container-type=\"article\">\\n    <main class=\"c-article-main-column u-float-left js-main-column\" data-track-component=\"article body\">\\n        \\n            <div class=\"c-context-bar u-hide\"\\n                 data-test=\"context-bar\"\\n                 data-context-bar\\n                 aria-hidden=\"true\">\\n                <div class=\"c-context-bar__container u-container\">\\n                    <div class=\"c-context-bar__title\">\\n                        Single-cell brain organoid screening identifies developmental defects in autism\\n                    </div>\\n                    \\n    \\n        <div class=\"c-pdf-download u-clear-both js-pdf-download\">\\n            <a href=\"/articles/s41586-023-06473-y.pdf\" class=\"u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link\" data-article-pdf=\"true\" data-readcube-pdf-url=\"true\" data-test=\"download-pdf\" data-draft-ignore=\"true\" data-track=\"click\" data-track-action=\"download pdf\" data-track-label=\"link\" data-track-external download>\\n                <span class=\"c-pdf-download__text\">Download PDF</span>\\n                <svg aria-hidden=\"true\" focusable=\"false\" width=\"16\" height=\"16\" class=\"u-icon\"><use xlink:href=\"#icon-download\"/></svg>\\n            </a>\\n        </div>\\n    \\n\\n                </div>\\n            </div>\\n        \\n        <article lang=\"en\">\\n            \\n                <div class=\"c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile\">\\n                    <div class=\"c-pdf-container\">\\n                        \\n                            \\n    \\n        <div class=\"c-pdf-download u-clear-both js-pdf-download\">\\n            <a href=\"/articles/s41586-023-06473-y.pdf\" class=\"u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link\" data-article-pdf=\"true\" data-readcube-pdf-url=\"true\" data-test=\"download-pdf\" data-draft-ignore=\"true\" data-track=\"click\" data-track-action=\"download pdf\" data-track-label=\"link\" data-track-external download>\\n                <span class=\"c-pdf-download__text\">Download PDF</span>\\n                <svg aria-hidden=\"true\" focusable=\"false\" width=\"16\" height=\"16\" class=\"u-icon\"><use xlink:href=\"#icon-download\"/></svg>\\n            </a>\\n        </div>\\n    \\n\\n                        \\n                    </div>\\n                </div>\\n            \\n            <div class=\"c-article-header\">\\n                <header>\\n                    <ul class=\"c-article-identifiers\" data-test=\"article-identifier\">\\n                        \\n        <li class=\"c-article-identifiers__item\" data-test=\"article-category\">Article</li>\\n    \\n        <li class=\"c-article-identifiers__item\">\\n            <a href=\"https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research\" data-track=\"click\" data-track-action=\"open access\" data-track-label=\"link\" class=\"u-color-open-access\" data-test=\"open-access\">Open access</a>\\n        </li>\\n    \\n    \\n\\n                        <li class=\"c-article-identifiers__item\"><a href=\"#article-info\" data-track=\"click\" data-track-action=\"publication date\" data-track-label=\"link\">Published: <time datetime=\"2023-09-13\">13 September 2023</time></a></li>\\n                    </ul>\\n\\n                    <h1 class=\"c-article-title\" data-test=\"article-title\" data-article-title=\"\">Single-cell brain organoid screening identifies developmental defects in autism</h1>\\n                    <ul class=\"c-article-author-list c-article-author-list--short\" data-test=\"authors-list\" data-component-authors-activator=\"authors-list\"><li class=\"c-article-author-list__item\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Chong-Li-Aff1\" data-author-popup=\"auth-Chong-Li-Aff1\" data-author-search=\"Li, Chong\" data-corresp-id=\"c1\">Chong Li<svg width=\"16\" height=\"16\" focusable=\"false\" role=\"img\" aria-hidden=\"true\" class=\"u-icon\"><use xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"#icon-eds-i-mail-medium\"></use></svg></a><span class=\"u-js-hide\">\\xa0\\n            <a class=\"js-orcid\" href=\"http://orcid.org/0000-0003-2823-7360\"><span class=\"u-visually-hidden\">ORCID: </span>orcid.org/0000-0003-2823-7360</a></span><sup class=\"u-js-hide\"><a href=\"#Aff1\">1</a></sup><sup class=\"u-js-hide\">\\xa0<a href=\"#na1\">na1</a></sup>, </li><li class=\"c-article-author-list__item\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Jonas_Simon-Fleck-Aff2\" data-author-popup=\"auth-Jonas_Simon-Fleck-Aff2\" data-author-search=\"Fleck, Jonas Simon\">Jonas Simon Fleck</a><sup class=\"u-js-hide\"><a href=\"#Aff2\">2</a></sup><sup class=\"u-js-hide\">\\xa0<a href=\"#na1\">na1</a></sup>, </li><li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Catarina-Martins_Costa-Aff1\" data-author-popup=\"auth-Catarina-Martins_Costa-Aff1\" data-author-search=\"Martins-Costa, Catarina\">Catarina Martins-Costa</a><span class=\"u-js-hide\">\\xa0\\n            <a class=\"js-orcid\" href=\"http://orcid.org/0000-0003-2018-898X\"><span class=\"u-visually-hidden\">ORCID: </span>orcid.org/0000-0003-2018-898X</a></span><sup class=\"u-js-hide\"><a href=\"#Aff1\">1</a></sup>, </li><li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Thomas_R_-Burkard-Aff1\" data-author-popup=\"auth-Thomas_R_-Burkard-Aff1\" data-author-search=\"Burkard, Thomas R.\">Thomas R. Burkard</a><span class=\"u-js-hide\">\\xa0\\n            <a class=\"js-orcid\" href=\"http://orcid.org/0000-0002-0884-2929\"><span class=\"u-visually-hidden\">ORCID: </span>orcid.org/0000-0002-0884-2929</a></span><sup class=\"u-js-hide\"><a href=\"#Aff1\">1</a></sup>, </li><li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Jan-Themann-Aff1\" data-author-popup=\"auth-Jan-Themann-Aff1\" data-author-search=\"Themann, Jan\">Jan Themann</a><span class=\"u-js-hide\">\\xa0\\n            <a class=\"js-orcid\" href=\"http://orcid.org/0009-0004-0892-2138\"><span class=\"u-visually-hidden\">ORCID: </span>orcid.org/0009-0004-0892-2138</a></span><sup class=\"u-js-hide\"><a href=\"#Aff1\">1</a></sup>, </li><li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Marlene-Stuempflen-Aff3\" data-author-popup=\"auth-Marlene-Stuempflen-Aff3\" data-author-search=\"Stuempflen, Marlene\">Marlene Stuempflen</a><span class=\"u-js-hide\">\\xa0\\n            <a class=\"js-orcid\" href=\"http://orcid.org/0000-0001-9763-9422\"><span class=\"u-visually-hidden\">ORCID: </span>orcid.org/0000-0001-9763-9422</a></span><sup class=\"u-js-hide\"><a href=\"#Aff3\">3</a></sup>, </li><li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Angela_Maria-Peer-Aff1\" data-author-popup=\"auth-Angela_Maria-Peer-Aff1\" data-author-search=\"Peer, Angela Maria\">Angela Maria Peer</a><sup class=\"u-js-hide\"><a href=\"#Aff1\">1</a></sup>, </li><li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-_bel-Vertesy-Aff1\" data-author-popup=\"auth-_bel-Vertesy-Aff1\" data-author-search=\"Vertesy, Ábel\">Ábel Vertesy</a><span class=\"u-js-hide\">\\xa0\\n            <a class=\"js-orcid\" href=\"http://orcid.org/0000-0001-6075-5702\"><span class=\"u-visually-hidden\">ORCID: </span>orcid.org/0000-0001-6075-5702</a></span><sup class=\"u-js-hide\"><a href=\"#Aff1\">1</a></sup>, </li><li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Jamie_B_-Littleboy-Aff1\" data-author-popup=\"auth-Jamie_B_-Littleboy-Aff1\" data-author-search=\"Littleboy, Jamie B.\">Jamie B. Littleboy</a><span class=\"u-js-hide\">\\xa0\\n            <a class=\"js-orcid\" href=\"http://orcid.org/0000-0002-1847-0794\"><span class=\"u-visually-hidden\">ORCID: </span>orcid.org/0000-0002-1847-0794</a></span><sup class=\"u-js-hide\"><a href=\"#Aff1\">1</a></sup>, </li><li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Christopher-Esk-Aff1-Aff4\" data-author-popup=\"auth-Christopher-Esk-Aff1-Aff4\" data-author-search=\"Esk, Christopher\">Christopher Esk</a><span class=\"u-js-hide\">\\xa0\\n            <a class=\"js-orcid\" href=\"http://orcid.org/0000-0003-1833-0801\"><span class=\"u-visually-hidden\">ORCID: </span>orcid.org/0000-0003-1833-0801</a></span><sup class=\"u-js-hide\"><a href=\"#Aff1\">1</a>,<a href=\"#Aff4\">4</a></sup>, </li><li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Ulrich-Elling-Aff1\" data-author-popup=\"auth-Ulrich-Elling-Aff1\" data-author-search=\"Elling, Ulrich\">Ulrich Elling</a><span class=\"u-js-hide\">\\xa0\\n            <a class=\"js-orcid\" href=\"http://orcid.org/0000-0002-4514-2809\"><span class=\"u-visually-hidden\">ORCID: </span>orcid.org/0000-0002-4514-2809</a></span><sup class=\"u-js-hide\"><a href=\"#Aff1\">1</a></sup>, </li><li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Gregor-Kasprian-Aff3\" data-author-popup=\"auth-Gregor-Kasprian-Aff3\" data-author-search=\"Kasprian, Gregor\">Gregor Kasprian</a><sup class=\"u-js-hide\"><a href=\"#Aff3\">3</a></sup>, </li><li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Nina_S_-Corsini-Aff1\" data-author-popup=\"auth-Nina_S_-Corsini-Aff1\" data-author-search=\"Corsini, Nina S.\">Nina S. Corsini</a><span class=\"u-js-hide\">\\xa0\\n            <a class=\"js-orcid\" href=\"http://orcid.org/0000-0001-6731-3544\"><span class=\"u-visually-hidden\">ORCID: </span>orcid.org/0000-0001-6731-3544</a></span><sup class=\"u-js-hide\"><a href=\"#Aff1\">1</a></sup>, </li><li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Barbara-Treutlein-Aff2\" data-author-popup=\"auth-Barbara-Treutlein-Aff2\" data-author-search=\"Treutlein, Barbara\" data-corresp-id=\"c2\">Barbara Treutlein<svg width=\"16\" height=\"16\" focusable=\"false\" role=\"img\" aria-hidden=\"true\" class=\"u-icon\"><use xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"#icon-eds-i-mail-medium\"></use></svg></a><span class=\"u-js-hide\">\\xa0\\n            <a class=\"js-orcid\" href=\"http://orcid.org/0000-0002-3299-5597\"><span class=\"u-visually-hidden\">ORCID: </span>orcid.org/0000-0002-3299-5597</a></span><sup class=\"u-js-hide\"><a href=\"#Aff2\">2</a></sup> &amp; </li><li class=\"c-article-author-list__show-more\" aria-label=\"Show all 15 authors for this article\" title=\"Show all 15 authors for this article\">…</li><li class=\"c-article-author-list__item\"><a data-test=\"author-name\" data-track=\"click\" data-track-action=\"open author\" data-track-label=\"link\" href=\"#auth-Juergen_A_-Knoblich-Aff1-Aff5\" data-author-popup=\"auth-Juergen_A_-Knoblich-Aff1-Aff5\" data-author-search=\"Knoblich, Juergen A.\" data-corresp-id=\"c3\">Juergen A. Knoblich<svg width=\"16\" height=\"16\" focusable=\"false\" role=\"img\" aria-hidden=\"true\" class=\"u-icon\"><use xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"#icon-eds-i-mail-medium\"></use></svg></a><span class=\"u-js-hide\">\\xa0\\n            <a class=\"js-orcid\" href=\"http://orcid.org/0000-0002-6751-3404\"><span class=\"u-visually-hidden\">ORCID: </span>orcid.org/0000-0002-6751-3404</a></span><sup class=\"u-js-hide\"><a href=\"#Aff1\">1</a>,<a href=\"#Aff5\">5</a></sup>\\xa0</li></ul><button aria-expanded=\"false\" class=\"c-article-author-list__button\"><svg width=\"16\" height=\"16\" focusable=\"false\" role=\"img\" aria-hidden=\"true\" class=\"u-icon\"><use xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"#icon-eds-i-chevron-down-medium\"></use></svg><span>Show authors</span></button>\\n\\n                    \\n\\n                    <p class=\"c-article-info-details\" data-container-section=\"info\">\\n                        \\n    <a data-test=\"journal-link\" href=\"/\" data-track=\"click\" data-track-action=\"journal homepage\" data-track-category=\"article body\" data-track-label=\"link\"><i data-test=\"journal-title\">Nature</i></a>\\n\\n                        <b data-test=\"journal-volume\"><span class=\"u-visually-hidden\">volume</span>\\xa0621</b>,\\xa0<span class=\"u-visually-hidden\">pages </span>373–380 (<span data-test=\"article-publication-year\">2023</span>)<a href=\"#citeas\" class=\"c-article-info-details__cite-as u-hide-print\" data-track=\"click\" data-track-action=\"cite this article\" data-track-label=\"link\">Cite this article</a>\\n                    </p>\\n                    \\n        <div class=\"c-article-metrics-bar__wrapper u-clear-both\">\\n            <ul class=\"c-article-metrics-bar u-list-reset\">\\n                \\n                    <li class=\" c-article-metrics-bar__item\">\\n                        <p class=\"c-article-metrics-bar__count\">51k <span class=\"c-article-metrics-bar__label\">Accesses</span></p>\\n                    </li>\\n                \\n                \\n                    <li class=\"c-article-metrics-bar__item\">\\n                        <p class=\"c-article-metrics-bar__count\">11 <span class=\"c-article-metrics-bar__label\">Citations</span></p>\\n                    </li>\\n                \\n                \\n                    \\n                        <li class=\"c-article-metrics-bar__item\">\\n                            <p class=\"c-article-metrics-bar__count\">479 <span class=\"c-article-metrics-bar__label\">Altmetric</span></p>\\n                        </li>\\n                    \\n                \\n                <li class=\"c-article-metrics-bar__item\">\\n                    <p class=\"c-article-metrics-bar__details\"><a href=\"/articles/s41586-023-06473-y/metrics\" data-track=\"click\" data-track-action=\"view metrics\" data-track-label=\"link\" rel=\"nofollow\">Metrics <span class=\"u-visually-hidden\">details</span></a></p>\\n                </li>\\n            </ul>\\n        </div>\\n    \\n                    \\n                </header>\\n\\n                \\n    <div class=\"u-js-hide\" data-component=\"article-subject-links\">\\n        <h3 class=\"c-article__sub-heading\">Subjects</h3>\\n        <ul class=\"c-article-subject-list\">\\n            <li class=\"c-article-subject-list__subject\"><a href=\"/subjects/development-of-the-nervous-system\" data-track=\"click\" data-track-action=\"view subject\" data-track-label=\"link\">Development of the nervous system</a></li><li class=\"c-article-subject-list__subject\"><a href=\"/subjects/stem-cell-biotechnology\" data-track=\"click\" data-track-action=\"view subject\" data-track-label=\"link\">Stem-cell biotechnology</a></li><li class=\"c-article-subject-list__subject\"><a href=\"/subjects/stem-cells\" data-track=\"click\" data-track-action=\"view subject\" data-track-label=\"link\">Stem cells</a></li>\\n        </ul>\\n    </div>\\n\\n                \\n    \\n    \\n        <div class=\"u-mb-8 c-status-message c-status-message--boxed c-status-message--info\">\\n            \\n                <span class=\"c-status-message__icon\">\\n                    <svg class=\"u-icon\" width=\"18\" height=\"18\" aria-hidden=\"true\" focusable=\"false\">\\n                        <use xlink:href=\"#icon-eds-i-info-filled-medium\"></use>\\n                    </svg>\\n                </span>\\n            \\n            \\n                \\n                    <p class=\"u-mt-0\">An <a href=\"https://doi.org/10.1038/s41586-023-06836-5\" class=\"relation-link\" data-track=\"click\" data-track-action=\"view linked article\" data-track-label=\"link\">Author Correction</a> to this article was published on 14 November 2023</p>\\n                \\n            \\n        </div>\\n    \\n\\n    \\n        <div class=\"u-mb-8 c-status-message c-status-message--boxed c-status-message--info\">\\n            \\n                <span class=\"c-status-message__icon\">\\n                    <svg class=\"u-icon\" width=\"18\" height=\"18\" aria-hidden=\"true\" focusable=\"false\">\\n                        <use xlink:href=\"#icon-eds-i-info-filled-medium\"></use>\\n                    </svg>\\n                </span>\\n            \\n            <p class=\"u-mt-0\">This article has been <a href=\"#change-history\">updated</a></p>\\n        </div>\\n    \\n    \\n\\n                \\n            </div>\\n\\n        <div class=\"c-article-body\">\\n            <section aria-labelledby=\"Abs1\" data-title=\"Abstract\" lang=\"en\"><div class=\"c-article-section\" id=\"Abs1-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Abs1\">Abstract</h2><div class=\"c-article-section__content\" id=\"Abs1-content\"><p>The development of the human brain involves unique processes (not observed in many other species) that can contribute to neurodevelopmental disorders<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" title=\"Hu, W. F., Chahrour, M. H. &amp; Walsh, C. A. The diverse genetic landscape of neurodevelopmental disorders. Genom. Hum. Genet. 15, 195–213 (2014).\" href=\"#ref-CR1\" id=\"ref-link-section-d3800159e661\">1</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" title=\"Klingler, E., Francis, F., Jabaudon, D. &amp; Cappello, S. Mapping the molecular and cellular complexity of cortical malformations. Science 371, eaba4517 (2021).\" href=\"#ref-CR2\" id=\"ref-link-section-d3800159e661_1\">2</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" title=\"Libé-Philippot, B. &amp; Vanderhaeghen, P. Cellular and molecular mechanisms linking human cortical development and evolution. Annu. Rev. Genet. 55, 555–581 (2021).\" href=\"#ref-CR3\" id=\"ref-link-section-d3800159e661_2\">3</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 4\" title=\"Lui, J. H., Hansen, D. V. &amp; Kriegstein, A. R. Development and evolution of the human neocortex. Cell 146, 18–36 (2011).\" href=\"/articles/s41586-023-06473-y#ref-CR4\" id=\"ref-link-section-d3800159e664\">4</a></sup>. Cerebral organoids enable the study of neurodevelopmental disorders in a human context. We have developed the CRISPR–human organoids–single-cell RNA sequencing\\xa0(CHOOSE) system, which uses verified pairs of guide RNAs, inducible CRISPR–Cas9-based genetic disruption and single-cell transcriptomics for pooled loss-of-function screening in mosaic organoids. Here we show that\\xa0perturbation of 36 high-risk autism spectrum disorder genes related to transcriptional regulation uncovers their effects on cell fate determination. We find that dorsal intermediate progenitors, ventral progenitors and upper-layer excitatory neurons are among the most vulnerable cell types. We construct a developmental gene regulatory network of cerebral organoids from single-cell transcriptomes and chromatin modalities and identify autism spectrum disorder-associated and perturbation-enriched regulatory modules. Perturbing members of the BRG1/BRM-associated factor (BAF) chromatin remodelling complex leads to enrichment of ventral telencephalon progenitors. Specifically, mutating the BAF subunit <i>ARID1B</i> affects the fate transition of progenitors to oligodendrocyte and interneuron precursor cells, a phenotype that we confirmed in patient-specific induced pluripotent stem cell-derived organoids. Our study paves the way for high-throughput phenotypic characterization of disease susceptibility genes in organoid models with cell state, molecular pathway and gene regulatory network readouts.</p></div></div></section>\\n\\n            <noscript>\\n                \\n            </noscript>\\n\\n            \\n\\n            \\n                \\n                    \\n                        \\n        \\n            <section aria-labelledby=\"inline-recommendations\" data-title=\"Inline Recommendations\" class=\"c-article-recommendations\" data-track-component=\"inline-recommendations\">\\n                <h3 class=\"c-article-recommendations-title\" id=\"inline-recommendations\">Similar content being viewed by others</h3>\\n                <div class=\"c-article-recommendations-list\">\\n                    \\n                        <div class=\"c-article-recommendations-list__item\">\\n                            <article class=\"c-article-recommendations-card\" itemscope itemtype=\"http://schema.org/ScholarlyArticle\">\\n                                \\n                                    <div class=\"c-article-recommendations-card__img\"><img src=\"https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-022-30968-3/MediaObjects/41467_2022_30968_Fig1_HTML.png\" loading=\"lazy\" alt=\"\"></div>\\n                                \\n                                <div class=\"c-article-recommendations-card__main\">\\n                                    <h3 class=\"c-article-recommendations-card__heading\" itemprop=\"name headline\">\\n                                        <a class=\"c-article-recommendations-card__link\" itemprop=\"url\" href=\"https://www.nature.com/articles/s41467-022-30968-3\" data-track=\"click\" data-track-action=\"click recommendations inline - 1\" data-track-label=\"10.1038/s41467-022-30968-3\">Orgo-Seq integrates single-cell and bulk transcriptomic data to identify cell type specific-driver genes associated with autism spectrum disorder</a>\\n                                    </h3>\\n                                    <div class=\"c-article-meta-recommendations\">\\n                                        <span class=\"c-article-meta-recommendations__item-type\">Article</span>\\n                                         <span class=\"c-article-meta-recommendations__access-type\">Open access</span>\\n                                         <span class=\"c-article-meta-recommendations__date\">10 June 2022</span>\\n                                    </div>\\n                                </div>\\n                                <p class=\"c-article-recommendations-card__authors u-sans-serif\">Elaine T. Lim, Yingleong Chan, … George M. Church</p>\\n                            </article>\\n                        </div>\\n                    \\n                        <div class=\"c-article-recommendations-list__item\">\\n                            <article class=\"c-article-recommendations-card\" itemscope itemtype=\"http://schema.org/ScholarlyArticle\">\\n                                \\n                                    <div class=\"c-article-recommendations-card__img\"><img src=\"https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-022-05279-8/MediaObjects/41586_2022_5279_Fig1_HTML.png\" loading=\"lazy\" alt=\"\"></div>\\n                                \\n                                <div class=\"c-article-recommendations-card__main\">\\n                                    <h3 class=\"c-article-recommendations-card__heading\" itemprop=\"name headline\">\\n                                        <a class=\"c-article-recommendations-card__link\" itemprop=\"url\" href=\"https://www.nature.com/articles/s41586-022-05279-8\" data-track=\"click\" data-track-action=\"click recommendations inline - 2\" data-track-label=\"10.1038/s41586-022-05279-8\">Inferring and perturbing cell fate regulomes in human brain organoids</a>\\n                                    </h3>\\n                                    <div class=\"c-article-meta-recommendations\">\\n                                        <span class=\"c-article-meta-recommendations__item-type\">Article</span>\\n                                         <span class=\"c-article-meta-recommendations__access-type\">Open access</span>\\n                                         <span class=\"c-article-meta-recommendations__date\">05 October 2022</span>\\n                                    </div>\\n                                </div>\\n                                <p class=\"c-article-recommendations-card__authors u-sans-serif\">Jonas Simon Fleck, Sophie Martina Johanna Jansen, … Barbara Treutlein</p>\\n                            </article>\\n                        </div>\\n                    \\n                        <div class=\"c-article-recommendations-list__item\">\\n                            <article class=\"c-article-recommendations-card\" itemscope itemtype=\"http://schema.org/ScholarlyArticle\">\\n                                \\n                                    <div class=\"c-article-recommendations-card__img\"><img src=\"https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-021-04358-6/MediaObjects/41586_2021_4358_Fig1_HTML.png\" loading=\"lazy\" alt=\"\"></div>\\n                                \\n                                <div class=\"c-article-recommendations-card__main\">\\n                                    <h3 class=\"c-article-recommendations-card__heading\" itemprop=\"name headline\">\\n                                        <a class=\"c-article-recommendations-card__link\" itemprop=\"url\" href=\"https://www.nature.com/articles/s41586-021-04358-6\" data-track=\"click\" data-track-action=\"click recommendations inline - 3\" data-track-label=\"10.1038/s41586-021-04358-6\">Autism genes converge on asynchronous development of shared neuron classes</a>\\n                                    </h3>\\n                                    <div class=\"c-article-meta-recommendations\">\\n                                        <span class=\"c-article-meta-recommendations__item-type\">Article</span>\\n                                        \\n                                         <span class=\"c-article-meta-recommendations__date\">02 February 2022</span>\\n                                    </div>\\n                                </div>\\n                                <p class=\"c-article-recommendations-card__authors u-sans-serif\">Bruna Paulsen, Silvia Velasco, … Paola Arlotta</p>\\n                            </article>\\n                        </div>\\n                    \\n                </div>\\n            </section>\\n        \\n            <script>\\n                window.dataLayer = window.dataLayer || [];\\n                window.dataLayer.push({\\n                    recommendations: {\\n                        recommender: \\'semantic\\',\\n                        model: \\'specter\\',\\n                        policy_id: \\'speedy-BootstrappedUCB\\',\\n                        timestamp: 1708952284,\\n                        embedded_user: \\'null\\'\\n                    }\\n                });\\n            </script>\\n        \\n    \\n                    \\n                \\n                    \\n                <div class=\"main-content\">\\n                    <section data-title=\"Main\"><div class=\"c-article-section\" id=\"Sec1-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec1\">Main</h2><div class=\"c-article-section__content\" id=\"Sec1-content\"><p>Human cortical development involves unique and intricate processes. Following neural tube formation, neuroepithelial cells within the telencephalon proliferate, expand and generate radial glial progenitors, intermediate progenitors and outer radial glial progenitors. In the dorsal region, these progenitors give rise to layered excitatory neurons. In the ventral telencephalon, they generate interneurons that migrate into the dorsal cortex to integrate with excitatory neurons. These processes are governed by precise and highly orchestrated genetic and molecular programs, many of which have remained elusive<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 3\" title=\"Libé-Philippot, B. &amp; Vanderhaeghen, P. Cellular and molecular mechanisms linking human cortical development and evolution. Annu. Rev. Genet. 55, 555–581 (2021).\" href=\"/articles/s41586-023-06473-y#ref-CR3\" id=\"ref-link-section-d3800159e706\">3</a></sup>. Research into neurodevelopmental disorders (NDDs) has advanced our understanding of human brain development and helped to\\xa0reveal how it can go awry. However, many NDDs, such as autism spectrum disorder (ASD), are diagnosed only after birth, when brain development is almost complete. Analysing the developmental and cell type-specific defects associated with ASD in a human context is crucial but is often constrained to neuroimaging and postmortem tissue studies. Moreover, coexpression network analyses of ASD genes indicate that the\\xa0developmental defects associated with ASD may arise during fetal stages<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 5\" title=\"Willsey, A. J. et al. Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism. Cell 155, 997–1007 (2013).\" href=\"/articles/s41586-023-06473-y#ref-CR5\" id=\"ref-link-section-d3800159e710\">5</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 6\" title=\"Parikshak, N. N. et al. Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism. Cell 155, 1008–1021 (2013).\" href=\"/articles/s41586-023-06473-y#ref-CR6\" id=\"ref-link-section-d3800159e713\">6</a></sup>, periods that are difficult to investigate.</p><p>Studying the genetic aetiology of NDDs enhances our understanding of disease mechanisms<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 1\" title=\"Hu, W. F., Chahrour, M. H. &amp; Walsh, C. A. The diverse genetic landscape of neurodevelopmental disorders. Genom. Hum. Genet. 15, 195–213 (2014).\" href=\"/articles/s41586-023-06473-y#ref-CR1\" id=\"ref-link-section-d3800159e720\">1</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 7\" title=\"Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR7\" id=\"ref-link-section-d3800159e723\">7</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 8\" title=\"Torre-Ubieta, L., de la Won, H., Stein, J. L. &amp; Geschwind, D. H. Advancing the understanding of autism disease mechanisms through genetics. Nat. Med. 22, 345–361 (2016).\" href=\"/articles/s41586-023-06473-y#ref-CR8\" id=\"ref-link-section-d3800159e726\">8</a></sup>, but it usually requires access to the developmental processes of the human brain. Brain organoids recapitulate early brain development and generate diverse cell types found in vivo<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 9\" title=\"Lancaster, M. A. et al. Cerebral organoids model human brain development and microcephaly. Nature 501, 373–379 (2013).\" href=\"/articles/s41586-023-06473-y#ref-CR9\" id=\"ref-link-section-d3800159e730\">9</a></sup>. Although organoids have been used to investigate disease-associated genes<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" title=\"Lancaster, M. A. et al. Cerebral organoids model human brain development and microcephaly. Nature 501, 373–379 (2013).\" href=\"#ref-CR9\" id=\"ref-link-section-d3800159e734\">9</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" title=\"Mariani, J. et al. FOXG1-dependent dysregulation of GABA/glutamate neuron differentiation in autism spectrum disorders. Cell 162, 375–390 (2015).\" href=\"#ref-CR10\" id=\"ref-link-section-d3800159e734_1\">10</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 11\" title=\"Paulsen, B. et al. Autism genes converge on asynchronous development of shared neuron classes. Nature 602, 268–273 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR11\" id=\"ref-link-section-d3800159e737\">11</a></sup>, they are limited by phenotypic variability and low throughput. Recent studies combining CRISPR screening technology with organoids have\\xa0revealed the power of such strategies for discovering new gene functions<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 12\" title=\"Esk, C. et al. A human tissue screen identifies a regulator of ER secretion as a brain-size determinant. Science 370, 935–941 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR12\" id=\"ref-link-section-d3800159e741\">12</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 13\" title=\"Michels, B. E. et al. Pooled in vitro and in vivo CRISPR-Cas9 screening identifies tumor suppressors in human colon organoids. Cell Stem Cell 26, 782–792.e7 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR13\" id=\"ref-link-section-d3800159e744\">13</a></sup>. However, such screens are limited by low-content readouts, which often use guide RNA (gRNA) counts to assess proliferation phenotypes when challenged with genetic perturbations. Although CRISPR screening coupled with single-cell transcriptomic readout provides unprecedented resolution for phenotypic characterization<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" title=\"Dixit, A. et al. Perturb-seq: dissecting molecular circuits with scalable single-cell RNA profiling of pooled genetic screens. Cell 167, 1853–1866.e17 (2016).\" href=\"#ref-CR14\" id=\"ref-link-section-d3800159e748\">14</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" title=\"Jaitin, D. A. et al. Dissecting immune circuits by linking CRISPR-pooled screens with single-cell RNA-seq. Cell 167, 1883–1896.e15 (2016).\" href=\"#ref-CR15\" id=\"ref-link-section-d3800159e748_1\">15</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 16\" title=\"Adamson, B. et al. A multiplexed single-cell CRISPR screening platform enables systematic dissection of the unfolded protein response. Cell 167, 1867–1882.e21 (2016).\" href=\"/articles/s41586-023-06473-y#ref-CR16\" id=\"ref-link-section-d3800159e751\">16</a></sup>, such approaches have not been fully explored in organoids. The feasibility of single-cell perturbation screening in heterogeneous tissues that undergo long-term differentiation and consist of diverse cell types remains unclear.</p><p>Here, we describe the CHOOSE system which combines parallel genetic perturbations with single-cell transcriptomic readout in mosaic cerebral organoids. We deliver barcoded pairs of gRNAs as a pooled lentiviral library to stem cells and generate telencephalic organoids to identify the\\xa0loss-of-function phenotypes of 36 high-risk ASD genes at the level of cell types and molecular pathways. Using single-cell multiomic data, we construct a developmental gene regulatory network (GRN) of cerebral organoids and identify ASD-enriched regulatory hubs connected to the genes\\xa0that are dysregulated in response to genetic perturbations. Among the 36 genes, one of the most significant changes in cell type composition was identified in the context of <i>ARID1B</i>. Specifically, perturbing <i>ARID1B</i> expands ventral radial glia cells and increases their transition to early oligodendrocyte precursor cells (OPCs), a phenotype we verify in brain organoids generated from <i>ARID1B</i> patient-derived induced pluripotent stem cell (iPS\\xa0cell) lines.</p></div></div></section><section data-title=\"Single-cell CRISPR screening in barcoded organoids\"><div class=\"c-article-section\" id=\"Sec2-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec2\">Single-cell CRISPR screening in barcoded organoids</h2><div class=\"c-article-section__content\" id=\"Sec2-content\"><p>Single-cell RNA sequencing (scRNA-seq) is a high-throughput method used to analyse cellular heterogeneity in complex tissues. To establish an organoid system that enables CRISPR perturbations with single-cell transcriptomic readout, we used a human embryonic stem cell (hES\\xa0cell) line expressing an enhanced specificity SpCas9 (eCas9), which has substantially reduced off-target effects and is controlled by an upstream loxp stop element<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 12\" title=\"Esk, C. et al. A human tissue screen identifies a regulator of ER secretion as a brain-size determinant. Science 370, 935–941 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR12\" id=\"ref-link-section-d3800159e775\">12</a></sup> (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig1\">1a</a>). To regulate eCas9 induction, we engineered a lentiviral vector to deliver 4-hydroxytamoxifen-inducible CRE recombinase and a dual single-guide RNA (sgRNA) cassette (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig1\">1a</a>). The dual sgRNA consists of two sgRNAs targeting the same gene, expressed under the <i>U6</i> or <i>H1</i> promoter. The dual gRNA is located within the 3′ long terminal repeat (LTR) and is thus transcribed by RNA polymerase II to be captured by scRNA-seq assays<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 17\" title=\"Datlinger, P. et al. Pooled CRISPR screening with single-cell transcriptome read-out. Nat. Methods 14, 297–301 (2017).\" href=\"/articles/s41586-023-06473-y#ref-CR17\" id=\"ref-link-section-d3800159e792\">17</a></sup>. To ensure efficient generation of loss-of-function alleles, we determined the editing efficiency of each sgRNA pair using a flow cytometry-based gRNA reporter assay (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig1\">1b</a> and Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig5\">1a</a>). In this assay, a pre-assembled array of gRNA-targeting sequences fused with TagBFP is used to generate 3T3 fibroblast reporter cell lines. sgRNA and eCas9 are then delivered into the reporter cell lines by lentiviruses. Successful genome editing causes frameshift mutations, resulting in the loss of\\xa0blue fluorescent protein (BFP) fluorescence, which enables the quantitative evaluation of gRNA efficiency (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig1\">1c</a> and Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig5\">1b,c</a>), although it does not allow for the determination of whether a heterozygous or homozygous mutation was introduced. Using our reporter assay, we selected efficient sgRNA pairs for 36 ASD genes (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig5\">1d,e</a> and Supplementary Table <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"supplementary material anchor\" href=\"/articles/s41586-023-06473-y#MOESM1\">1</a>). Immunohistochemistry analysis of several perturbations in organoids further confirmed the loss of protein products for the majority of selected gRNAs (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig6\">2</a>).</p><div class=\"c-article-section__figure js-c-reading-companion-figures-item\" data-test=\"figure\" data-container-section=\"figure\" id=\"figure-1\" data-title=\"The CHOOSE system for multiplexed screening of ASD risk genes in human cerebral organoids.\"><figure><figcaption><b id=\"Fig1\" class=\"c-article-section__figure-caption\" data-test=\"figure-caption-text\">Fig. 1: The CHOOSE system for multiplexed screening of ASD risk genes in human cerebral organoids.</b></figcaption><div class=\"c-article-section__figure-content\"><div class=\"c-article-section__figure-item\"><a class=\"c-article-section__figure-link\" data-test=\"img-link\" data-track=\"click\" data-track-label=\"image\" data-track-action=\"view figure\" href=\"/articles/s41586-023-06473-y/figures/1\" rel=\"nofollow\"><picture><source type=\"image/webp\" srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig1_HTML.png?as=webp\"><img aria-describedby=\"Fig1\" src=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig1_HTML.png\" alt=\"figure 1\" loading=\"lazy\" width=\"685\" height=\"583\"></picture></a></div><div class=\"c-article-section__figure-description\" data-test=\"bottom-caption\" id=\"figure-1-desc\"><p><b>a</b>, CHOOSE system overview. Barcoded dual sgRNA cassette located within the 3′ LTR of the lentivirus. <b>b</b>, Reporter assay to test gRNA efficiencies for 36 ASD risk genes. <b>c</b>, Editing efficiencies of gRNAs determined by flow cytometry. Plots show examples of gRNAs with no or efficient editing. <b>d</b>, sgRNA sequence read distributions of gRNAs sequenced from the ASD plasmid library, lentivirus-infected hES\\xa0cells and embryoid bodies at day 5. <b>e</b>, Numbers of clones from the starting hES\\xa0cells for each perturbation used to generate mosaic cerebral organoids. Control, non-targeting gRNAs. <b>f</b>, UMAP embedding of the scRNA-seq dataset containing dorsal and ventral telencephalon trajectories. Subclustering and UMAP embedding of the ventral telencephalon trajectory excluding astrocytes and ccv-RGCs\\xa0to annotate the OPC cluster. <b>g</b>, Heat map showing the expression of marker genes in different cell types. ccRGC, cycling radial glial cell; CThPN, cortical thalamic projection neuron; oRGC, outer radial glial cell; IDM, improved differentiation medium; NI, neural induction.</p></div></div><div class=\"u-text-right u-hide-print\"><a class=\"c-article__pill-button\" data-test=\"article-link\" data-track=\"click\" data-track-label=\"button\" data-track-action=\"view figure\" href=\"/articles/s41586-023-06473-y/figures/1\" data-track-dest=\"link:Figure1 Full size image\" aria-label=\"Full size image figure 1\" rel=\"nofollow\"><span>Full size image</span><svg width=\"16\" height=\"16\" focusable=\"false\" role=\"img\" aria-hidden=\"true\" class=\"u-icon\"><use xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"#icon-eds-i-chevron-right-small\"></use></svg></a></div></figure></div><p>We individually cloned sgRNA pairs and pooled them equally\\xa0to construct a lentiviral plasmid library (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig7\">3a,b</a>). To ensure that lentiviral integration frequency was limited to one per cell, weused a low infection rate of 2.5% (ref. <sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 18\" title=\"Doench, J. G. Am I ready for CRISPR? A user’s guide to genetic screens. Nat. Rev. Genet. 19, 67–80 (2018).\" href=\"/articles/s41586-023-06473-y#ref-CR18\" id=\"ref-link-section-d3800159e865\">18</a></sup>) (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig7\">3c–e</a>). Our analysis indicated homogeneous distribution of the gRNAs in both the plasmid library and hES\\xa0cells, which was maintained after the formation of embryoid bodies (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig1\">1d</a>). It is important to note that the development of human brain tissues exhibits the generation of clones with highly variable sizes both in vivo and in vitro<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 12\" title=\"Esk, C. et al. A human tissue screen identifies a regulator of ER secretion as a brain-size determinant. Science 370, 935–941 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR12\" id=\"ref-link-section-d3800159e875\">12</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 19\" title=\"Bizzotto, S. et al. Landmarks of human embryonic development inscribed in somatic mutations. Science 371, 1249–1253 (2021).\" href=\"/articles/s41586-023-06473-y#ref-CR19\" id=\"ref-link-section-d3800159e878\">19</a></sup>. To monitor the clonal complexity of the founder cells, we introduced a unique clone barcode (UCB; 1.4\\u2009×\\u200910<sup>7</sup> combinations) for each dual sgRNA cassette to label individual lentiviral integration events (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig1\">1a</a> and Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig8\">4a,b</a>). Using this strategy, we obtained an average of 2,770 unique clones for each perturbation, which we used to generate mosaic embryoid bodies (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig1\">1e</a>). Altogether, we established a highly efficient and controlled pooled screening system with high clonal complexity in the organoid.</p></div></div></section><section data-title=\"CHOOSE organoids generate diverse cell types\"><div class=\"c-article-section\" id=\"Sec3-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec3\">CHOOSE organoids generate diverse cell types</h2><div class=\"c-article-section__content\" id=\"Sec3-content\"><p>Cortical abnormalities are a prominent feature of ASD<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 8\" title=\"Torre-Ubieta, L., de la Won, H., Stein, J. L. &amp; Geschwind, D. H. Advancing the understanding of autism disease mechanisms through genetics. Nat. Med. 22, 345–361 (2016).\" href=\"/articles/s41586-023-06473-y#ref-CR8\" id=\"ref-link-section-d3800159e902\">8</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 20\" title=\"Chen, J. A., Peñagarikano, O., Belgard, T. G., Swarup, V. &amp; Geschwind, D. H. The emerging picture of autism spectrum disorder: genetics and pathology. Pathology Mech. Dis. 10, 111–144 (2015).\" href=\"/articles/s41586-023-06473-y#ref-CR20\" id=\"ref-link-section-d3800159e905\">20</a></sup>. Many ASD risk genes associated with transcriptional regulation and chromatin remodelling are crucial for cortical development<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 21\" title=\"Study, T. D. et al. Synaptic, transcriptional, and chromatin genes disrupted in autism. Nature 515, 209–215 (2014).\" href=\"/articles/s41586-023-06473-y#ref-CR21\" id=\"ref-link-section-d3800159e909\">21</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 22\" title=\"Trevino, A. E. et al. Chromatin and gene-regulatory dynamics of the developing human cerebral cortex at single-cell resolution. Cell 184, 5053–5069.e23 (2021).\" href=\"/articles/s41586-023-06473-y#ref-CR22\" id=\"ref-link-section-d3800159e912\">22</a></sup>. Therefore, we aimed to leverage our methodology to explore loss-of-function phenotypes for 36 transcriptional control genes with high ASD causal confidence (Simons Foundation Autism Research Initiative (SFARI) gene score 1)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 7\" title=\"Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR7\" id=\"ref-link-section-d3800159e916\">7</a></sup>.</p><p>We used previously established protocols that reproducibly generate human telencephalon organoids<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 23\" title=\"Lancaster, M. A. et al. Guided self-organization and cortical plate formation in human brain organoids. Nat. Biotechnol. 35, 659–666 (2017).\" href=\"/articles/s41586-023-06473-y#ref-CR23\" id=\"ref-link-section-d3800159e923\">23</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 24\" title=\"Eichmüller, O. L. et al. Amplification of human interneuron progenitors promotes brain tumors and neurological defects. Science 375, eabf5546 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR24\" id=\"ref-link-section-d3800159e926\">24</a></sup> (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig9\">5a–c</a>). eCas9 was induced in 5-day-old embryoid bodies, followed by neural induction. Fluorescence-activated cell sorting (FACS)-based analyses suggest that mutant cells (GFP<sup>+</sup>/dTomato<sup>+</sup>) remained at low percentages throughout development, with an average of 21.8% on day 120 (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig9\">5d–f</a>). It is likely that this could limit the mutant cell–cell interactions within the mosaic tissues. Single-cell transcriptome profiling of cerebral organoids at 4 months revealed a large diversity of dorsal and ventral telencephalon cell populations (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig1\">1f,g</a> and Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig10\">6a–j</a>) (14 independent pools of organoids, 3–7 organoids each pool, 65 organoids in total, three independent batches). We first annotated cell clusters based on control cells (non-targeting gRNA control) and eCas9-uninduced cells (35,203 cells) (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig10\">6d–f</a>). Cell-type labels for the full CHOOSE dataset (49,754 cells) were then derived through a label transfer. Broadly, we found that perturbed cells adopted cell states resembling those found in the unperturbed controls (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig1\">1f</a>). We identified progenitor cells with dorsal (<i>PAX6</i>) or ventral (<i>ASCL1</i>, <i>OLIG2</i>) origins. Among the cells with dorsal identity, we identified radial glial cells (RGCs; <i>VIM</i>), cycling RGCs (<i>ASPM</i>), outer RGCs (oRGCs;\\xa0<i>HOPX</i>) and intermediate progenitor cells (IPCs; <i>EOMES</i>). These progenitors differentiated into excitatory neurons with specific layer identities, including layer 5/6 neurons (L5/6; <i>BCL11B</i>), L6 cortical thalamic projection neurons (<i>FOXP2</i>, <i>TLE4</i>)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 25\" title=\"Tsyporin, J. et al. Transcriptional repression by FEZF2 restricts alternative identities of cortical projection neurons. Cell Rep. 35, 109269–109269 (2021).\" href=\"/articles/s41586-023-06473-y#ref-CR25\" id=\"ref-link-section-d3800159e985\">25</a></sup>, L4 neurons (<i>RORB</i>, <i>UNC5D</i>, <i>NR2F1</i>)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 26\" title=\"Zhong, Y. et al. Identification of the genes that are expressed in the upper layers of the neocortex. Cereb. Cortex 14, 1144–1152 (2004).\" href=\"/articles/s41586-023-06473-y#ref-CR26\" id=\"ref-link-section-d3800159e999\">26</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 27\" title=\"Oishi, K. et al. Identity of neocortical layer 4 neurons is specified through correct positioning into the cortex. eLife 5, e10907 (2016).\" href=\"/articles/s41586-023-06473-y#ref-CR27\" id=\"ref-link-section-d3800159e1002\">27</a></sup> and L2/3 neurons (<i>SATB2</i>). Ventral radial glial cells (v-RGCs; cell cycling\\xa0ventral\\xa0RGCs,\\xa0ccv-RGCs) differentiated into interneuron precursor cells (INPs; <i>DLX2</i>), which generated interneurons with lateral ganglionic eminence (LGE) origin (LGE-IN; <i>MEIS2</i>) or caudal ganglionic eminence (CGE) origin (CGE-IN; <i>NR2F2</i>)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 28\" title=\"Shi, Y. et al. Mouse and human share conserved transcriptional programs for interneuron development. Science 374, eabj6641 (2021).\" href=\"/articles/s41586-023-06473-y#ref-CR28\" id=\"ref-link-section-d3800159e1018\">28</a></sup>. Notably, we found a cluster of interneurons expressing <i>MEIS2</i> and high levels of <i>PAX6</i> (LGE PAX6<sup>+</sup>\\xa0interneurons), a signature that resembles mouse olfactory bulb precursors that were recently reported to generate neurons redirected to white matter in primates<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 29\" title=\"Schmitz, M. T. et al. The development and evolution of inhibitory neurons in primate cerebrum. Nature 603, 871–877 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR29\" id=\"ref-link-section-d3800159e1031\">29</a></sup>. In addition to neuronal populations, we identified glial cell populations including astrocytes (<i>S100B</i>, <i>APOE</i>, <i>ALDH1L1</i>) and OPCs (<i>OLIG2</i>, <i>PDGFRA</i>) (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig1\">1f,g</a>). RNA velocity analysis<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 30\" title=\"Manno, G. L. et al. RNA velocity of single cells. Nature 560, 494–498 (2018).\" href=\"/articles/s41586-023-06473-y#ref-CR30\" id=\"ref-link-section-d3800159e1054\">30</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 31\" title=\"Bergen, V., Lange, M., Peidli, S., Wolf, F. A. &amp; Theis, F. J. Generalizing RNA velocity to transient cell states through dynamical modeling. Nat. Biotechnol. 38, 1408–1414 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR31\" id=\"ref-link-section-d3800159e1057\">31</a></sup> revealed developmental trajectories from neural progenitor cells to neuronal populations in both the dorsal and ventral telencephalons (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig10\">6k</a>). Further analysis of our organoid dataset and a primary developing human brain dataset<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 32\" title=\"Braun, E. et al. Comprehensive cell atlas of the first-trimester developing human brain. Preprint at bioRxiv &#xA;                  https://doi.org/10.1101/2022.10.24.513487&#xA;                  &#xA;                 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR32\" id=\"ref-link-section-d3800159e1065\">32</a></sup> revealed cell type-specific expression patterns for several ASD genes that we targeted in our screen (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig10\">6l,m</a>). In summary, our scRNA-seq dataset of 4-month-old cerebral organoids recapitulates diverse telencephalic cell types that are present in the developing human brain.</p></div></div></section><section data-title=\"Cell proliferation and depletion phenotypes\"><div class=\"c-article-section\" id=\"Sec4-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec4\">Cell proliferation and depletion phenotypes</h2><div class=\"c-article-section__content\" id=\"Sec4-content\"><p>Aberrant cell proliferation during brain development has been suggested to contribute to ASD pathology<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 33\" title=\"Courchesne, E. et al. The ASD Living Biology: from cell proliferation to clinical phenotype. Mol. Psychiatr. 24, 88–107 (2019).\" href=\"/articles/s41586-023-06473-y#ref-CR33\" id=\"ref-link-section-d3800159e1080\">33</a></sup>. To test whether ASD genetic perturbations could affect cell proliferation, we recovered gRNA information from scRNA-seq complementary\\xa0DNA libraries as well as bulk-extracted genomic DNA from organoid pools of four different batches (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig11\">7a</a>). We observed a heterogenous gRNA representation in eCas9-induced cells at 4 months. A time course analysis revealed a deviation from the initial gRNA distribution as early as day 20 (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig11\">7b</a>). When comparing eCas9-induced with uninduced cells from scRNA-seq libraries, we found significantly enriched and depleted gRNAs (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig2\">2a,b</a>) (induced cells, <i>n</i>\\u2009=\\u200914 independent pools of organoids, three batches; uninduced cells, <i>n</i>\\u2009=\\u20098 independent pools of organoids, two batches). Using a FACS-based approach, we further confirmed the enrichment and depletion phenotypes in individually perturbed organoids for the four genes (<i>KMT2C</i>, <i>LEO1</i>, <i>ADNP</i> and <i>WAC</i>) that showed the largest effect sizes (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig12\">8</a>).</p><div class=\"c-article-section__figure js-c-reading-companion-figures-item\" data-test=\"figure\" data-container-section=\"figure\" id=\"figure-2\" data-title=\"Cell type-specific effects of ASD risk gene perturbations.\"><figure><figcaption><b id=\"Fig2\" class=\"c-article-section__figure-caption\" data-test=\"figure-caption-text\">Fig. 2: Cell type-specific effects of ASD risk gene perturbations.</b></figcaption><div class=\"c-article-section__figure-content\"><div class=\"c-article-section__figure-item\"><a class=\"c-article-section__figure-link\" data-test=\"img-link\" data-track=\"click\" data-track-label=\"image\" data-track-action=\"view figure\" href=\"/articles/s41586-023-06473-y/figures/2\" rel=\"nofollow\"><picture><source type=\"image/webp\" srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig2_HTML.png?as=webp\"><img aria-describedby=\"Fig2\" src=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig2_HTML.png\" alt=\"figure 2\" loading=\"lazy\" width=\"685\" height=\"420\"></picture></a></div><div class=\"c-article-section__figure-description\" data-test=\"bottom-caption\" id=\"figure-2-desc\"><p><b>a</b>, Box plot showing the representation of gRNAs in eCas9-induced and uninduced organoid pools (induced cells, <i>n</i>\\u2009=\\u200914 independent pools of organoids, three batches; uninduced cells, <i>n</i>\\u2009=\\u20098 independent pools of organoids, two batches). The centre lines represent the medians, the box limits show the 25–75% interquartile ranges and the whiskers indicate 1.5× the interquartile ranges. <b>b</b>, Bar plot showing fold change of gRNA representation in eCas9-induced versus uninduced cell pools. <b>c</b>, Heat map showing enrichment of gRNAs versus control in dorsal versus ventral telencephalon (telen.) cells (left panel, orange to purple) and individual cell types (centre and right panels, red to blue). Colours indicate the log odds ratio, and sizes indicate −log<sub>10</sub> FDR-corrected <i>P</i> values of a two-sided CMH test stratified by organoid pool.</p></div></div><div class=\"u-text-right u-hide-print\"><a class=\"c-article__pill-button\" data-test=\"article-link\" data-track=\"click\" data-track-label=\"button\" data-track-action=\"view figure\" href=\"/articles/s41586-023-06473-y/figures/2\" data-track-dest=\"link:Figure2 Full size image\" aria-label=\"Full size image figure 2\" rel=\"nofollow\"><span>Full size image</span><svg width=\"16\" height=\"16\" focusable=\"false\" role=\"img\" aria-hidden=\"true\" class=\"u-icon\"><use xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"#icon-eds-i-chevron-right-small\"></use></svg></a></div></figure></div><p>To ensure that the observed cell proliferation phenotypes were not driven by clonal effects, we determined the complexity of clones recovered from scRNA-seq libraries. On average, we recovered 125 clones for each perturbation and found that the clones were distributed across all libraries (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig8\">4c,d</a>). Analysing the size of each clone, we found a mean average cell number per clone of 4.4 (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig8\">4e,f</a>). These data suggest that cells captured in the CHOOSE screen came from diverse and relatively small clones, which are crucial for mitigating dominant clonal effects. Bulk analysis of the genomic DNA with a higher cell number input (50,000–150,000) revealed high clonal complexity in both eCas9-uninduced and eCas9-induced cells, with a homogenous distribution only in uninduced cells (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig8\">4g,h</a>). In conclusion, using a pooled, high-complexity barcoding screening system, we successfully identified ASD risk genes that play essential roles in cell proliferation and survival.</p></div></div></section><section data-title=\"Cell-type-specific effects of perturbing ASD genes\"><div class=\"c-article-section\" id=\"Sec5-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec5\">Cell-type-specific effects of perturbing ASD genes</h2><div class=\"c-article-section__content\" id=\"Sec5-content\"><p>Cell type-specific alterations have been observed in patients with\\xa0ASD and brain organoid disease models<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 11\" title=\"Paulsen, B. et al. Autism genes converge on asynchronous development of shared neuron classes. Nature 602, 268–273 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR11\" id=\"ref-link-section-d3800159e1176\">11</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 34\" title=\"Velmeshev, D. et al. Single-cell genomics identifies cell type–specific molecular changes in autism. Science 364, 685–689 (2019).\" href=\"/articles/s41586-023-06473-y#ref-CR34\" id=\"ref-link-section-d3800159e1179\">34</a></sup>. The large cellular diversity detected in our system enables us to systematically assess, compare and categorize the effects of ASD gene perturbations on cell states. Using a Cochran–Mantel–Haenszel (CMH) test stratified by library replicates, we first assessed the differential abundance of dorsal versus ventral telencephalic cells, as well as the abundance of each individual cell type in each perturbation versus a non-targeting gRNA control (11 independent pools of organoids, two independent batches) (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig2\">2c</a> and Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig13\">9</a>).</p><p>For 24 perturbations, we detected significant changes in the ratio of dorsal to ventral cells (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig2\">2c</a>). Notably, most perturbations (21 of 24) lowered the dorsal-to-ventral ratio. Perturbation of <i>KMT2C</i>, for example, led to a strong enrichment of ventral cells. For 23 perturbations, we detected changes in the abundance of at least one cell type (CMH test, false discovery rate (FDR)\\u2009&lt;\\u20090.05) (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig2\">2c</a>). On the other hand, six perturbations specifically targeted one cell type without affecting others, including <i>ADNP</i> (L2/3 enrichment), <i>POGZ</i> (RGC enrichment) and <i>SETD5</i> (ccv-RGC enrichment).</p><p>Among the progenitors, we identified IPC depletion as a strong, convergent effect in 12 perturbations (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig2\">2c</a>) (for example, <i>CHD2</i>, <i>KAT2B</i> and <i>KMT2C</i>). Additionally, we observed an enrichment of v-RGCs and/or ccv-RGCs in 10 perturbations (for example, <i>ARID1B</i>, <i>BCL11A</i> and <i>DEAF1</i>) and an enrichment of INPs in 10 perturbations (for example, <i>ILF2</i>, <i>MED13</i> and <i>TCF20</i>). To validate these phenotypes, we generated individually perturbed organoids for several genes. We performed immunohistochemical analyses of IPCs and INPs at day 60, a stage when organoids present\\xa0a radially organized structure including ventricular zone\\xa0(VZ), subventricular zone\\xa0(SVZ) and cortical plate, enabling robust examination of the progenitors<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 35\" title=\"Qian, X. et al. Sliced human cortical organoids for modeling distinct cortical layer formation. Cell Stem Cell 26, 766–781.e9 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR35\" id=\"ref-link-section-d3800159e1245\">35</a></sup>. Consistent with the screen data, we detected significantly decreased EOMES<sup>+</sup> IPCs in <i>KMT2C</i> and <i>PHF3</i> perturbations (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig15\">11</a>) and increased DLX2<sup>+</sup> INPs in <i>KMT2C</i>, <i>MED13</i>, <i>PHF3</i> and <i>TBL1XR1</i> perturbations (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig16\">12</a>). Furthermore, we analysed specific neuronal subpopulations and found that L2/3 excitatory neurons were more impacted and depleted in the majority of perturbations (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig2\">2c</a>). Analysis of interneurons revealed enrichment of both CGE-INs and LGE-INs in three perturbations and enrichment of only CGE-INs or LGE-INs in four perturbations. In addition, LGE PAX6<sup>+</sup> interneurons were depleted by <i>PHF3</i> perturbation. These data indicate an interneuron subtype-specific response to ASD genetic perturbations. Beyond neuronal cell clusters, we found that astrocytes were significantly enriched in <i>CIC</i> perturbation.</p><p>To assess the consistency of the effects across replicates from different batches, we performed a <i>t</i>-test on individual enrichment and depletion effects (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig14\">10</a>). We observed that most effects detected at the single-cell level were also largely consistent across organoids grown from different batches, further supporting the robustness and reproducibility of our system.</p><p>Collectively, the CHOOSE system allowed us to simultaneously investigate the effects of multiple ASD genes on cell fate determination. We found that progenitors, including IPCs and INPs, as well as L2/3 excitatory neuronss, were among the most affected cell types. Furthermore, our data indicate that ASD pathology could emerge as early as the neural progenitor stage.</p></div></div></section><section data-title=\"Altered gene expression upon ASD gene perturbation\"><div class=\"c-article-section\" id=\"Sec6-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec6\">Altered gene expression upon ASD gene perturbation</h2><div class=\"c-article-section__content\" id=\"Sec6-content\"><p>To further assess the molecular changes caused by each perturbation, we performed a differential gene expression analysis, comparing each perturbation with controls within dorsal and ventral trajectories. We detected 2,071 differentially expressed genes (DEGs) across all perturbations (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig3\">3a</a> and Supplementary Data\\xa0<a data-track=\"click\" data-track-label=\"link\" data-track-action=\"supplementary material anchor\" href=\"/articles/s41586-023-06473-y#MOESM4\">2</a>). Additionally, we could identify genes dysregulated in both dorsal and ventral populations, as well as those specifically dysregulated in one population (<i>KMT2C</i>, <i>LEO1</i> perturbation) (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig17\">13a</a>). We ranked DEGs by detection frequency and discovered that many genes were differentially expressed in multiple perturbations (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig3\">3b</a>). Notably, in the dorsal populations, seven perturbations caused <i>CHCHD2</i> downregulation (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig3\">3a</a>). <i>CHCHD2</i> encodes a mitochondrial protein, and its downregulation has been observed in neurons from the postmortem brains of patients with\\xa0ASD<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 34\" title=\"Velmeshev, D. et al. Single-cell genomics identifies cell type–specific molecular changes in autism. Science 364, 685–689 (2019).\" href=\"/articles/s41586-023-06473-y#ref-CR34\" id=\"ref-link-section-d3800159e1341\">34</a></sup>. In the ventral cell populations, the most frequently detected DEG is the adhesion molecule\\xa0gene <i>CSMD1</i> (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig3\">3a,b</a>), which is upregulated by <i>ARID1B</i>, <i>CIC</i>, <i>MED13</i> and <i>PHF3</i> perturbations and downregulated by <i>LEO1</i> and <i>KMT2C</i> perturbations. To ensure a balanced impact of differential DEG numbers across all perturbations on downstream analyses, we selected the top 30 DEGs for each perturbation (TOP-DEGs) for gene ontology term enrichment analysis. We found that cell adhesion, cell differentiation, forebrain development and axogenesis were among the most associated biological processes (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig3\">3c</a> and Supplementary Data\\xa0<a data-track=\"click\" data-track-label=\"link\" data-track-action=\"supplementary material anchor\" href=\"/articles/s41586-023-06473-y#MOESM5\">3</a>). We also observed many perturbation-specific gene ontology terms, many of which confirm previous studies, further supporting the power of detecting complex biological phenotypes with the CHOOSE system (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig17\">13b</a> and Supplementary Data\\xa0<a data-track=\"click\" data-track-label=\"link\" data-track-action=\"supplementary material anchor\" href=\"/articles/s41586-023-06473-y#MOESM6\">4</a>). Some notable examples include the gene ontology terms ribosome assembly (<i>SETD5</i> perturbation)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 36\" title=\"Nakagawa, T. et al. The autism-related protein SETD5 controls neural cell proliferation through epigenetic regulation of rDNA expression. iScience 23, 101030 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR36\" id=\"ref-link-section-d3800159e1387\">36</a></sup>, mitochondrion organization (<i>FOXP1</i> perturbation)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 37\" title=\"Wang, J. et al. Mitochondrial dysfunction and oxidative stress contribute to cognitive and motor impairment in FOXP1 syndrome. Proc. Natl Acad. Sci. USA 119, e2112852119 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR37\" id=\"ref-link-section-d3800159e1394\">37</a></sup>, lipid homoeostasis (<i>IRF2BPL</i> perturbation)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 38\" title=\"Chen, H.-H. et al. IRF2BP2 reduces macrophage inflammation and susceptibility to atherosclerosis. Circ. Res. 117, 671–683 (2015).\" href=\"/articles/s41586-023-06473-y#ref-CR38\" id=\"ref-link-section-d3800159e1401\">38</a></sup>, autophagosome maturation (<i>KAT2B</i> perturbation)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 39\" title=\"Jia, Y.-L. et al. P300/CBP-associated factor (PCAF) inhibits the growth of hepatocellular carcinoma by promoting cell autophagy. Cell Death Dis. 7, e2400–e2400 (2016).\" href=\"/articles/s41586-023-06473-y#ref-CR39\" id=\"ref-link-section-d3800159e1409\">39</a></sup> and cilium development (<i>MECP2</i> perturbation)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 40\" title=\"Frasca, A. et al. MECP2 mutations affect ciliogenesis: a novel perspective for Rett syndrome and related disorders. EMBO Mol. Med. 12, e10270 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR40\" id=\"ref-link-section-d3800159e1416\">40</a></sup>.</p><div class=\"c-article-section__figure js-c-reading-companion-figures-item\" data-test=\"figure\" data-container-section=\"figure\" id=\"figure-3\" data-title=\"Dysregulated gene expression and regulatory networks caused by perturbations of ASD risk genes.\"><figure><figcaption><b id=\"Fig3\" class=\"c-article-section__figure-caption\" data-test=\"figure-caption-text\">Fig. 3: Dysregulated gene expression and regulatory networks caused by perturbations of ASD risk genes.</b></figcaption><div class=\"c-article-section__figure-content\"><div class=\"c-article-section__figure-item\"><a class=\"c-article-section__figure-link\" data-test=\"img-link\" data-track=\"click\" data-track-label=\"image\" data-track-action=\"view figure\" href=\"/articles/s41586-023-06473-y/figures/3\" rel=\"nofollow\"><picture><source type=\"image/webp\" srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig3_HTML.png?as=webp\"><img aria-describedby=\"Fig3\" src=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig3_HTML.png\" alt=\"figure 3\" loading=\"lazy\" width=\"685\" height=\"582\"></picture></a></div><div class=\"c-article-section__figure-description\" data-test=\"bottom-caption\" id=\"figure-3-desc\"><p><b>a</b>, Jitter plots showing DEGs detected in dorsal and ventral trajectories from each genetic perturbation. <b>b</b>, Jitter plot shows the frequency of DEGs detected from all perturbations separated by dorsal (orange) and ventral (blue) trajectories. Points belonging to the same gene are connected with a grey line. <b>c</b>, Top biological pathways enriched for TOP-DEGs (top 30 DEGs per perturbation) identified from all perturbations. <b>d</b>, Enrichment test of TOP-DEGs per perturbation in ID (sysID database) or ASD (SFARI database) genes. Two-sided Fisher exact test, <i>P</i>\\u2009=\\u20093.8\\u2009×\\u200910<sup>−7</sup>. <b>e</b>, GRN of 4-month-old cerebral organoids inferred by Pando showing developmental TF modules constructed on the basis of\\xa0their coexpression and interaction strengths. ASD-associated TF modules are highlighted in yellow (SFARI genes) and/or black (regulator of SFARI genes). <b>f</b>, Lolliplots show CHOOSE DEG-enriched TF modules in dorsal and ventral trajectories. The <i>x</i> axis represents the sign of the log odds ratio multiplied by the −log<sub>10</sub> FDR-corrected <i>P</i> value of a two-sided Fisher exact test (signed −log<sub>10</sub> FDR). The dashed lines indicate an FDR of 0.05. <b>g</b>, Circular gene regulatory subnetwork plots show primary and secondary targets of <i>OLIG1</i> and <i>EOMES</i>. ASD-specific TF modules are highlighted in black and yellow as described above. Blue edges indicate repressive connections, and red edges indicate activating connections. NPC, neural progenitor cell.</p></div></div><div class=\"u-text-right u-hide-print\"><a class=\"c-article__pill-button\" data-test=\"article-link\" data-track=\"click\" data-track-label=\"button\" data-track-action=\"view figure\" href=\"/articles/s41586-023-06473-y/figures/3\" data-track-dest=\"link:Figure3 Full size image\" aria-label=\"Full size image figure 3\" rel=\"nofollow\"><span>Full size image</span><svg width=\"16\" height=\"16\" focusable=\"false\" role=\"img\" aria-hidden=\"true\" class=\"u-icon\"><use xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"#icon-eds-i-chevron-right-small\"></use></svg></a></div></figure></div></div></div></section><section data-title=\"ASD-associated regulatory modules\"><div class=\"c-article-section\" id=\"Sec7-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec7\">ASD-associated regulatory modules</h2><div class=\"c-article-section__content\" id=\"Sec7-content\"><p>When combining the TOP-DEGs from all perturbations, we found that they were significantly enriched in risk genes associated with ASD (SFARI database, 1,031 genes; approximately twofold enrichment, Fisher exact test <i>P</i>\\u2009&lt;\\u200910<sup>−5</sup> against genes expressed in 5% of cells as background) (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig3\">3d</a> and Supplementary Data\\xa0<a data-track=\"click\" data-track-label=\"link\" data-track-action=\"supplementary material anchor\" href=\"/articles/s41586-023-06473-y#MOESM4\">2</a>). Notably, we did not observe enrichment in risk genes for other NDDs, such as intellectual disability (ID; sysID database, 936 primary ID genes) (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig3\">3d</a>), suggesting that certain biological processes and regulatory programs might be specifically relevant to ASD-associated gene perturbations. To explore these potential gene regulatory ‘hubs’, we first generated single-cell multiome data, including single-cell transcriptome and chromatin accessibility modalities, from 4-month-old cerebral organoids (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig18\">14a–g</a>). Using Pando<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 41\" title=\"Fleck, J.S., Jansen, S.M.J., Wollny, D. et al. Inferring and perturbing cell fate regulomes in human brain organoids. Nature &#xA;                  https://doi.org/10.1038/s41586-022-05279-8&#xA;                  &#xA;                 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR41\" id=\"ref-link-section-d3800159e1509\">41</a></sup>, we harnessed these multimodal measurements to infer a GRN of the developing telencephalon and extract sets of genes regulated by each transcription factor (TF) module, as well as positive and negative regulatory interactions between the TFs (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig18\">14h,i</a> and Supplementary Data\\xa0<a data-track=\"click\" data-track-label=\"link\" data-track-action=\"supplementary material anchor\" href=\"/articles/s41586-023-06473-y#MOESM7\">5</a>). We visualized this GRN on the level of TFs using a uniform manifold approximation and projection (UMAP) embedding<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 42\" title=\"Becht, E. et al. Dimensionality reduction for visualizing single-cell data using UMAP. Nat. Biotechnol. 37, 38–44 (2019).\" href=\"/articles/s41586-023-06473-y#ref-CR42\" id=\"ref-link-section-d3800159e1519\">42</a></sup>, which revealed distinct TF groups active in neural progenitor cells (<i>PAX6</i>, <i>GLI3</i>), inhibitory neurons (<i>DLX2</i>, <i>DLX6</i>) and excitatory neurons (<i>NEUROD2</i>, <i>NFIB</i>, <i>SATB2</i>) as well as regulatory interactions between the TFs (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig3\">3e</a>).</p><p>To test whether regulatory subnetworks indeed exist at which ASD risk genes accumulate, we tested all TF modules for enrichment with SFARI genes. We found significant enrichment for a set of 40 TFs (adjusted Fisher test <i>P</i>\\u2009&lt;\\u20090.01, more than twofold enrichment; for example, <i>EOMES</i>, <i>OLIG1</i>, <i>DLX2</i>) (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig18\">14j</a>), among which 14 TFs were encoded by ASD risk genes (for example, <i>NFIA</i>, <i>BCL11A</i>, <i>MEF2C</i>) (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig3\">3e</a>). All TF regulatory modules enriched in SFARI genes together form an ASD-associated sub-GRN (Supplementary Data\\xa0<a data-track=\"click\" data-track-label=\"link\" data-track-action=\"supplementary material anchor\" href=\"/articles/s41586-023-06473-y#MOESM8\">6</a>).</p><p>Next, we assessed the transcriptomic effect of ASD genetic perturbations in the context of the inferred GRN. We performed enrichment tests (Fisher exact test) on perturbation-induced TOP-DEGs (CHOOSE DEGs) from dorsal and ventral telencephalic cells separately. We found that, similar to ASD risk genes, CHOOSE DEGs were enriched in specific TF modules (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig3\">3f</a> and Supplementary Data\\xa0<a data-track=\"click\" data-track-label=\"link\" data-track-action=\"supplementary material anchor\" href=\"/articles/s41586-023-06473-y#MOESM9\">7</a>). In the ventral telencephalic cells, <i>CREB5</i>, <i>MEIS2</i>, <i>NFIA</i> and <i>OLIG1</i> were most strongly affected, whereas dorsal telencephalon-specific DEGs were strongly enriched in <i>SATB2</i>, <i>BACH2</i>, <i>MEF2C</i> and <i>EOMES</i> modules. Notably, some of the ASD-associated TF modules were among the most strongly enriched in CHOOSE DEGs, supporting their role in ASD-associated gene dysregulation (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig3\">3f</a>). We finally present gene regulatory subnetworks of <i>OLIG1</i> and <i>EOMES</i>, which are both enriched in ASD risk genes and strongly affected by ASD genetic perturbations (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig3\">3g</a>). Oligodendrocyte transcription factor 1 (OLIG1) is preferentially expressed in the ventral telencephalon and is a key regulator for interneuron and oligodendrocyte lineages. Eomesodermin (EOMES) is a key TF for the fate specification of IPCs in the dorsal telencephalon. The enrichment of <i>OLIG1</i> and <i>EOMES</i> regulomes suggests potentially vulnerable cell fate specification-related regulatory networks upon ASD genetic perturbations.</p><p>Thus, we have characterized gene expression changes for each genetic perturbation in both dorsal and ventral telencephalon and uncovered molecular programs shared between different perturbations. Leveraging GRN inference from multiomic data, we further identified ASD-associated TF modules during cortical development and critical regulatory hubs underlying the detected gene expression changes.</p></div></div></section><section data-title=\"Effects of ARID1B\\xa0perturbation on v-RGCs\"><div class=\"c-article-section\" id=\"Sec8-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec8\">Effects of <i>ARID1B</i>\\xa0perturbation on v-RGCs</h2><div class=\"c-article-section__content\" id=\"Sec8-content\"><p>Among the 36 genes, we found that <i>ARID1B</i> perturbation caused one of the most significant enrichments of v-RGCs (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig2\">2c</a>). Notably, the <i>OLIG1</i> regulatory module was also enriched in DEGs caused by <i>ARID1B</i> perturbation (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig19\">15a</a>). These data motivated us to further investigate how v-RGCs are affected by <i>ARID1B</i> perturbation in the screen. We used Cellrank<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 43\" title=\"Lange, M. et al. CellRank for directed single-cell fate mapping. Nat. Methods 19, 159–170 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR43\" id=\"ref-link-section-d3800159e1670\">43</a></sup> to delineate the developmental trajectories leading to different interneuron subtypes and OPC populations (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig4\">4a</a>). We visualized the terminal fate probabilities for each cell as a circular projection, which revealed a distinct differentiation trajectory from ventral progenitors towards early OPCs (<i>OLIG2</i>, <i>PDGFRA</i>) and a branching of INPs (<i>DLX2</i>) into different inhibitory neuronal fates (<i>DLX5</i>) (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig4\">4b</a>). We found that <i>ARID1B-</i>perturbed cells were strongly enriched in the OPC trajectory and had a higher percentage of <i>OLIG2</i><sup>+</sup> v-RGCs (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig4\">4c,d</a>). This is an interesting finding given that <i>OLIG2</i> is known to regulate progenitor self-renewal at earlier developmental stages and is a master regulator for oligodendrocyte lineage specification in the ventral telencephalon<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 44\" title=\"Petryniak, M. A., Potter, G. B., Rowitch, D. H. &amp; Rubenstein, J. L. R. Dlx1 and Dlx2 control neuronal versus oligodendroglial cell fate acquisition in the developing forebrain. Neuron 55, 417–433 (2007).\" href=\"/articles/s41586-023-06473-y#ref-CR44\" id=\"ref-link-section-d3800159e1707\">44</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 45\" title=\"Sun, Y. et al. Phosphorylation state of Olig2 regulates proliferation of neural progenitors. Neuron 69, 906–917 (2011).\" href=\"/articles/s41586-023-06473-y#ref-CR45\" id=\"ref-link-section-d3800159e1710\">45</a></sup>. We then analysed the\\xa0fate transition probabilities of ventral progenitors and found that <i>ARID1B</i>-perturbed v-RGCs have significantly higher transition probabilities towards early OPCs than neuronal fates (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig4\">4e</a>).</p><div class=\"c-article-section__figure js-c-reading-companion-figures-item\" data-test=\"figure\" data-container-section=\"figure\" id=\"figure-4\" data-title=\"Perturbation of ARID1B increases the transition of v-RGCs to early OPCs.\"><figure><figcaption><b id=\"Fig4\" class=\"c-article-section__figure-caption\" data-test=\"figure-caption-text\">Fig. 4: Perturbation of <i>ARID1B</i> increases the transition of v-RGCs to early OPCs.</b></figcaption><div class=\"c-article-section__figure-content\"><div class=\"c-article-section__figure-item\"><a class=\"c-article-section__figure-link\" data-test=\"img-link\" data-track=\"click\" data-track-label=\"image\" data-track-action=\"view figure\" href=\"/articles/s41586-023-06473-y/figures/4\" rel=\"nofollow\"><picture><source type=\"image/webp\" srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig4_HTML.png?as=webp\"><img aria-describedby=\"Fig4\" src=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig4_HTML.png\" alt=\"figure 4\" loading=\"lazy\" width=\"685\" height=\"738\"></picture></a></div><div class=\"c-article-section__figure-description\" data-test=\"bottom-caption\" id=\"figure-4-desc\"><p><b>a</b>, Circular projection of terminal fate probabilities shows ventral telencephalon differentiation trajectories. <b>b</b>, Trajectory branches defined by gene expressions of <i>DLX2</i> (INP), <i>DLX5</i> (IN), <i>OLIG2</i> (early OPC) and <i>PDGFRA</i> (late OPC). <b>c</b>, Differential density of cells with <i>ARID1B</i> perturbation versus control. <b>d</b>, Bar graph shows cells within the v-RGCs that are positive for <i>DLX2</i> (68.6 versus 81.5%), <i>OLIG2</i> (19.6 versus 36.2%) and both (9.8 versus 30.6%) in control versus <i>ARID1B</i> perturbation. <b>e</b>, Box plots showing transition probabilities of control (<i>n</i>\\u2009=\\u200951) and <i>ARID1B-</i>perturbed (<i>n</i>\\u2009=\\u200946) ventral progenitor cells towards OPCs. The centre lines represent the medians, the box limits show the 25–75% interquartile ranges and the whiskers indicate 1.5× the interquartile ranges. Two-sided Wilcoxon test <i>P</i> value\\u2009=\\u20097.7\\u2009×\\u200910<sup>−8</sup>. <b>f</b>, Immunohistochemistry for early OPCs (OLIG2) and INPs (DLX2) of day 40 ventralized brain organoids derived from control (c.220dupG repair) and two ARID1B patient iPS\\xa0cells. Scale bar, 200\\u2009μm. <b>g</b>, Violin plots (all data points and median values) show numbers of cells positive for OLIG2 and/or DLX2. Control, <i>n</i>\\u2009=\\u2009108 areas from 13 organoids, four batches; <i>ARID1B</i><sup>+/−</sup> (c.2201dupG), <i>n</i>\\u2009=\\u2009104 areas from 15 organoids, four batches; <i>ARID1B</i><sup>+/−</sup> (6q25.3del), <i>n</i>\\u2009=\\u200994 areas from 15 organoids, three batches. One-way analysis of variance (ANOVA) post hoc Tukey test; ***<i>P</i>\\u2009&lt;\\u20090.001. <b>h</b>, Prenatal magnetic resonance imaging scan and 3D reconstruction of LGE and CGE (marked as GE) from age-matched controls and a\\xa0patient with an <i>ARID1B</i> mutation showing enlarged GE in the patient (quantified in Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig14\">10</a>). Scale bar, 1\\u2009cm. <b>i</b>, Diagram showing <i>ARID1B</i> perturbation-induced cellular responses of ventral progenitors.\\xa0WT, wild type.</p></div></div><div class=\"u-text-right u-hide-print\"><a class=\"c-article__pill-button\" data-test=\"article-link\" data-track=\"click\" data-track-label=\"button\" data-track-action=\"view figure\" href=\"/articles/s41586-023-06473-y/figures/4\" data-track-dest=\"link:Figure4 Full size image\" aria-label=\"Full size image figure 4\" rel=\"nofollow\"><span>Full size image</span><svg width=\"16\" height=\"16\" focusable=\"false\" role=\"img\" aria-hidden=\"true\" class=\"u-icon\"><use xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"#icon-eds-i-chevron-right-small\"></use></svg></a></div></figure></div><p>Loss-of-function mutations in <i>ARID1B</i> have been shown to cause ID and ASD<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 7\" title=\"Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell 180, 568–584.e23 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR7\" id=\"ref-link-section-d3800159e1848\">7</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 46\" title=\"Moffat, J. J., Smith, A. L., Jung, E.-M., Ka, M. &amp; Kim, W.-Y. Neurobiology of ARID1B haploinsufficiency related to neurodevelopmental and psychiatric disorders. Mol. Psychiatr. 27, 476–489 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR46\" id=\"ref-link-section-d3800159e1851\">46</a></sup>. To confirm whether our findings are relevant to human disorders, we recruited two patients with heterozygous <i>ARID1B</i> mutations. Patient 1 harbours a nucleotide duplication (c.2201dupG), resulting in an early STOP codon. Patient 2 carries a microdeletion (6q25.3del) that includes exon 8 and the downstream region of the <i>ARID1B</i> locus (Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig19\">15b,c</a>). We established iPS\\xa0cell lines from both patients and a mutation-corrected cell line for patient 1 as an isogenic control. To investigate the behaviour of v-RGCs, we used a previously published protocol that uses smoothened agonist (SAG) and inhibitor of Wnt production-2 (IWP2) to specifically guide organoids to develop ventral telencephalic tissue<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 47\" title=\"Bagley, J. A., Reumann, D., Bian, S., Lévi-Strauss, J. &amp; Knoblich, J. A. Fused cerebral organoids model interactions between brain regions. Nat. Methods 14, 743–751 (2017).\" href=\"/articles/s41586-023-06473-y#ref-CR47\" id=\"ref-link-section-d3800159e1865\">47</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 48\" title=\"Bajaj, S. et al. Neurotransmitter signaling regulates distinct phases of multimodal human interneuron migration. EMBO J. 40, e108714 (2021).\" href=\"/articles/s41586-023-06473-y#ref-CR48\" id=\"ref-link-section-d3800159e1868\">48</a></sup>. In line with our previous findings, we observed considerably increased <i>OLIG2</i><sup>+</sup>, <i>DLX2</i><sup>+</sup> and <i>DLX2</i><sup>+</sup><i>OLIG2</i><sup>+</sup> cells in 40-day-old organoids from both patients compared with control organoids (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig4\">4f,g</a>). To further explore the potential consequences of such defects in patients, we analysed the prenatal brain structure of patient 2 at two gestation stages (gestational weeks (GW)\\xa022 and GW31) by intrauterine super-resolution magnetic resonance imaging (MRI). Three-dimensional reconstruction of the ganglionic eminence (GE), the source of ventral telencephalon progenitors, and normalization to cortical and total brain volume revealed an enlarged GE compared with multiple age-matched controls at both examined developmental stages (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig4\">4h</a> and Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig19\">15d–h</a>), which could be partially due to an increase in ventral progenitors. Taken together, the enrichment of v-RGCs and ccv-RGCs (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig2\">2a</a>), the higher transition probability of v-RGCs to early OPCs, and the increased proportion of <i>OLIG2</i><sup>+</sup> cells in our screen and in organoids generated from two patient iPS\\xa0cell lines all suggest that <i>ARID1B</i> perturbation leads to abnormal ventral progenitor expansion and aberrant cell fate specification (Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig4\">4i</a>). The enlarged volume of GE in the patient with an\\xa0<i>ARID1B</i> mutation is consistent with these observations.</p></div></div></section><section data-title=\"Discussion\"><div class=\"c-article-section\" id=\"Sec9-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec9\">Discussion</h2><div class=\"c-article-section__content\" id=\"Sec9-content\"><p>We have developed the CHOOSE system to characterize loss-of-function phenotypes of high-risk ASD genes across dozens of cell types spanning early brain developmental stages in cerebral organoids. By employing a pooled CRISPR screening system in conjunction with validation, our study provides a developmental and cell type-specific phenotypic database for ASD gene loss-of-function research. IPCs, transit-amplifying dorsal progenitors that generate neurons for all cortical layers and contribute to the evolutionary expansion of the human cortex<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 49\" title=\"Martínez-Cerdeño, V., Noctor, S. C. &amp; Kriegstein, A. R. The role of intermediate progenitor cells in the evolutionary expansion of the cerebral cortex. Cereb. Cortex 16, i152–i161 (2006).\" href=\"/articles/s41586-023-06473-y#ref-CR49\" id=\"ref-link-section-d3800159e1922\">49</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 50\" title=\"Pebworth, M.-P., Ross, J., Andrews, M., Bhaduri, A. &amp; Kriegstein, A. R. Human intermediate progenitor diversity during cortical development. Proc. Natl Acad. Sci. USA 118, e2019415118 (2021).\" href=\"/articles/s41586-023-06473-y#ref-CR50\" id=\"ref-link-section-d3800159e1925\">50</a></sup>, appeared to be particularly susceptible to ASD genetic perturbations. Among the ventral telencephalon cells, we have identified strong enrichment of v-RGCs, ccv-RGCs and INPs, which are progenitors that differentiate into interneurons and oligodendrocytes<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 44\" title=\"Petryniak, M. A., Potter, G. B., Rowitch, D. H. &amp; Rubenstein, J. L. R. Dlx1 and Dlx2 control neuronal versus oligodendroglial cell fate acquisition in the developing forebrain. Neuron 55, 417–433 (2007).\" href=\"/articles/s41586-023-06473-y#ref-CR44\" id=\"ref-link-section-d3800159e1929\">44</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 51\" title=\"Kessaris, N. et al. Competing waves of oligodendrocytes in the forebrain and postnatal elimination of an embryonic lineage. Nat. Neurosci. 9, 173–179 (2006).\" href=\"/articles/s41586-023-06473-y#ref-CR51\" id=\"ref-link-section-d3800159e1932\">51</a></sup>. Furthermore, our findings indicate that L2/3 excitatory neurons are more vulnerable than other neuronal populations\\xa0to ASD perturbations. This aligns with the observation that ASD risk gene coexpression networks are enriched in upper-layer neurons during development and that these neurons are preferentially affected in ASD patients<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 6\" title=\"Parikshak, N. N. et al. Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism. Cell 155, 1008–1021 (2013).\" href=\"/articles/s41586-023-06473-y#ref-CR6\" id=\"ref-link-section-d3800159e1936\">6</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 34\" title=\"Velmeshev, D. et al. Single-cell genomics identifies cell type–specific molecular changes in autism. Science 364, 685–689 (2019).\" href=\"/articles/s41586-023-06473-y#ref-CR34\" id=\"ref-link-section-d3800159e1939\">34</a></sup>. In our screen, we assessed 19 ASD genes known to be involved in epigenetic regulation. Despite their broad role in cell differentiation, perturbations of these genes impacted specific cell types and biological processes during brain development. DEG and gene ontology enrichment analyses revealed both common and distinct molecular processes impaired in different perturbations, suggesting that both convergent and divergent mechanisms contribute to ASD pathophysiology. Furthermore, we constructed a telencephalon developmental trajectories-based GRN and identified ASD-associated regulatory modules in dorsal and ventral cell populations. The <i>OLIG1</i> module is particularly interesting as many of its downstream targets are ASD risk genes. This module was previously identified as crucial for oligodendrocyte differentiation in the developing human cortex<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 22\" title=\"Trevino, A. E. et al. Chromatin and gene-regulatory dynamics of the developing human cerebral cortex at single-cell resolution. Cell 184, 5053–5069.e23 (2021).\" href=\"/articles/s41586-023-06473-y#ref-CR22\" id=\"ref-link-section-d3800159e1946\">22</a></sup>, highlighting the involvement of the oligodendrocytes in ASD pathophysiology.</p><p>The combination of high-content perturbation screening and validation in a patient-specific context exemplifies the effectiveness of employing organoid systems to study NDDs. We discovered that loss of <i>ARID1B</i> leads to increased transition of ventral progenitors to early OPCs. Importantly, perturbations of three BAF complex members (<i>ARID1B</i>, <i>BCL11A</i> and <i>SMARRC2</i>) all lead to enrichment of v-RGCs, indicating the critical role of the BAF complex in regulating ventral telencephalon cell fate specification. Given the cell type-specific expression of each BAF subunit and their involvement in NDDs<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 52\" title=\"Sokpor, G., Xie, Y., Rosenbusch, J. &amp; Tuoc, T. Chromatin remodeling BAF (SWI/SNF) complexes in neural development and disorders. Front. Mol. Neurosci. 10, 243 (2017).\" href=\"/articles/s41586-023-06473-y#ref-CR52\" id=\"ref-link-section-d3800159e1965\">52</a></sup>, it would be interesting to investigate how ARID1B or other subunits regulate oligodendrocyte and interneuron specification, as well as their contribution to NDDs.</p><p>Our study has limitations. First, our system lacks certain brain cell types, such as microglia, and does not include interneurons derived from the medial ganglionic eminence. Future studies should explore the impact of ASD gene alterations on these cell types or use a system that better resembles in vivo cell-type complexity. Second, we do not know whether perturbed cells are heterozygous or homozygous for each mutant. It would be interesting to generate precisely edited cells to compare mutation-specific phenotypes. Additionally, the effects of perturbations of ASD risk genes on cell-type abundances can sometimes be transient<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 11\" title=\"Paulsen, B. et al. Autism genes converge on asynchronous development of shared neuron classes. Nature 602, 268–273 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR11\" id=\"ref-link-section-d3800159e1972\">11</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 53\" title=\"Villa, C. E. et al. CHD8 haploinsufficiency links autism to transient alterations in excitatory and inhibitory trajectories. Cell Rep. 39, 110615 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR53\" id=\"ref-link-section-d3800159e1975\">53</a></sup>, and therefore certain abnormalities during development may not be captured.</p><p>The ability to determine cell type-specific contributions to genetic disorders in a systematic, scalable and efficient manner will greatly enhance our understanding of disease mechanisms. As the CHOOSE system provides a robust, precisely controlled screening strategy, we anticipate that it will be widely applied beyond brain organoids to study disease-associated genes.</p></div></div></section><section data-title=\"Methods\"><div class=\"c-article-section\" id=\"Sec10-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec10\">Methods</h2><div class=\"c-article-section__content\" id=\"Sec10-content\"><h3 class=\"c-article__sub-heading\" id=\"Sec11\">Stem cell and cerebral organoid culture conditions</h3><p>Feeder-free hES\\xa0cells or iPS\\xa0cells were cultured on hES\\xa0cell-qualified Matrigel (Corning, catalogue no. 354277)-coated plates with Essential8 stem cell medium supplemented with bovine serum albumin (BSA). H9 embryonic stem cells were obtained from WiCell. Cells were maintained in a 5% CO<sub>2</sub> incubator at 37\\u2009°C. All cell lines were authenticated using a short tandem repeat assay, tested for genomic integrity using single-nucleotide polymorphism (SNP) array genotyping and routinely tested negative for mycoplasma.</p><p>Cerebral organoids were generated using a previously published protocol with modifications<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 23\" title=\"Lancaster, M. A. et al. Guided self-organization and cortical plate formation in human brain organoids. Nat. Biotechnol. 35, 659–666 (2017).\" href=\"/articles/s41586-023-06473-y#ref-CR23\" id=\"ref-link-section-d3800159e1999\">23</a></sup>. In brief, cells were cultured to 70–80% confluent, and 16,000 live cells in 150\\u2009μl Essential8 medium supplemented with Revitacell (ThermoFisher, catalogue no. A2644501) were added to each well of a U-bottom ultralow attachment 96-well plate (Corning, catalogue no. CLS3473) to form embryoid bodies. For eCas9 induction, 4-hydroxytamoxifen (Sigma-Aldrich, catalogue no. H7904) was added on day 5 at a concentration of 0.3\\u2009μg\\u2009ml<sup>−1</sup>. Neural induction was started on day 6. Embryoid bodies were embedded in Matrigel (Corning, catalogue no. 3524234) at day 11 or 12 based on morphology check. CHIR99021 (Merck, catalogue no. 361571) at 3\\u2009μM was added from day 13 to day 16, and medium was switched to improved differentiation medium supplemented with\\xa0B27 minus vitamin A\\xa0(IDM-A) at day 14. On day 25, medium was switched to improved differentiation medium\\u2009supplemented with\\xa0B27 plus vitamin A\\xa0(IDM+A); 1% dissolved Matrigel was added to the medium from day 40 to day 90. From day 60 to day 70, medium was gradually switched to Brainphys neuronal medium (Stemcell Technologies, catalogue no. 05790) and supplemented with brain-derived neurotrophic factor (BDNF) (20\\u2009ng\\u2009ml<sup>−1</sup>; Stemcell Technologies, catalogue no. 78005.3), glial cell line-derived neurotrophic factor (GDNF) (20\\u2009mg\\u2009ml<sup>−1</sup>; Stemcell Technologies, catalogue no. 78058.3) and bucladesine sodium (1\\u2009mM; MedChemExpress, catalogue no. HY-B0764)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 24\" title=\"Eichmüller, O. L. et al. Amplification of human interneuron progenitors promotes brain tumors and neurological defects. Science 375, eabf5546 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR24\" id=\"ref-link-section-d3800159e2009\">24</a></sup>. For ventralized organoids, we followed a previously published protocol<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 47\" title=\"Bagley, J. A., Reumann, D., Bian, S., Lévi-Strauss, J. &amp; Knoblich, J. A. Fused cerebral organoids model interactions between brain regions. Nat. Methods 14, 743–751 (2017).\" href=\"/articles/s41586-023-06473-y#ref-CR47\" id=\"ref-link-section-d3800159e2014\">47</a></sup>. Embryoid bodies were not embedded, and patterning factors, including 100\\u2009nm SAG (Merck-Millipore, catalogue no. US1566660) and 2.5\\u2009μM IWP2 (Sigma-Aldrich, catalogue no. IO536), were added from day 5 to day 11.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec12\">CHOOSE screen</h3><h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec13\">sgRNA selection and cloning</h4><p>The top four sgRNAs were first selected on the basis of\\xa0predictions using multilayered Vienna Bioactivity CRISPR (VBC) score<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 54\" title=\"Michlits, G. et al. Multilayered VBC score predicts sgRNAs that efficiently generate loss-of-function alleles. Nat. Methods 17, 708–716 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR54\" id=\"ref-link-section-d3800159e2030\">54</a></sup> and then subjected to the reporter assay (below) to test editing efficiency. sgRNAs were cloned into the gRNA reporter assay lentivirus construct (containing the dual sgRNA cassette: U6-sgRNA1-H1-sgRNA2) using the GeCKO cloning protocol<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 55\" title=\"Sanjana, N. E., Shalem, O. &amp; Zhang, F. Improved vectors and genome-wide libraries for CRISPR screening. Nat. Methods 11, 783–784 (2014).\" href=\"/articles/s41586-023-06473-y#ref-CR55\" id=\"ref-link-section-d3800159e2034\">55</a></sup>. The two sgRNAs were cloned using type\\xa0IIS class restriction enzymes FastDigest BpiI (ThermoFisher, catalogue no. FD1014) and Esp3I (ThermoFisher, catalogue no. FD0454) separately and verified using Sanger sequencing. All gRNAs used for this study can be found in Supplementary Table <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"supplementary material anchor\" href=\"/articles/s41586-023-06473-y#MOESM1\">1</a>.</p><h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec14\">sgRNA reporter assay</h4><p>A construct containing dTomato-2A-gRNA target array-TagBFP under the RSV promoter was assembled using Gibson assembly. The construct was packaged into retrovirus using the Platinum-E retroviral packaging cell line via the calcium phosphate-based transfection method. Virus-containing supernatant (Dulbecco’s modified Eagle’s\\xa0medium (DMEM), 10% fetal bovine serum (FBS), 2\\u2009mM <span class=\"u-small-caps\">l</span>-glutamine, 100\\u2009U\\u2009ml<sup>−1</sup> penicillin and 0.1\\u2009mg\\u2009ml<sup>−1</sup> streptomycin) was collected for up to 72\\u2009h, filtered through a 0.45-μm filter and then stored on ice. Retroviruses were then used to infect NIH-3T3 cells, and dTomato-positive cells were sorted using flow cytometry into single cells to establish reporter cell lines. To deliver sgRNAs, the lentiviral construct containing the dual gRNA cassette and the\\xa0spleen focus-forming virus (SFFV) promoter driving eCas9 were packaged using HEK293 cells to produce lentivirus. The reporter 3T3 cell lines generated above were cultured in six-well plates and infected with lentivirus containing dual sgRNA cassette targeting each gene individually. BFP fluorescence was measured at 7, 14 and 20\\u2009days postinfection. Fluorescent changes at 20 days postinfection were used to evaluate the gRNA editing efficiency. In total, 98 dual sgRNA cassettes were tested for 36 genes.</p><h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec15\">Generation of barcoded CHOOSE lentivirus, hES\\xa0cell infection and embryoid body generation</h4><p>The CHOOSE lentiviral vector was constructed based on a previously published lentiviral vector that carries a CAG driving ERT2-Cre-ERT2-P2A-EGFP-P2A-puro cassette<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 12\" title=\"Esk, C. et al. A human tissue screen identifies a regulator of ER secretion as a brain-size determinant. Science 370, 935–941 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR12\" id=\"ref-link-section-d3800159e2064\">12</a></sup>. A multicloning site including NheI and SgsI recognition sequences was introduced to the 3′ LTR of the lentivirus backbone according to the CROP-seq vector design<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 17\" title=\"Datlinger, P. et al. Pooled CRISPR screening with single-cell transcriptome read-out. Nat. Methods 14, 297–301 (2017).\" href=\"/articles/s41586-023-06473-y#ref-CR17\" id=\"ref-link-section-d3800159e2068\">17</a></sup>. Then, the original U6 sgRNA expression cassette was removed; instead, the dual sgRNA (U6-sgRNA1-H1-sgRNA2) cassette was introduced to the 3′ LTR cloning site. To generate a barcoded library, the following primers were used to individually amplify (8–10 cycles, monitored using a quantitative polymerase chain reaction (qPCR) machine, stopped when reaching to logarithmic phase) each dual sgRNA cassette from the lentiviral construct used in the reporter assay while introducing a 15\\u2009base\\xa0pair barcode.</p><p>FW primer: 5′-tcgaccgctagcagggcctatttcccatga-3′.</p><p>RV primer: 5′-cagtagggcgcgcc<u>NVDNHBNVDNHBNVD</u>ccggcgaaccatgatcaaa-3′.</p><p>Equal molar amounts of amplicons for the ASD library (36 paired sgRNAs targeting ASD genes) or control library (a paired non-targeting control gRNA) were pooled. Amplicons and lentiviral backbone were then digested with FastDigest NheI (ThermoFisher, catalogue no. FD0973) and FastDigest SgsI (ThermoFisher, catalogue no. FD1894) and gel purified. Ligation was performed using T4 DNA ligase (ThermoFisher, catalogue no. EL0011) and cleaned up by phenol-chloroform extraction. In total, 90\\u2009ng of ASD library plasmids and 30\\u2009ng of control library plasmids were used for electroporation of MegaX DH10B T1R Electrocomp Cells (ThermoFisher, catalogue no. C640003) following the manufacturer’s guide. Bacteria were plated on lithium borate medium plates containing ampicillin. Dilutions were performed to calculate the complexity; 2.6\\u2009×\\u200910<sup>7</sup> colonies were obtained for the ASD library, and 0.5\\u2009×\\u200910<sup>7</sup> colonies were obtained for the control library. Lentiviruses were packaged using HEK293T cells, and infection of hES\\xa0cells was performed as before<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 12\" title=\"Esk, C. et al. A human tissue screen identifies a regulator of ER secretion as a brain-size determinant. Science 370, 935–941 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR12\" id=\"ref-link-section-d3800159e2088\">12</a></sup>. Infection rate was controlled to be lower than 5% to prevent multiple infections<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 18\" title=\"Doench, J. G. Am I ready for CRISPR? A user’s guide to genetic screens. Nat. Rev. Genet. 19, 67–80 (2018).\" href=\"/articles/s41586-023-06473-y#ref-CR18\" id=\"ref-link-section-d3800159e2092\">18</a></sup>; 6.6\\u2009×\\u200910<sup>5</sup> ASD library cells and 2.3\\u2009×\\u200910<sup>5</sup> control library cells positive for GFP were sorted by flow cytometry. Cells were recovered and passaged two times in 10\\u2009cm dishes to maintain maximum complexity. Cells were mixed with a ratio of 96:4 (ASD:control) and then used to make embryoid bodies. For individual gene validations, lentivirus carrying a dual gRNA cassette only targeting one gene was packaged and used to infect the eCas9-inducible cell line. Cells were then collected by FACS and used to make embryoid bodies. Organoids were cultured using the conditions described above.</p><h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec16\">Cerebral organoid tissue dissociation, FACS and scRNA-seq</h4><p>For each library, three to seven organoids at 4\\u2009months were pooled, washed twice in Dulbecco’s phosphate-buffered saline (DPBS)<sup>−/−</sup> and dissociated using the gentleMACS dissociator in trypsin–accutase (1×) solution with TURBO DNase (2\\u2009μl\\u2009ml<sup>−1</sup>; ThermoFisher, catalogue no. AM2238). After dissociation, DPBS<sup>−/−</sup> supplemented with 10% FBS (DPBS–10% FBS) was gradually added to stop the reaction. Samples were then centrifuged at 400<i>g</i> for 5\\u2009min at 4\\u2009°C, and the supernatant was aspirated without touching the pellet. The pellet was then resuspended in an additional 1–2\\u2009ml of DPBS–10% FBS and then, filtered through a 70\\u2009μm strainer and FACS tubes. Cells were then stained with viability dye DRAQ7 (Biostatus; DR70250, 0.3\\u2009mM). Target live cells were sorted with a BD FACSAria III on Alexa 700 filter with low pressure (100\\u2009μm nozzle) and collected in DPBS–10% FBS at 4\\u2009°C. Cells were then centrifuged and resuspended in DPBS–10% FBS to achieve a target concentration of 450–1,000\\u2009cells per microliter. Samples with more than 85% viability were processed. For each library, 16,000 cells were loaded onto a 10× chromium controller to target a recovery of 10,000 cells. Libraries using the Chromium Single Cell 3′ Reagent Kits (v.3.1) were prepared following the 10× user guide. Libraries were sequenced on a Novaseq S2 or S4 flow cell with a target of 25,000 paired-end reads per cell.</p><h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec17\">Custom genomic reference</h4><p>Each cell expresses eCas9 from a genomic locus (AAVS1) and a polyadenylated dual sgRNA cassette, which is delivered by lentivirus and integrated into the genome. To cover these extrinsic elements, we built a custom genomic reference for mapping 10× single-cell data by amending the GRCh38 human reference. As the individual gRNA sequences differed, we masked them by Ns so as\\xa0not to interfere with mapping (individual gRNA information is distinguished in a separate counting pipeline). The sequences added covered the genomic loci of <i>AAVS1</i> with eCas9-dTomato-WPRE-SV40 and the masked lentiviral construct.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec18\">Emulsion PCR and target amplification</h3><p>Emulsion polymerase chain reaction (PCR) was used to recover gRNA and UCB sequences from plasmid libraries, genomic DNA extracted from lentivirus-infected hES\\xa0cells and cells sorted from CHOOSE mosaic organoids as well as 10× single-cell complementary\\xa0DNA libraries to reduce PCR bias and to prevent the generation of chimeric PCR products<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 56\" title=\"Omelina, E. S., Ivankin, A. V., Letiagina, A. E. &amp; Pindyurin, A. V. Optimized PCR conditions minimizing the formation of chimeric DNA molecules from MPRA plasmid libraries. BMC Genom. 20, 536 (2019).\" href=\"/articles/s41586-023-06473-y#ref-CR56\" id=\"ref-link-section-d3800159e2139\">56</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 57\" title=\"Williams, R. et al. Amplification of complex gene libraries by emulsion PCR. Nat. Methods 3, 545–550 (2006).\" href=\"/articles/s41586-023-06473-y#ref-CR57\" id=\"ref-link-section-d3800159e2142\">57</a></sup>. AmpliTaq Gold 360 master mix (ThermoFisher, catalogue no. 4398876) was used for all PCR reactions. Emulsion PCR was performed using the Micellula DNA Emulsion &amp; Purification Kit (EURX, catalogue no. E3600) according to the manufacturer’s guide. For target amplification from 10× single-cell libraries, heminested emulsion PCRs were performed using the following primers:</p><p>First PCR:\\xa0forward primer (FW): 5′-gcagacaaatggctgaacgctgacg-3′, reverse primer (RV): 5′-ccctacacgacgctcttccgatct-3′; second PCR: FW: 5′-ggagttcagacgtgtgctcttccgatcttgggaatcttataagttctgtatgagaccactctttcc-3′, RV: 5′-ccctacacgacgctcttccgatct-3′.</p><p>Amplicons were then indexed with unique NEB dual indexing primers, and amplifications were monitored in a qPCR machine and stopped when reaching the logarithmic phase. Amplicons were sequenced using the Illumina Nextseq2000 or Novaseq6000 system. All primers used can also be found in Supplementary Table <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"supplementary material anchor\" href=\"/articles/s41586-023-06473-y#MOESM1\">1</a>.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec19\">gRNA and UCB recovery and analyses</h3><p>gRNA sequences were extracted by cutting 5′- and 3′-flanking regions with cutadapt (10% error rate, 1–3\\xa0nucleotide\\u2009(nt) overlap, no indels)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 58\" title=\"Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet J 17, 10–12 (2011).\" href=\"/articles/s41586-023-06473-y#ref-CR58\" id=\"ref-link-section-d3800159e2163\">58</a></sup>. Sequences were filtered to be between 15 and 21\\u2009nt long. The corrected cell barcode (CBC) and the unique molecular identifier (UMI) of each read were derived via the 10× Genomics Cell Ranger 6.0.1 alignment<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 59\" title=\"Zheng, G. X. Y. et al. Massively parallel digital transcriptional profiling of single cells. Nat. Commun. 8, 14049 (2017).\" href=\"/articles/s41586-023-06473-y#ref-CR59\" id=\"ref-link-section-d3800159e2167\">59</a></sup>. Only reads with a corresponding gene expression (GEX) cell were accepted. Reads and target sequences were joined by allowing partial overlaps and hamming distances of two. Reads are counted towards unique CBC–UMI–gRNA combinations. A read count cutoff of 1% of the median read count of the UMI with the highest reads count per cell was applied. Cells with only one gRNA and more than one read were kept. In addition, within unique CBC–UMI combinations, only gRNA with more than 20% of the maximal read count of that group was kept. After read filtering, UMIs were counted for each CBC–gRNA combination. If more than one gRNA was found within a cell, only the gRNAs with equal UMI count compared with the maximum UMI count were kept. Only one-to-one combinations were considered further. Analogous to gRNA extraction, UCB was extracted with at least 6\\u2009nt overlap to the flanks. Sequences with 12\\u2009nt length were selected and had to follow the synthesis pattern. Further processing was done analogous to gRNA.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec20\">Preprocessing of single-cell transcriptomics data</h3><p>We first aligned reads to the above defined custom genomic reference with Cell Ranger 6.0 (10x Genomics) using pre-mRNA gene models and default parameters to produce the cell by gene UMI count matrix. UMI counts were then analysed in R using the Seurat v.4 (ref. <sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 60\" title=\"Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573–3587.e29 (2021).\" href=\"/articles/s41586-023-06473-y#ref-CR60\" id=\"ref-link-section-d3800159e2180\">60</a></sup>). We first filtered features detected in a minimum of three cells. Next, we filtered high-quality cells based on the number of genes detected (minimum 1,000, maximum 8,000), removing cells with high mitochondrial (less than 15%) or ribosomal (less than 20%) messenger\\xa0RNA content. Thereafter, expression matrices of high-quality cells were normalized (LogNormalize) and scaled to a total expression of 10,000 UMIs for each cell. Principal component analysis (PCA) was performed based on the <i>z</i>-scaled expression of the 2,000 most variable features (FindVariableFeatures()).</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec21\">Integration and annotation of single-cell transcriptomics data</h3><p>To annotate the dataset, we first extracted cells with control gRNAs and merged them with cells from uninduced organoids (35,203 cells). We integrated these unperturbed cells across libraries using Harmony<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 61\" title=\"Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Nat. Methods 16, 1289–1296 (2019).\" href=\"/articles/s41586-023-06473-y#ref-CR61\" id=\"ref-link-section-d3800159e2195\">61</a></sup> with default parameters. Using the integrated space, we clustered the dataset at a resolution of one using the Louvain algorithm<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 62\" title=\"Blondel, V. D., Guillaume, J.-L., Lambiotte, R. &amp; Lefebvre, E. Fast unfolding of communities in large networks. J Statist. Mech. Theory Exp. 2008, P10008 (2008).\" href=\"/articles/s41586-023-06473-y#ref-CR62\" id=\"ref-link-section-d3800159e2199\">62</a></sup> and annotated the clusters as dorsal and ventral telencephalons based on marker gene expression. We then split both trajectories and clustered again with a resolution of two to annotate the cell types more finely. This annotation of unperturbed cells was used to perform a label transfer onto the full dataset with perturbed cells using Seurat. The full CHOOSE dataset was further filtered for cells for which gRNAs were detected and integrated across libraries using the Seurat anchoring method. The integrated count matrix was log-normalized and scaled before computing a PCA. To visualize the dataset, the first 20 principal components were used to compute a UMAP embedding.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec22\">Assessment target gene expression in organoid and primary cell types</h3><p>To assess how the target genes in our screen were expressed in organoid and primary tissue, we obtained gene expression data from cell clusters in the developing human brain<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 32\" title=\"Braun, E. et al. Comprehensive cell atlas of the first-trimester developing human brain. Preprint at bioRxiv &#xA;                  https://doi.org/10.1101/2022.10.24.513487&#xA;                  &#xA;                 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR32\" id=\"ref-link-section-d3800159e2211\">32</a></sup> from <a href=\"https://storage.googleapis.com/linnarsson-lab-human/HumanFetalBrainPool.h5\">https://storage.googleapis.com/linnarsson-lab-human/HumanFetalBrainPool.h5</a>. For both the primary data and our organoid dataset, we summarized log-normalized expression for each cell type (‘CellClass’ in the primary dataset) by computing the arithmetic mean. We visualized the expression of CHOOSE target genes with a heat map as displayed in Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig10\">6l,m</a>.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec23\">RNA velocity</h3><p>To obtain count matrices for spliced and unspliced transcripts, we used kallisto (v.0.46.2)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 63\" title=\"Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. Nat. Biotechnol. 34, 525–527 (2016).\" href=\"/articles/s41586-023-06473-y#ref-CR63\" id=\"ref-link-section-d3800159e2233\">63</a></sup> through the command line tool loompy from fastq from the python package loompy (v.3.0.7; <a href=\"https://linnarssonlab.org/loompy/\">https://linnarssonlab.org/loompy/</a>). Using scVelo (v.0.2.4)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 31\" title=\"Bergen, V., Lange, M., Peidli, S., Wolf, F. A. &amp; Theis, F. J. Generalizing RNA velocity to transient cell states through dynamical modeling. Nat. Biotechnol. 38, 1408–1414 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR31\" id=\"ref-link-section-d3800159e2244\">31</a></sup>, moments were computed based on the first 20 principal components using the function scvelo.pp.moments() with n_neighbors\\u2009=\\u200930. RNA velocity was subsequently calculated using the function scvelo.tl.velocity() (mode\\u2009=\\u2009‘stochastic’), and a velocity graph was constructed using scvelo.tl.velocity_graph(). To obtain a pseudotemporal ordering describing the two differentiation trajectories, we first removed clusters annotated as cycling cells (MKI67+) and astrocytes (S100B+) from the dataset. We then calculated a pseudotime based on the velocity graph using the function scv.tl.velocity_pseudotime() for both trajectories separately.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec24\">Differential gRNA representation analysis</h3><p>To test whether perturbations affected fitness or proliferation capacity of cells, we compared gRNA representation in eCas9-induced (<i>n</i>\\u2009=\\u200914 pools of organoids from three batches) versus uninduced (<i>n</i>\\u2009=\\u20098 pools of organoids from two batches) samples. For each pool of organoids, we computed the fractions of cells with each gRNA. We then computed the average fold change of detection percentage between induced and uninduced samples and performed a two-sided <i>t</i>-test comparing both distributions. Multiple-testing correction on the resulting <i>P</i> values was performed using the Benjamini–Hochberg method.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec25\">Differential abundance testing</h3><p>To assess how the perturbation of ASD risk genes changes abundances of different organoid cell populations, we tested for enrichment of each gRNA in each annotated cell state versus the control. To control for confounding effects through differential gRNA sampling in different libraries, we used a CMH test stratified by library. Multiple-testing correction was performed using the Benjamini–Hochberg method, and a significance threshold of 0.05 was applied to the resulting FDR. Enrichment effects were plotted using the signed −log<sub>10</sub> FDR: that is, the sign of the log odds ratio (effect size) multiplied by the −log<sub>10</sub> FDR-corrected <i>P</i> value. To further assess the variability of the differential abundance effects across independent pools of organoids, we computed cell-type fold enrichment for each organoids pool and gRNA. For this, we used 14 scRNA-seq libraries obtained from independent pools of organoids as replicates from three batches. Two batches (11 replicates) used non-targeting gRNA as a control, and a third batch (three replicates) used eCas9-uninduced cells as an alternative control. We additionally computed a background distribution of enrichment effects from randomly permuted gRNA labels. We then performed a <i>t</i>-test for each perturbation and cell type against this background distribution.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec26\">Local cell compositional enrichment test</h3><p>To visualize the compositional changes induced by the genetic perturbations at a finer resolution, we used a method outlined in Nikolova et al.<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 64\" title=\"Nikolova, M. T. et al. Fate and state transitions during human blood vessel organoid development. Preprint at bioRxiv &#xA;                  https://doi.org/10.1101/2022.03.23.485329&#xA;                  &#xA;                 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR64\" id=\"ref-link-section-d3800159e2295\">64</a></sup> In brief, a\\xa0<i>k</i>-nearest neighbour (kNN) graph (<i>k</i>\\u2009=\\u2009200) of cells was constructed on the basis of Euclidean distance on the PCA-reduced CCA (canonical-correlation analysis) space. Next, a CMH test stratified by library was performed on the neighbourhood of each cell, comparing frequencies of the gRNA or gRNA pool and the pool of control gRNAs within and outside of the neighbourhood. The resulting neighbourhood enrichment score of each cell was defined as signed −log(<i>P</i>), where the sign was determined by the sign of the log-transformed odds ratio. A random walk with restart procedure was then applied to smooth the neighbourhood enrichment score of each cell. The smoothened enrichment scores were visualized on the UMAP embedding using the ggplot2 (ref. <sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 65\" title=\"Wickham, H. in ggplot2: Elegant Graphics for Data Analysis\\xa0147–168 (Springer, 2016).\" href=\"/articles/s41586-023-06473-y#ref-CR65\" id=\"ref-link-section-d3800159e2308\">65</a></sup>) function stat_summary_hex() (bins\\u2009=\\u200950).</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec27\">Differential expression analysis</h3><p>To investigate the transcriptomic changes caused by each perturbation, we performed differential expression analysis based on logistic regression. We used the Seurat function FindMarkers() (test.use\\u2009=\\u2009‘LR’) to find DEGs for each gRNA label versus control. Tests were performed on log-normalized transcript counts <i>Y</i> while treating library, cell_type and n_UMI as covariates in the model:</p><div id=\"Equa\" class=\"c-article-equation\"><div class=\"c-article-equation__content\"><span class=\"mathjax-tex\">$${Y}_{i}\\\\approx {\\\\rm{n}}\\\\_{\\\\rm{UMI}}+{\\\\rm{library}}+{\\\\rm{cell}}\\\\_{\\\\rm{type}}+{\\\\rm{condition}}.$$</span></div></div><p>Testing within each developmental trajectory was performed by omitting the cell_type covariate.\\xa0Multiple-testing correction was performed using the Benjamini–Hochberg method, and a significance threshold of 0.05 was applied to the resulting FDR to obtain a set of DEGs (CHOOSE DEGs). We further selected top 30 DEGs on the basis of absolute fold change for each gRNA (TOP-DEGs).</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec28\">DEG enrichment analysis</h3><p>To assess the biological processes in which the detected DEGs were involved, we performed gene ontology enrichment across all TOP-DEGs globally as well as using all detected DEGs for each target gene in excitatory and inhibitory neuron trajectories separately. As a background gene set, we used all genes expressed in more than 5% of cells in our dataset. To perform gene ontology analysis, we used the function ‘enrichGO’ from the R package clusterProfiler<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 66\" title=\"Wu, T. et al. clusterProfiler 4.0: a universal enrichment tool for interpreting omics data. Innovation 2, 100141 (2021).\" href=\"/articles/s41586-023-06473-y#ref-CR66\" id=\"ref-link-section-d3800159e2398\">66</a></sup> with ‘pAdjustMethod\\u2009=\\u2009′fdr′’. We filtered the results using a significance threshold of FDR\\u2009&lt;\\u20090.01. To test whether the set of TOP-DEGs was enriched for ASD-associated genes, we first obtained a list of risk genes from SFARI (<a href=\"https://gene.sfari.org/database/gene-scoring/\">https://gene.sfari.org/database/gene-scoring/</a>, 11 April 2021). We then tested the enrichment using a Fisher exact test with all genes expressed in more than 5% of cells in our dataset as the background. To assess the specificity of this enrichment, we obtained a list of ID risk genes from sysID (936 primary ID genes, <a href=\"https://sysndd.dbmr.unibe.ch\">https://sysndd.dbmr.unibe.ch</a>, 17 March 2022) and tested for enrichment among TOP-DEGs in the same way.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec29\">Processing of single-cell multiome data and GRN inference</h3><p>Initial transcript count and peak accessibility matrices for the multiome data were obtained from sequencing reads with Cell Ranger Arc and further processed using the Seurat (v.4.0.1) and Signac (v.1.4.0)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 67\" title=\"Stuart, T., Srivastava, A., Madad, S., Lareau, C. A. &amp; Satija, R. Single-cell chromatin state analysis with Signac. Nat. Methods 18, 1333–1341 (2021).\" href=\"/articles/s41586-023-06473-y#ref-CR67\" id=\"ref-link-section-d3800159e2424\">67</a></sup> R packages. Peaks were called from the fragment file using MACS2 (v.2.2.6)<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 68\" title=\"Feng, J., Liu, T., Qin, B., Zhang, Y. &amp; Liu, X. S. Identifying ChIP-seq enrichment using MACS. Nat. Protoc. 7, 1728–1740 (2012).\" href=\"/articles/s41586-023-06473-y#ref-CR68\" id=\"ref-link-section-d3800159e2428\">68</a></sup> and combined in a common peak set before merging. Transcript counts were log-normalized, and peak counts were normalized using term frequency–inverse document frequency normalization. To assess the cell composition of the multiome data, integration with the CHOOSE scRNA-seq data was performed using Seurat (FindIntegrationAnchors() -&gt; IntegrateData()) with default parameters. As a preprocessing step to GRN inference with Pando<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 41\" title=\"Fleck, J.S., Jansen, S.M.J., Wollny, D. et al. Inferring and perturbing cell fate regulomes in human brain organoids. Nature &#xA;                  https://doi.org/10.1038/s41586-022-05279-8&#xA;                  &#xA;                 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR41\" id=\"ref-link-section-d3800159e2432\">41</a></sup>, chromatin accessibility data were first coarse grained to a high-resolution cluster level. For this, control cells from the CHOOSE dataset were combined with the multiome dataset, and Louvain clustering was performed at a resolution of 20 based on the first 20 principal components calculated from the 2,000 most variable features (RNA). For each cluster, peak accessibility was summarized by computing the arithmetic mean from binarized peak counts so that each cell in the cluster was represented by the detection probability vector of each peak. To constrain the set of peaks considered by Pando, we used the union of PhastCons conserved elements<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 69\" title=\"Siepel, A. et al. Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res. 15, 1034–1050 (2005).\" href=\"/articles/s41586-023-06473-y#ref-CR69\" id=\"ref-link-section-d3800159e2436\">69</a></sup> from an alignment of 30 mammals (obtained from <a href=\"https://genome.ucsc.edu/\">https://genome.ucsc.edu/</a>) and candidate <i>cis</i>-regulatory elements derived from the ENCODE project<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 70\" title=\"Consortium, T. E. P. et al. Expanded encyclopaedias of DNA elements in the human and mouse genomes. Nature 583, 699–710 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR70\" id=\"ref-link-section-d3800159e2451\">70</a></sup> (initiate_grn()). In these regions, we scanned for TF motifs (find_motifs()) based on the motif database shipped with Pando, which was compiled from motifs derived from JASPAR and CIS-BP. Based on motif matches, cell-level log-normalized transcript counts and cluster-level peak accessibilities, we then inferred the GRN using the Pando function infer_grn() (peak_to_gene_method\\u2009=\\u2009‘GREAT’, upstream\\u2009=\\u2009100,000, downstream\\u2009=\\u2009100,000) for the 5,000 most variable features. Here, genes were associated with candidate regulatory regions in a 100,000 radius around the gene body using the method proposed by GREAT<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 71\" title=\"McLean, C. Y. et al. GREAT improves functional interpretation of cis-regulatory regions. Nat. Biotechnol. 28, 495–501 (2010).\" href=\"/articles/s41586-023-06473-y#ref-CR71\" id=\"ref-link-section-d3800159e2455\">71</a></sup>. From the model coefficients returned by Pando, TF modules were constructed using the function find_modules() (P_thresh\\u2009=\\u20090.05, rsq_thresh\\u2009=\\u20090.1, nvar_thresh\\u2009=\\u200910, min_genes_per_module\\u2009=\\u20095). To visualize subnetworks centred around one TF, we computed the shortest path from the TF to every gene in the GRN graph. If there were multiple shortest paths, we retained the one with the lowest average <i>P</i> value. The resulting graph was visualized with the R package ggraph (<a href=\"https://github.com/thomasp85/ggraph\">https://github.com/thomasp85/ggraph</a>) using the circular tree layout.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec30\">Enrichment testing for TF modules</h3><p>To find subnetworks of the GRN at which ASD-associated genes accumulate, we first obtained a list of ASD risk genes from SFARI (<a href=\"https://gene.sfari.org/database/gene-scoring/\">https://gene.sfari.org/database/gene-scoring/</a>). For all genes included in SFARI (1,031 genes), we tested for enrichment in TF modules using a Fisher exact test. All genes expressed in more than 5% of cells in our dataset (12,079 genes) were treated as the background. Fisher test <i>P</i> values were corrected for\\xa0multiple\\xa0testing using the Benjamini–Hochberg method, and significant enrichment was defined as FDR\\u2009&lt;\\u20090.01 and more than twofold enrichment (odds ratio). To assess which TF modules were most affected by genetic perturbations of ASD-associated genes, we similarly used a Fisher exact test. For the set of TOP-DEGs, we tested for enrichment in any of the inferred TF modules. Here, all genes included in the GRN (5,000 most variable features) were treated as the background.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec31\">Cell rank analysis</h3><p>To better understand the differentiation trajectories leading up to inhibitory neuron populations, we used CellRank<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 43\" title=\"Lange, M. et al. CellRank for directed single-cell fate mapping. Nat. Methods 19, 159–170 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR43\" id=\"ref-link-section-d3800159e2495\">43</a></sup> to compute transition probabilities into each terminal fate based on the previously computed velocity pseudotime. First, the clusters with the highest pseudotime for each terminal cell state were annotated as terminal states. We then constructed a Palantir kernel<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 72\" title=\"Setty, M. et al. Characterization of cell fate probabilities in single-cell data with Palantir. Nat. Biotechnol. 37, 451–460 (2019).\" href=\"/articles/s41586-023-06473-y#ref-CR72\" id=\"ref-link-section-d3800159e2499\">72</a></sup> (PalantirKernel()) based on velocity pseudotime and used Generalized Perron Cluster Cluster Analysis<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 73\" title=\"Reuter, B., Weber, M., Fackeldey, K., Röblitz, S. &amp; Garcia, M. E. Generalized Markov state modeling method for nonequilibrium biomolecular dynamics: exemplified on amyloid β conformational dynamics driven by an oscillating electric field. J. Chem. Theory Comput. 14, 3579–3594 (2018).\" href=\"/articles/s41586-023-06473-y#ref-CR73\" id=\"ref-link-section-d3800159e2503\">73</a></sup> (GPCCA()) to compute a terminal fate probability matrix (compute_absorption_probabilities()). All cell rank functions were run with default parameters. Fate probabilities for each cell were visualized using a circular projection<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 74\" title=\"Velten, L. et al. Human haematopoietic stem cell lineage commitment is a continuous process. Nat. Cell Biol. 19, 271–281 (2017).\" href=\"/articles/s41586-023-06473-y#ref-CR74\" id=\"ref-link-section-d3800159e2507\">74</a></sup>. In brief, we evenly spaced terminal states around a circle and assigned each state an angle <i>t</i>. We then computed two-dimensional coordinates (<span class=\"mathjax-tex\">\\\\({x}_{i}\\\\)</span>, <span class=\"mathjax-tex\">\\\\({y}_{i}\\\\)</span>) from the <span class=\"mathjax-tex\">\\\\(F\\\\in {R}^{{Nx}{n}_{{t}}}\\\\)</span> transition probability matrix for <i>N</i> cells and <span class=\"mathjax-tex\">\\\\({n}_{{t}}\\\\)</span> terminal states as</p><div id=\"Equb\" class=\"c-article-equation\"><div class=\"c-article-equation__content\"><span class=\"mathjax-tex\">$${x}_{i}=\\\\sum _{t}{f}_{it}\\\\cos {\\\\alpha }_{t}$$</span></div></div><div id=\"Equc\" class=\"c-article-equation\"><div class=\"c-article-equation__content\"><span class=\"mathjax-tex\">$${y}_{i}=\\\\sum _{t}{f}_{it}\\\\sin {\\\\alpha }_{t}.$$</span></div></div><p>To visualize enrichment of perturbed cells in this space, we used the method outlined in Nikolova et al.<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 64\" title=\"Nikolova, M. T. et al. Fate and state transitions during human blood vessel organoid development. Preprint at bioRxiv &#xA;                  https://doi.org/10.1101/2022.03.23.485329&#xA;                  &#xA;                 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR64\" id=\"ref-link-section-d3800159e2811\">64</a></sup>. Here, the kNN graph (<i>k</i>\\u2009=\\u2009100) was computed using euclidean distances in fate probability space, and enrichment scores were visualized on the circular projection. Otherwise, the method was performed as described above.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec32\">Immunofluorescence</h3><p>Organoid tissues were fixed in paraformaldehyde at 4\\u2009°C overnight followed by washing in PBS three times for 10\\u2009min. Tissues were then allowed to sink in 30% sucrose overnight, followed by embedding in O.C.T. compound (Sakura, catalogue no. 4583). Tissues were frozen on dry ice and cryosectioned at 20\\u2009μm. For staining, sections were first blocked and permeabilized in 0.1% Triton X-100 in PBS (0.1%\\xa0PBTx) with 4% normal donkey serum. Sections were then stained with primary and secondary antibodies diluted in 0.1% PBTx with 4% normal donkey serum. Sections were washed in PBS three times for 10\\u2009min after each antibody staining and mounted in DAKO fluorescent mounting medium (Agilent Technologies, catalogue no. S3023). The following antibodies were used in this study: DLX2 (Santa Cruz, catalogue no. SC393879, 1:100); OLIG2 (Abcam, catalogue no. ab109186, 1:100); SOX2 (R&amp;D, catalogue no. MAB2018, 1:500); FOXG1 (Abcam, catalogue no. ab18259, 1:200); EOMES (R&amp;D, catalogue no. AF6166, 1: 200); ARID1B (Cell Signaling, catalogue no. 92964, 1:100); ADNP (ThermoFisher, catalogue no. 702911, 1:250); BCL11A (Abcam, catalogue no. 191401, 1:250); PHF3 (Sigma, catalogue no. HPA024678, 1:250); SMARCC2 (ThermoFisher, catalogue no. PA5-54351, 1:250); KMT2C (Sigma, catalogue no. HPA074736, 1:250); Alexa 488, 568 and 647 conjugated secondary anti-bodies (ThermoFisher, 1:250); and Hoechst (ThermoFisher, catalogue no. H3569, 1:10,000).</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec33\">Microscopy, image processing and quantification</h3><p>Tissue sections were imaged using an Olympus IX3 Series inverted microscope equipped with a dual-camera Yokogawa W1 spinning disk. Images were acquired with 10×\\u20090.75 (air) working distance (WD) 0.6\\u2009mm or 40×\\u20090.75 (air) WD 0.5\\u2009mm objectives and produced by the Cellsense software.</p><p>For DLX2 and OLIG2 quantification in Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig4\">4</a>, images were processed and quantified using Fiji. Based on the size of the tissue, 5–12 regions from each organoid were selected using the Hoechst channel. In total, <i>n</i>\\u2009=\\u2009108 areas (13 organoids from four batches) from the <i>ARID1B</i> control group (c.2201dupG repair), <i>n</i>\\u2009=\\u2009104 areas (15 organoids from four batches) from the <i>ARID1B</i><sup>+/−</sup> (c.2201dupG) group and <i>n</i>\\u2009=\\u200994 areas (15 organoids from three batches) from the <i>ARID1B</i><sup>+/−</sup> (6q25.3del) group are collected and subjected to an automatic segmentation using a Fiji macro. Both DLX2 and OLIG2 channels are used to define the cell body area, followed by the intensity measurement. Area mean intensity was used for setting up the threshold. For protein expression quantification in Extended Data Fig. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"figure anchor\" href=\"/articles/s41586-023-06473-y#Fig6\">2</a>, organoids with individual gene perturbations costained for each gene were processed and quantified using Fiji. Five to fourteen cortical plate regions were analysed per gene. Areas containing both uninduced (dTomato<sup>−</sup>) as well as induced (dTomato<sup>+</sup>) cells were selected and subjected to an automated segmentation using a Fiji macro. The Hoechst channel is used to define the cell body area, followed by intensity measurement. Detected cells were separated into wild-type and perturbed cells by setting up a threshold of mean intensity in the dTomato channel. Additionally, KMT2C protein expression was compared between wild-type (dTomato<sup>−</sup>) and mutant (dTomato<sup>+</sup>) VZ area. VZs were individually outlined, and mean dTomato as well as KMT2C intensities were measured. For IPC abundance analysis, organoids with individual gene perturbations were costained for EOMES. Mutant columns expressing dTomato were individually segmented, and EOMES<sup>+</sup> cells were identified by setting a threshold for EOMES intensity. The number of EOMES<sup>+</sup> cells was normalized to the total number of cells. Percentages of EOMES<sup>+</sup> cells were compared between individual gene perturbations and non-targeting gRNA control groups. For INP abundance analysis, organoids were costained with DLX2. A Fiji macro for automated segmentation was used to identify DLX2<sup>+</sup> cells throughout the entire tissue. Areas containing multiple rosettes from each organoid were collected for quantification. The number of DLX2<sup>+</sup> cells was normalized to the tissue area and compared between individual gene perturbations and non-targeting gRNA control groups.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec34\">Patient sample collection</h3><p>The study was approved by the local ethics committee of the Medical University of Vienna. Study inclusion criteria were as follows: (1) mutation in the <i>ARID1B</i> gene proven by whole-exome sequencing, (2) age between 0 and 18\\u2009years old, (3) continuous follow-up at the Vienna General Hospital and (4) availability of fetal brain MRI data. After informed consent, 10\\u2009ml of blood was collected from two selected patients for iPS\\xa0cell reprogramming.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec35\">Reprogramming of PBMCs into iPS\\xa0cells</h3><p>iPS\\xa0cells were generated from peripheral blood mononuclear cells (PBMCs) isolated from patient blood samples as previously described<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 75\" title=\"Agu, C. A. et al. Successful generation of human induced pluripotent stem cell lines from blood samples held at room temperature for up to 48 hr. Stem Cell Rep. 5, 660–671 (2015).\" href=\"/articles/s41586-023-06473-y#ref-CR75\" id=\"ref-link-section-d3800159e2904\">75</a></sup>. In brief, 10\\u2009ml blood was collected in sodium citrate collection tubes. PBMCs were isolated via a Ficoll–Paque density gradient, and erythroblasts were expanded for 9\\u2009days. Erythroblast-enriched populations were infected with Sendai Vectors expressing human OCT3/4, SOX2, KLF4 and cMYC (CytoTune; Life Technologies, A1377801). Three days after infection, cells were switched to mouse embryonic fibroblast feeder layers. Five days after infection, the medium was changed to iPS\\xa0cell medium (KoSR\\u2009+\\u2009FGF2). Ten to 21\\u2009days after infection, the transduced cells began to form colonies that exhibited iPS\\xa0cell morphology. iPS\\xa0cell colonies were picked and passaged every 5–7\\u2009days after transfer to the mTeSR culture system (Stemcell Technologies).</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec36\">Generation of isogenic control cell line for patient\\xa01</h3><p>Isogenic control cell lines for patient 1 were generated using CRISPR–Cas9. <i>Streptococcus\\xa0pyogenes</i> Cas9 protein with two nuclear localization signals was purified as previously described<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 76\" title=\"Jinek, M. et al. A programmable dual-RNA–guided DNA endonuclease in adaptive bacterial immunity. Science 337, 816–821 (2012).\" href=\"/articles/s41586-023-06473-y#ref-CR76\" id=\"ref-link-section-d3800159e2919\">76</a></sup>. gRNA transcription was performed with the HiScribe T7 High Yield RNA Synthesis Kit (NEB) according to the manufacturer’s protocol, and gRNAs were purified via phenol:chloroform:isoamyl alcohol (25:24:1; Applichem) extraction followed by ethanol precipitation. The homology-directed repair (HDR) template (custom single-stranded oligodeoxynucleotides; Integrated DNA Technologies) was designed to span 100\\u2009base\\xa0pairs up- and downstream of the mutation site. iPS\\xa0cells had been grown in mTeSR for 14 passages before the procedure. For generation of isogenic control cell lines, cells were washed with DPBS<sup>−/−</sup> and incubated for 5\\u2009min at 37\\u2009°C with 1\\u2009ml of accutase solution (Sigma-Aldrich, A6964-500ML). The plate was gently tapped to detach cells, and cells were gently pipetted to generate a single-cell suspension, pelleted by spinning at 200<i>g</i> for 3\\u2009min and counted using Trypan Blue solution (ThermoFisher Scientific). For nucleofection, 1.0\\u2009×\\u200910<sup>6</sup> cells were spun down and resuspended in Buffer R of the Neon Transfection System (ThermoFisher Scientific) at a concentration of 2\\u2009×\\u200910<sup>7</sup>\\u2009cells per millilitre. Twelve nanograms of sgRNA and 5\\u2009ng of Cas9 protein were combined in resuspension buffer to form the Cas9–sgRNA ribonucleoprotein complex. The reaction was mixed and incubated at 37\\u2009°C for 5\\u2009min. Five microliters of the HDR template (100\\u2009μM) were added to the Cas9–sgRNA ribonucleoprotein complex and combined with the cell suspension. Electroporations were performed using a Neon Transfection System (ThermoFisher Scientific) with 100\\u2009μl Neon Pipette Tips using the embryonic stem cells electroporation protocol (1,400\\u2009V, 10\\u2009ms, three pulses). Cells were seeded in one matrigel-coated well of a six-well plate in mTeSR. After a recovery period of 3\\u2009days, a single-cell suspension was generated, and cells were split into another well of a six-well plate for banking and sparsely into two 10-cm dishes for colony formation from single cells. After colony growth for 1\\u2009week, individual colonies were picked and seeded each into one well of a 96-well plate. After colony expansion, gDNA was extracted using DNA QuickExtract Solution (Lucigen), followed by PCR and Sanger sequencing to determine efficient repair of the mutation.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec37\">Fetal MRI and 3D reconstruction</h3><p>Women with singleton pregnancies undergoing fetal MRI at a tertiary care centre from January 2016 to December 2021 were retrospectively reviewed. This study was approved by the institutional ethics board, and all examinations were clinically indicated. A retrospective review of patient records was performed, and a patient with a positive genetic testing report for <i>ARID1B</i> mutation was selected. The participant was included in further analysis, and the gestational age (given in gestational weeks and days postmenstruation) was determined by first-trimester ultrasound. High-quality super-resolution reconstruction was obtained<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 13\" title=\"Michels, B. E. et al. Pooled in vitro and in vivo CRISPR-Cas9 screening identifies tumor suppressors in human colon organoids. Cell Stem Cell 26, 782–792.e7 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR13\" id=\"ref-link-section-d3800159e2944\">13</a></sup>. Age-matched control subjects were identified and included if they presented an absence of confounding comorbidities, including structural cerebral or cardiac anomalies or fetal growth restriction.</p><p>Fetal MRI scans were conducted using 1.5-T (Philips Ingenia/Intera) and 3-T magnets (Philips Achieva). The mother was examined in a supine position or if necessary, left recumbent to achieve sufficient imaging quality. The examinations were performed within 45\\u2009min, neither sedation nor MRI contrast medium was applied, and both the fetal head and body were imaged. Fetal brain imaging included T2-weighted sequences in three orthogonal planes (slice thickness\\u2009=\\u20093–4\\u2009mm, echo time\\u2009=\\u2009140\\u2009ms, field of view\\u2009=\\u2009230\\u2009mm) of the fetal head. Postprocessing was conducted as previously described<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 77\" title=\"Gholipour, A. et al. A normative spatiotemporal MRI atlas of the fetal brain for automatic segmentation and analysis of early brain growth. Sci. Rep. 7, 476 (2017).\" href=\"/articles/s41586-023-06473-y#ref-CR77\" id=\"ref-link-section-d3800159e2951\">77</a></sup>. Superresolution imaging was generated using a volumetric superresolution algorithm<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 77\" title=\"Gholipour, A. et al. A normative spatiotemporal MRI atlas of the fetal brain for automatic segmentation and analysis of early brain growth. Sci. Rep. 7, 476 (2017).\" href=\"/articles/s41586-023-06473-y#ref-CR77\" id=\"ref-link-section-d3800159e2955\">77</a></sup>. The resulting superresolution data were quality assessed, and only cases that met high-quality standards (score of less than or equal to two of five) were included in the analysis. Atlas-based segmentation was performed for the fetal cortex and total brain volume by nonrigid mapping of a publicly available spatiotemporal, anatomical fetal brain atlas for each investigated case<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 77\" title=\"Gholipour, A. et al. A normative spatiotemporal MRI atlas of the fetal brain for automatic segmentation and analysis of early brain growth. Sci. Rep. 7, 476 (2017).\" href=\"/articles/s41586-023-06473-y#ref-CR77\" id=\"ref-link-section-d3800159e2959\">77</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 78\" title=\"Yushkevich, P. A. et al. User-guided 3D active contour segmentation of anatomical structures: significantly improved efficiency and reliability. Neuroimage 31, 1116–1128 (2006).\" href=\"/articles/s41586-023-06473-y#ref-CR78\" id=\"ref-link-section-d3800159e2962\">78</a></sup>. Segmentation of the GE was performed manually using the open-source application ITK-SNAP<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 79\" title=\"Ebner, M. et al. An automated framework for localization, segmentation and super-resolution reconstruction of fetal brain MRI. Neuroimage 206, 116324 (2020).\" href=\"/articles/s41586-023-06473-y#ref-CR79\" id=\"ref-link-section-d3800159e2966\">79</a></sup>. To delineate the T2-weighted hypointense GE, histological fetal atlases by Bayer and Altman<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 80\" title=\"Bayer, S. A. &amp; Altman, J. The Human Brain During the Second Trimester (1st ed.) &#xA;                  https://doi.org/10.1201/9780203507483&#xA;                  &#xA;                 (CRC, 2005).\" href=\"/articles/s41586-023-06473-y#ref-CR80\" id=\"ref-link-section-d3800159e2970\">80</a>,<a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 81\" title=\"Bayer, S. A. &amp; Altman, J. The Human Brain During the Third Trimester (1st ed.) &#xA;                  https://doi.org/10.1201/9780203494943&#xA;                  &#xA;                \\xa0(CRC, 2003).\" href=\"/articles/s41586-023-06473-y#ref-CR81\" id=\"ref-link-section-d3800159e2973\">81</a></sup> were used as a reference guide. Volumetric data were generated and calculations for the GE were made based on the investigated gestational ages.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec38\">Statistics</h3><p>Information on the statistical analyses used is described in each method section.\\xa0\\xa0No statistical methods were used to predetermine sample size unless specified. No blinding and randomization were used unless specified.</p><h3 class=\"c-article__sub-heading c-article__sub-heading--divider\" id=\"Sec39\">Reporting summary</h3><p>Further information on research design is available in the\\xa0<a data-track=\"click\" data-track-label=\"link\" data-track-action=\"supplementary material anchor\" href=\"/articles/s41586-023-06473-y#MOESM2\">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>\\n                </div>\\n            \\n\\n            <div>\\n                <section data-title=\"Data availability\"><div class=\"c-article-section\" id=\"data-availability-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"data-availability\">Data availability</h2><div class=\"c-article-section__content\" id=\"data-availability-content\">\\n              \\n              <p>Raw sequencing datasets were deposited into ArrayExpress with the following accession codes: single-cell RNA sequencing and associated amplicon (<a href=\"http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-13148/\">E-MTAB-13148</a>), bulk genomic DNA-derived amplicon (<a href=\"http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-13140/\">E-MTAB-13140</a>) and single-cell mutiomics (<a href=\"http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-13144/\">E-MTAB-13144</a>). <i>ARID1B</i> cell line genotype data were deposited into the European Genome-Phenome Archive (<a href=\"https://ega-archive.org/studies/EGAS00001007381\">EGAS00001007381</a>). Processed Seurat objects were deposited into Zenodo (<a href=\"https://zenodo.org/record/7083558\">https://zenodo.org/record/7083558</a>).</p>\\n            </div></div></section><section data-title=\"Code availability\"><div class=\"c-article-section\" id=\"code-availability-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"code-availability\">Code availability</h2><div class=\"c-article-section__content\" id=\"code-availability-content\">\\n              \\n              <p>The Pando R package is available on GitHub (<a href=\"https://github.com/quadbiolab/Pando\">https://github.com/quadbiolab/Pando</a>). Other custom code used in the analyses has been deposited on GitHub (<a href=\"https://github.com/quadbiolab/ASD_CHOOSE\">https://github.com/quadbiolab/ASD_CHOOSE</a>).</p>\\n            </div></div></section><section data-title=\"Change history\"><div class=\"c-article-section\" id=\"change-history-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"change-history\">Change history</h2><div class=\"c-article-section__content\" id=\"change-history-content\"><ul class=\"c-article-change-list\"><li class=\"c-article-change-list__item u-mb-24\" id=\"chg1\"><ins datetime=\"2023-11-14\"><h3 class=\"c-article-change-list__heading u-h3 u-pr-8 u-display-inline\">14 November 2023</h3><div class=\"c-article-change-list__details\"><p>A Correction to this paper has been published: <a href=\"https://doi.org/10.1038/s41586-023-06836-5\">https://doi.org/10.1038/s41586-023-06836-5</a></p></div></ins></li></ul></div></div></section><div id=\"MagazineFulltextArticleBodySuffix\"><section aria-labelledby=\"Bib1\" data-title=\"References\"><div class=\"c-article-section\" id=\"Bib1-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Bib1\">References</h2><div class=\"c-article-section__content\" id=\"Bib1-content\"><div data-container-section=\"references\"><ol class=\"c-article-references\" data-track-component=\"outbound reference\"><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"1.\"><p class=\"c-article-references__text\" id=\"ref-CR1\">Hu, W. F., Chahrour, M. H. &amp; Walsh, C. A. The diverse genetic landscape of neurodevelopmental disorders. <i>Genom. Hum. Genet.</i> <b>15</b>, 195–213 (2014).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1146/annurev-genom-090413-025600\" data-track-action=\"article reference\" href=\"https://doi.org/10.1146%2Fannurev-genom-090413-025600\" aria-label=\"Article reference 1\" data-doi=\"10.1146/annurev-genom-090413-025600\">Article</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 1\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20diverse%20genetic%20landscape%20of%20neurodevelopmental%20disorders&amp;journal=Genom.%20Hum.%20Genet.&amp;doi=10.1146%2Fannurev-genom-090413-025600&amp;volume=15&amp;pages=195-213&amp;publication_year=2014&amp;author=Hu%2CWF&amp;author=Chahrour%2CMH&amp;author=Walsh%2CCA\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"2.\"><p class=\"c-article-references__text\" id=\"ref-CR2\">Klingler, E., Francis, F., Jabaudon, D. &amp; Cappello, S. Mapping the molecular and cellular complexity of cortical malformations. <i>Science</i> <b>371</b>, eaba4517 (2021).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1126/science.aba4517\" data-track-action=\"article reference\" href=\"https://doi.org/10.1126%2Fscience.aba4517\" aria-label=\"Article reference 2\" data-doi=\"10.1126/science.aba4517\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1yltr8%3D\" aria-label=\"CAS reference 2\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33479124\" aria-label=\"PubMed reference 2\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 2\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Mapping%20the%20molecular%20and%20cellular%20complexity%20of%20cortical%20malformations&amp;journal=Science&amp;doi=10.1126%2Fscience.aba4517&amp;volume=371&amp;publication_year=2021&amp;author=Klingler%2CE&amp;author=Francis%2CF&amp;author=Jabaudon%2CD&amp;author=Cappello%2CS\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"3.\"><p class=\"c-article-references__text\" id=\"ref-CR3\">Libé-Philippot, B. &amp; Vanderhaeghen, P. Cellular and molecular mechanisms linking human cortical development and evolution. <i>Annu. Rev. Genet.</i> <b>55</b>, 555–581 (2021).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1146/annurev-genet-071719-020705\" data-track-action=\"article reference\" href=\"https://doi.org/10.1146%2Fannurev-genet-071719-020705\" aria-label=\"Article reference 3\" data-doi=\"10.1146/annurev-genet-071719-020705\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34535062\" aria-label=\"PubMed reference 3\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 3\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Cellular%20and%20molecular%20mechanisms%20linking%20human%20cortical%20development%20and%20evolution&amp;journal=Annu.%20Rev.%20Genet.&amp;doi=10.1146%2Fannurev-genet-071719-020705&amp;volume=55&amp;pages=555-581&amp;publication_year=2021&amp;author=Lib%C3%A9-Philippot%2CB&amp;author=Vanderhaeghen%2CP\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"4.\"><p class=\"c-article-references__text\" id=\"ref-CR4\">Lui, J. H., Hansen, D. V. &amp; Kriegstein, A. R. Development and evolution of the human neocortex. <i>Cell</i> <b>146</b>, 18–36 (2011).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.cell.2011.06.030\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.cell.2011.06.030\" aria-label=\"Article reference 4\" data-doi=\"10.1016/j.cell.2011.06.030\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3MXoslCksrg%3D\" aria-label=\"CAS reference 4\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21729779\" aria-label=\"PubMed reference 4\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3610574\" aria-label=\"PubMed Central reference 4\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 4\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Development%20and%20evolution%20of%20the%20human%20neocortex&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2011.06.030&amp;volume=146&amp;pages=18-36&amp;publication_year=2011&amp;author=Lui%2CJH&amp;author=Hansen%2CDV&amp;author=Kriegstein%2CAR\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"5.\"><p class=\"c-article-references__text\" id=\"ref-CR5\">Willsey, A. J. et al. Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism. <i>Cell</i> <b>155</b>, 997–1007 (2013).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.cell.2013.10.020\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.cell.2013.10.020\" aria-label=\"Article reference 5\" data-doi=\"10.1016/j.cell.2013.10.020\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFWis7zP\" aria-label=\"CAS reference 5\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24267886\" aria-label=\"PubMed reference 5\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3995413\" aria-label=\"PubMed Central reference 5\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 5\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Coexpression%20networks%20implicate%20human%20midfetal%20deep%20cortical%20projection%20neurons%20in%20the%20pathogenesis%20of%20autism&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2013.10.020&amp;volume=155&amp;pages=997-1007&amp;publication_year=2013&amp;author=Willsey%2CAJ\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"6.\"><p class=\"c-article-references__text\" id=\"ref-CR6\">Parikshak, N. N. et al. Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism. <i>Cell</i> <b>155</b>, 1008–1021 (2013).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.cell.2013.10.031\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.cell.2013.10.031\" aria-label=\"Article reference 6\" data-doi=\"10.1016/j.cell.2013.10.031\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFWjt73N\" aria-label=\"CAS reference 6\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24267887\" aria-label=\"PubMed reference 6\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3934107\" aria-label=\"PubMed Central reference 6\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 6\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Integrative%20functional%20genomic%20analyses%20implicate%20specific%20molecular%20pathways%20and%20circuits%20in%20autism&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2013.10.031&amp;volume=155&amp;pages=1008-1021&amp;publication_year=2013&amp;author=Parikshak%2CNN\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"7.\"><p class=\"c-article-references__text\" id=\"ref-CR7\">Satterstrom, F. K. et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. <i>Cell</i> <b>180</b>, 568–584.e23 (2020).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.cell.2019.12.036\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.cell.2019.12.036\" aria-label=\"Article reference 7\" data-doi=\"10.1016/j.cell.2019.12.036\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvVShsr8%3D\" aria-label=\"CAS reference 7\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31981491\" aria-label=\"PubMed reference 7\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7250485\" aria-label=\"PubMed Central reference 7\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 7\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20exome%20sequencing%20study%20implicates%20both%20developmental%20and%20functional%20changes%20in%20the%20neurobiology%20of%20autism&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.12.036&amp;volume=180&amp;pages=568-584.e23&amp;publication_year=2020&amp;author=Satterstrom%2CFK\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"8.\"><p class=\"c-article-references__text\" id=\"ref-CR8\">Torre-Ubieta, L., de la Won, H., Stein, J. L. &amp; Geschwind, D. H. Advancing the understanding of autism disease mechanisms through genetics. <i>Nat. Med.</i> <b>22</b>, 345–361 (2016).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nm.4071\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnm.4071\" aria-label=\"Article reference 8\" data-doi=\"10.1038/nm.4071\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27050589\" aria-label=\"PubMed reference 8\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5072455\" aria-label=\"PubMed Central reference 8\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 8\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Advancing%20the%20understanding%20of%20autism%20disease%20mechanisms%20through%20genetics&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fnm.4071&amp;volume=22&amp;pages=345-361&amp;publication_year=2016&amp;author=Torre-Ubieta%2CL&amp;author=Won%2CH&amp;author=Stein%2CJL&amp;author=Geschwind%2CDH\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"9.\"><p class=\"c-article-references__text\" id=\"ref-CR9\">Lancaster, M. A. et al. Cerebral organoids model human brain development and microcephaly. <i>Nature</i> <b>501</b>, 373–379 (2013).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nature12517\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnature12517\" aria-label=\"Article reference 9\" data-doi=\"10.1038/nature12517\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"ads reference\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013Natur.501..373L\" aria-label=\"ADS reference 9\">ADS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtlCntb3F\" aria-label=\"CAS reference 9\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23995685\" aria-label=\"PubMed reference 9\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 9\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Cerebral%20organoids%20model%20human%20brain%20development%20and%20microcephaly&amp;journal=Nature&amp;doi=10.1038%2Fnature12517&amp;volume=501&amp;pages=373-379&amp;publication_year=2013&amp;author=Lancaster%2CMA\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"10.\"><p class=\"c-article-references__text\" id=\"ref-CR10\">Mariani, J. et al. FOXG1-dependent dysregulation of GABA/glutamate neuron differentiation in autism spectrum disorders. <i>Cell</i> <b>162</b>, 375–390 (2015).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.cell.2015.06.034\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.cell.2015.06.034\" aria-label=\"Article reference 10\" data-doi=\"10.1016/j.cell.2015.06.034\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1KgtLrN\" aria-label=\"CAS reference 10\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26186191\" aria-label=\"PubMed reference 10\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4519016\" aria-label=\"PubMed Central reference 10\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 10\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=FOXG1-dependent%20dysregulation%20of%20GABA%2Fglutamate%20neuron%20differentiation%20in%20autism%20spectrum%20disorders&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2015.06.034&amp;volume=162&amp;pages=375-390&amp;publication_year=2015&amp;author=Mariani%2CJ\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"11.\"><p class=\"c-article-references__text\" id=\"ref-CR11\">Paulsen, B. et al. Autism genes converge on asynchronous development of shared neuron classes. <i>Nature</i> <b>602</b>, 268–273 (2022).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41586-021-04358-6\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41586-021-04358-6\" aria-label=\"Article reference 11\" data-doi=\"10.1038/s41586-021-04358-6\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"ads reference\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022Natur.602..268P\" aria-label=\"ADS reference 11\">ADS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB38Xisleltr0%3D\" aria-label=\"CAS reference 11\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35110736\" aria-label=\"PubMed reference 11\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8852827\" aria-label=\"PubMed Central reference 11\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 11\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Autism%20genes%20converge%20on%20asynchronous%20development%20of%20shared%20neuron%20classes&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-04358-6&amp;volume=602&amp;pages=268-273&amp;publication_year=2022&amp;author=Paulsen%2CB\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"12.\"><p class=\"c-article-references__text\" id=\"ref-CR12\">Esk, C. et al. A human tissue screen identifies a regulator of ER secretion as a brain-size determinant. <i>Science</i> <b>370</b>, 935–941 (2020).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1126/science.abb5390\" data-track-action=\"article reference\" href=\"https://doi.org/10.1126%2Fscience.abb5390\" aria-label=\"Article reference 12\" data-doi=\"10.1126/science.abb5390\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"ads reference\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Sci...370..935E\" aria-label=\"ADS reference 12\">ADS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVWjt7%2FI\" aria-label=\"CAS reference 12\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33122427\" aria-label=\"PubMed reference 12\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 12\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=A%20human%20tissue%20screen%20identifies%20a%20regulator%20of%20ER%20secretion%20as%20a%20brain-size%20determinant&amp;journal=Science&amp;doi=10.1126%2Fscience.abb5390&amp;volume=370&amp;pages=935-941&amp;publication_year=2020&amp;author=Esk%2CC\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"13.\"><p class=\"c-article-references__text\" id=\"ref-CR13\">Michels, B. E. et al. Pooled in vitro and in vivo CRISPR-Cas9 screening identifies tumor suppressors in human colon organoids. <i>Cell Stem Cell</i> <b>26</b>, 782–792.e7 (2020).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.stem.2020.04.003\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.stem.2020.04.003\" aria-label=\"Article reference 13\" data-doi=\"10.1016/j.stem.2020.04.003\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXosVWjur8%3D\" aria-label=\"CAS reference 13\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32348727\" aria-label=\"PubMed reference 13\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 13\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Pooled%20in%20vitro%20and%20in%20vivo%20CRISPR-Cas9%20screening%20identifies%20tumor%20suppressors%20in%20human%20colon%20organoids&amp;journal=Cell%20Stem%20Cell&amp;doi=10.1016%2Fj.stem.2020.04.003&amp;volume=26&amp;pages=782-792.e7&amp;publication_year=2020&amp;author=Michels%2CBE\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"14.\"><p class=\"c-article-references__text\" id=\"ref-CR14\">Dixit, A. et al. Perturb-seq: dissecting molecular circuits with scalable single-cell RNA profiling of pooled genetic screens. <i>Cell</i> <b>167</b>, 1853–1866.e17 (2016).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.cell.2016.11.038\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.cell.2016.11.038\" aria-label=\"Article reference 14\" data-doi=\"10.1016/j.cell.2016.11.038\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28XitFWlsrvI\" aria-label=\"CAS reference 14\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27984732\" aria-label=\"PubMed reference 14\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5181115\" aria-label=\"PubMed Central reference 14\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 14\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Perturb-seq%3A%20dissecting%20molecular%20circuits%20with%20scalable%20single-cell%20RNA&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2016.11.038&amp;volume=167&amp;pages=1853-1866.e17&amp;publication_year=2016&amp;author=Dixit%2CA\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"15.\"><p class=\"c-article-references__text\" id=\"ref-CR15\">Jaitin, D. A. et al. Dissecting immune circuits by linking CRISPR-pooled screens with single-cell RNA-seq. <i>Cell</i> <b>167</b>, 1883–1896.e15 (2016).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.cell.2016.11.039\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.cell.2016.11.039\" aria-label=\"Article reference 15\" data-doi=\"10.1016/j.cell.2016.11.039\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28XitFWlsrvP\" aria-label=\"CAS reference 15\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27984734\" aria-label=\"PubMed reference 15\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 15\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Dissecting%20immune%20circuits%20by%20linking%20CRISPR-pooled%20screens%20with%20single-cell%20RNA-seq&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2016.11.039&amp;volume=167&amp;pages=1883-1896.e15&amp;publication_year=2016&amp;author=Jaitin%2CDA\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"16.\"><p class=\"c-article-references__text\" id=\"ref-CR16\">Adamson, B. et al. A multiplexed single-cell CRISPR screening platform enables systematic dissection of the unfolded protein response. <i>Cell</i> <b>167</b>, 1867–1882.e21 (2016).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.cell.2016.11.048\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.cell.2016.11.048\" aria-label=\"Article reference 16\" data-doi=\"10.1016/j.cell.2016.11.048\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28XitFWlsrjK\" aria-label=\"CAS reference 16\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27984733\" aria-label=\"PubMed reference 16\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5315571\" aria-label=\"PubMed Central reference 16\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 16\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=A%20multiplexed%20single-cell%20CRISPR%20screening%20platform%20enables%20systematic%20dissection%20of%20the%20unfolded%20protein%20response&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2016.11.048&amp;volume=167&amp;pages=1867-1882.e21&amp;publication_year=2016&amp;author=Adamson%2CB\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"17.\"><p class=\"c-article-references__text\" id=\"ref-CR17\">Datlinger, P. et al. Pooled CRISPR screening with single-cell transcriptome read-out. <i>Nat. Methods</i> <b>14</b>, 297–301 (2017).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nmeth.4177\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnmeth.4177\" aria-label=\"Article reference 17\" data-doi=\"10.1038/nmeth.4177\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlKjsrk%3D\" aria-label=\"CAS reference 17\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28099430\" aria-label=\"PubMed reference 17\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5334791\" aria-label=\"PubMed Central reference 17\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 17\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Pooled%20CRISPR%20screening%20with%20single-cell%20transcriptome%20read-out&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.4177&amp;volume=14&amp;pages=297-301&amp;publication_year=2017&amp;author=Datlinger%2CP\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"18.\"><p class=\"c-article-references__text\" id=\"ref-CR18\">Doench, J. G. Am I ready for CRISPR? A user’s guide to genetic screens. <i>Nat. Rev. Genet.</i> <b>19</b>, 67–80 (2018).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nrg.2017.97\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnrg.2017.97\" aria-label=\"Article reference 18\" data-doi=\"10.1038/nrg.2017.97\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFWhu7jL\" aria-label=\"CAS reference 18\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29199283\" aria-label=\"PubMed reference 18\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 18\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Am%20I%20ready%20for%20CRISPR%3F%20A%20user%E2%80%99s%20guide%20to%20genetic%20screens&amp;journal=Nat.%20Rev.%20Genet.&amp;doi=10.1038%2Fnrg.2017.97&amp;volume=19&amp;pages=67-80&amp;publication_year=2018&amp;author=Doench%2CJG\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"19.\"><p class=\"c-article-references__text\" id=\"ref-CR19\">Bizzotto, S. et al. Landmarks of human embryonic development inscribed in somatic mutations. <i>Science</i> <b>371</b>, 1249–1253 (2021).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1126/science.abe1544\" data-track-action=\"article reference\" href=\"https://doi.org/10.1126%2Fscience.abe1544\" aria-label=\"Article reference 19\" data-doi=\"10.1126/science.abe1544\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"ads reference\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021Sci...371.1249B\" aria-label=\"ADS reference 19\">ADS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3MXmvFyqurc%3D\" aria-label=\"CAS reference 19\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33737485\" aria-label=\"PubMed reference 19\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8170505\" aria-label=\"PubMed Central reference 19\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 19\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Landmarks%20of%20human%20embryonic%20development%20inscribed%20in%20somatic%20mutations&amp;journal=Science&amp;doi=10.1126%2Fscience.abe1544&amp;volume=371&amp;pages=1249-1253&amp;publication_year=2021&amp;author=Bizzotto%2CS\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"20.\"><p class=\"c-article-references__text\" id=\"ref-CR20\">Chen, J. A., Peñagarikano, O., Belgard, T. G., Swarup, V. &amp; Geschwind, D. H. The emerging picture of autism spectrum disorder: genetics and pathology. <i>Pathology Mech. Dis.</i> <b>10</b>, 111–144 (2015).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1146/annurev-pathol-012414-040405\" data-track-action=\"article reference\" href=\"https://doi.org/10.1146%2Fannurev-pathol-012414-040405\" aria-label=\"Article reference 20\" data-doi=\"10.1146/annurev-pathol-012414-040405\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2MXptFGms7o%3D\" aria-label=\"CAS reference 20\">CAS</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 20\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20emerging%20picture%20of%20autism%20spectrum%20disorder%3A%20genetics%20and%20pathology&amp;journal=Pathology%20Mech.%20Dis.&amp;doi=10.1146%2Fannurev-pathol-012414-040405&amp;volume=10&amp;pages=111-144&amp;publication_year=2015&amp;author=Chen%2CJA&amp;author=Pe%C3%B1agarikano%2CO&amp;author=Belgard%2CTG&amp;author=Swarup%2CV&amp;author=Geschwind%2CDH\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"21.\"><p class=\"c-article-references__text\" id=\"ref-CR21\">Study, T. D. et al. Synaptic, transcriptional, and chromatin genes disrupted in autism. <i>Nature</i> <b>515</b>, 209–215 (2014).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nature13772\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnature13772\" aria-label=\"Article reference 21\" data-doi=\"10.1038/nature13772\">Article</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 21\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Synaptic%2C%20transcriptional%2C%20and%20chromatin%20genes%20disrupted%20in%20autism&amp;journal=Nature&amp;doi=10.1038%2Fnature13772&amp;volume=515&amp;pages=209-215&amp;publication_year=2014&amp;author=Study%2CTD\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"22.\"><p class=\"c-article-references__text\" id=\"ref-CR22\">Trevino, A. E. et al. Chromatin and gene-regulatory dynamics of the developing human cerebral cortex at single-cell resolution. <i>Cell</i> <b>184</b>, 5053–5069.e23 (2021).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.cell.2021.07.039\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.cell.2021.07.039\" aria-label=\"Article reference 22\" data-doi=\"10.1016/j.cell.2021.07.039\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsl2isLfE\" aria-label=\"CAS reference 22\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34390642\" aria-label=\"PubMed reference 22\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 22\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Chromatin%20and%20gene-regulatory%20dynamics%20of%20the%20developing%20human%20cerebral%20cortex%20at%20single-cell%20resolution&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2021.07.039&amp;volume=184&amp;pages=5053-5069.e23&amp;publication_year=2021&amp;author=Trevino%2CAE\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"23.\"><p class=\"c-article-references__text\" id=\"ref-CR23\">Lancaster, M. A. et al. Guided self-organization and cortical plate formation in human brain organoids. <i>Nat. Biotechnol.</i> <b>35</b>, 659–666 (2017).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nbt.3906\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnbt.3906\" aria-label=\"Article reference 23\" data-doi=\"10.1038/nbt.3906\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2sXpt1Omsr8%3D\" aria-label=\"CAS reference 23\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28562594\" aria-label=\"PubMed reference 23\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5824977\" aria-label=\"PubMed Central reference 23\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 23\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Guided%20self-organization%20and%20cortical%20plate%20formation%20in%20human%20brain%20organoids&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.3906&amp;volume=35&amp;pages=659-666&amp;publication_year=2017&amp;author=Lancaster%2CMA\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"24.\"><p class=\"c-article-references__text\" id=\"ref-CR24\">Eichmüller, O. L. et al. Amplification of human interneuron progenitors promotes brain tumors and neurological defects. <i>Science</i> <b>375</b>, eabf5546 (2022).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1126/science.abf5546\" data-track-action=\"article reference\" href=\"https://doi.org/10.1126%2Fscience.abf5546\" aria-label=\"Article reference 24\" data-doi=\"10.1126/science.abf5546\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35084981\" aria-label=\"PubMed reference 24\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7613689\" aria-label=\"PubMed Central reference 24\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 24\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Amplification%20of%20human%20interneuron%20progenitors%20promotes%20brain%20tumors%20and%20neurological%20defects&amp;journal=Science&amp;doi=10.1126%2Fscience.abf5546&amp;volume=375&amp;publication_year=2022&amp;author=Eichm%C3%BCller%2COL\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"25.\"><p class=\"c-article-references__text\" id=\"ref-CR25\">Tsyporin, J. et al. Transcriptional repression by FEZF2 restricts alternative identities of cortical projection neurons. <i>Cell Rep.</i> <b>35</b>, 109269–109269 (2021).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.celrep.2021.109269\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.celrep.2021.109269\" aria-label=\"Article reference 25\" data-doi=\"10.1016/j.celrep.2021.109269\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlyhtrjL\" aria-label=\"CAS reference 25\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34161768\" aria-label=\"PubMed reference 25\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8327856\" aria-label=\"PubMed Central reference 25\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 25\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Transcriptional%20repression%20by%20FEZF2%20restricts%20alternative%20identities%20of%20cortical%20projection%20neurons&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2021.109269&amp;volume=35&amp;pages=109269-109269&amp;publication_year=2021&amp;author=Tsyporin%2CJ\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"26.\"><p class=\"c-article-references__text\" id=\"ref-CR26\">Zhong, Y. et al. Identification of the genes that are expressed in the upper layers of the neocortex. <i>Cereb. Cortex</i> <b>14</b>, 1144–1152 (2004).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1093/cercor/bhh074\" data-track-action=\"article reference\" href=\"https://doi.org/10.1093%2Fcercor%2Fbhh074\" aria-label=\"Article reference 26\" data-doi=\"10.1093/cercor/bhh074\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15142956\" aria-label=\"PubMed reference 26\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 26\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20the%20genes%20that%20are%20expressed%20in%20the%20upper%20layers%20of%20the%20neocortex&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhh074&amp;volume=14&amp;pages=1144-1152&amp;publication_year=2004&amp;author=Zhong%2CY\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"27.\"><p class=\"c-article-references__text\" id=\"ref-CR27\">Oishi, K. et al. Identity of neocortical layer 4 neurons is specified through correct positioning into the cortex. <i>eLife</i> <b>5</b>, e10907 (2016).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.7554/eLife.10907\" data-track-action=\"article reference\" href=\"https://doi.org/10.7554%2FeLife.10907\" aria-label=\"Article reference 27\" data-doi=\"10.7554/eLife.10907\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26880563\" aria-label=\"PubMed reference 27\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4764574\" aria-label=\"PubMed Central reference 27\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 27\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Identity%20of%20neocortical%20layer%204%20neurons%20is%20specified%20through%20correct%20positioning%20into%20the%20cortex&amp;journal=eLife&amp;doi=10.7554%2FeLife.10907&amp;volume=5&amp;publication_year=2016&amp;author=Oishi%2CK\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"28.\"><p class=\"c-article-references__text\" id=\"ref-CR28\">Shi, Y. et al. Mouse and human share conserved transcriptional programs for interneuron development. <i>Science</i> <b>374</b>, eabj6641 (2021).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1126/science.abj6641\" data-track-action=\"article reference\" href=\"https://doi.org/10.1126%2Fscience.abj6641\" aria-label=\"Article reference 28\" data-doi=\"10.1126/science.abj6641\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3MXislKnsb3N\" aria-label=\"CAS reference 28\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34882453\" aria-label=\"PubMed reference 28\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 28\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Mouse%20and%20human%20share%20conserved%20transcriptional%20programs%20for%20interneuron%20development&amp;journal=Science&amp;doi=10.1126%2Fscience.abj6641&amp;volume=374&amp;publication_year=2021&amp;author=Shi%2CY\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"29.\"><p class=\"c-article-references__text\" id=\"ref-CR29\">Schmitz, M. T. et al. The development and evolution of inhibitory neurons in primate cerebrum. <i>Nature</i> <b>603</b>, 871–877 (2022).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41586-022-04510-w\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41586-022-04510-w\" aria-label=\"Article reference 29\" data-doi=\"10.1038/s41586-022-04510-w\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"ads reference\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022Natur.603..871S\" aria-label=\"ADS reference 29\">ADS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB38XnvFShsL0%3D\" aria-label=\"CAS reference 29\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35322231\" aria-label=\"PubMed reference 29\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8967711\" aria-label=\"PubMed Central reference 29\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 29\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20development%20and%20evolution%20of%20inhibitory%20neurons%20in%20primate%20cerebrum&amp;journal=Nature&amp;doi=10.1038%2Fs41586-022-04510-w&amp;volume=603&amp;pages=871-877&amp;publication_year=2022&amp;author=Schmitz%2CMT\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"30.\"><p class=\"c-article-references__text\" id=\"ref-CR30\">Manno, G. L. et al. RNA velocity of single cells. <i>Nature</i> <b>560</b>, 494–498 (2018).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41586-018-0414-6\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41586-018-0414-6\" aria-label=\"Article reference 30\" data-doi=\"10.1038/s41586-018-0414-6\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"ads reference\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018Natur.560..494L\" aria-label=\"ADS reference 30\">ADS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30089906\" aria-label=\"PubMed reference 30\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6130801\" aria-label=\"PubMed Central reference 30\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 30\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=RNA%20velocity%20of%20single%20cells&amp;journal=Nature&amp;doi=10.1038%2Fs41586-018-0414-6&amp;volume=560&amp;pages=494-498&amp;publication_year=2018&amp;author=Manno%2CGL\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"31.\"><p class=\"c-article-references__text\" id=\"ref-CR31\">Bergen, V., Lange, M., Peidli, S., Wolf, F. A. &amp; Theis, F. J. Generalizing RNA velocity to transient cell states through dynamical modeling. <i>Nat. Biotechnol.</i> <b>38</b>, 1408–1414 (2020).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41587-020-0591-3\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41587-020-0591-3\" aria-label=\"Article reference 31\" data-doi=\"10.1038/s41587-020-0591-3\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFait7%2FE\" aria-label=\"CAS reference 31\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32747759\" aria-label=\"PubMed reference 31\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 31\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Generalizing%20RNA%20velocity%20to%20transient%20cell%20states%20through%20dynamical%20modeling&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-020-0591-3&amp;volume=38&amp;pages=1408-1414&amp;publication_year=2020&amp;author=Bergen%2CV&amp;author=Lange%2CM&amp;author=Peidli%2CS&amp;author=Wolf%2CFA&amp;author=Theis%2CFJ\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"32.\"><p class=\"c-article-references__text\" id=\"ref-CR32\">Braun, E. et al. Comprehensive cell atlas of the first-trimester developing human brain. Preprint at <i>bioRxiv</i> <a href=\"https://doi.org/10.1101/2022.10.24.513487\" data-track=\"click\" data-track-action=\"external reference\" data-track-label=\"10.1101/2022.10.24.513487\">https://doi.org/10.1101/2022.10.24.513487</a> (2022).</p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"33.\"><p class=\"c-article-references__text\" id=\"ref-CR33\">Courchesne, E. et al. The ASD Living Biology: from cell proliferation to clinical phenotype. <i>Mol. Psychiatr.</i> <b>24</b>, 88–107 (2019).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41380-018-0056-y\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41380-018-0056-y\" aria-label=\"Article reference 33\" data-doi=\"10.1038/s41380-018-0056-y\">Article</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 33\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20ASD%20Living%20Biology%3A%20from%20cell%20proliferation%20to%20clinical%20phenotype&amp;journal=Mol.%20Psychiatr.&amp;doi=10.1038%2Fs41380-018-0056-y&amp;volume=24&amp;pages=88-107&amp;publication_year=2019&amp;author=Courchesne%2CE\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"34.\"><p class=\"c-article-references__text\" id=\"ref-CR34\">Velmeshev, D. et al. Single-cell genomics identifies cell type–specific molecular changes in autism. <i>Science</i> <b>364</b>, 685–689 (2019).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1126/science.aav8130\" data-track-action=\"article reference\" href=\"https://doi.org/10.1126%2Fscience.aav8130\" aria-label=\"Article reference 34\" data-doi=\"10.1126/science.aav8130\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"ads reference\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019Sci...364..685V\" aria-label=\"ADS reference 34\">ADS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1MXpvVChtLY%3D\" aria-label=\"CAS reference 34\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31097668\" aria-label=\"PubMed reference 34\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7678724\" aria-label=\"PubMed Central reference 34\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 34\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20genomics%20identifies%20cell%20type%E2%80%93specific%20molecular%20changes%20in%20autism&amp;journal=Science&amp;doi=10.1126%2Fscience.aav8130&amp;volume=364&amp;pages=685-689&amp;publication_year=2019&amp;author=Velmeshev%2CD\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"35.\"><p class=\"c-article-references__text\" id=\"ref-CR35\">Qian, X. et al. Sliced human cortical organoids for modeling distinct cortical layer formation. <i>Cell Stem Cell</i> <b>26</b>, 766–781.e9 (2020).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.stem.2020.02.002\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.stem.2020.02.002\" aria-label=\"Article reference 35\" data-doi=\"10.1016/j.stem.2020.02.002\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXksFaqu7Y%3D\" aria-label=\"CAS reference 35\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32142682\" aria-label=\"PubMed reference 35\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7366517\" aria-label=\"PubMed Central reference 35\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 35\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Sliced%20human%20cortical%20organoids%20for%20modeling%20distinct%20cortical%20layer%20formation&amp;journal=Cell%20Stem%20Cell&amp;doi=10.1016%2Fj.stem.2020.02.002&amp;volume=26&amp;pages=766-781.e9&amp;publication_year=2020&amp;author=Qian%2CX\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"36.\"><p class=\"c-article-references__text\" id=\"ref-CR36\">Nakagawa, T. et al. The autism-related protein SETD5 controls neural cell proliferation through epigenetic regulation of rDNA expression. <i>iScience</i> <b>23</b>, 101030 (2020).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.isci.2020.101030\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.isci.2020.101030\" aria-label=\"Article reference 36\" data-doi=\"10.1016/j.isci.2020.101030\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"ads reference\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020iSci...23j1030N\" aria-label=\"ADS reference 36\">ADS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXnsVSqurw%3D\" aria-label=\"CAS reference 36\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32299058\" aria-label=\"PubMed reference 36\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7160574\" aria-label=\"PubMed Central reference 36\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 36\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20autism-related%20protein%20SETD5%20controls%20neural%20cell%20proliferation%20through%20epigenetic%20regulation%20of%20rDNA%20expression&amp;journal=iScience&amp;doi=10.1016%2Fj.isci.2020.101030&amp;volume=23&amp;publication_year=2020&amp;author=Nakagawa%2CT\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"37.\"><p class=\"c-article-references__text\" id=\"ref-CR37\">Wang, J. et al. Mitochondrial dysfunction and oxidative stress contribute to cognitive and motor impairment in FOXP1 syndrome. <i>Proc. Natl Acad. Sci. USA</i> <b>119</b>, e2112852119 (2022).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1073/pnas.2112852119\" data-track-action=\"article reference\" href=\"https://doi.org/10.1073%2Fpnas.2112852119\" aria-label=\"Article reference 37\" data-doi=\"10.1073/pnas.2112852119\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB38Xmt1eqsbY%3D\" aria-label=\"CAS reference 37\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35165191\" aria-label=\"PubMed reference 37\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8872729\" aria-label=\"PubMed Central reference 37\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 37\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Mitochondrial%20dysfunction%20and%20oxidative%20stress%20contribute%20to%20cognitive%20and%20motor%20impairment%20in%20FOXP1%20syndrome&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2112852119&amp;volume=119&amp;publication_year=2022&amp;author=Wang%2CJ\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"38.\"><p class=\"c-article-references__text\" id=\"ref-CR38\">Chen, H.-H. et al. IRF2BP2 reduces macrophage inflammation and susceptibility to atherosclerosis. <i>Circ. Res.</i> <b>117</b>, 671–683 (2015).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1161/CIRCRESAHA.114.305777\" data-track-action=\"article reference\" href=\"https://doi.org/10.1161%2FCIRCRESAHA.114.305777\" aria-label=\"Article reference 38\" data-doi=\"10.1161/CIRCRESAHA.114.305777\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFCrtbrI\" aria-label=\"CAS reference 38\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26195219\" aria-label=\"PubMed reference 38\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 38\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=IRF2BP2%20reduces%20macrophage%20inflammation%20and%20susceptibility%20to%20atherosclerosis&amp;journal=Circ.%20Res.&amp;doi=10.1161%2FCIRCRESAHA.114.305777&amp;volume=117&amp;pages=671-683&amp;publication_year=2015&amp;author=Chen%2CH-H\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"39.\"><p class=\"c-article-references__text\" id=\"ref-CR39\">Jia, Y.-L. et al. P300/CBP-associated factor (PCAF) inhibits the growth of hepatocellular carcinoma by promoting cell autophagy. <i>Cell Death Dis.</i> <b>7</b>, e2400–e2400 (2016).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/cddis.2016.247\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fcddis.2016.247\" aria-label=\"Article reference 39\" data-doi=\"10.1038/cddis.2016.247\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1Ggu7bL\" aria-label=\"CAS reference 39\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27711074\" aria-label=\"PubMed reference 39\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5133959\" aria-label=\"PubMed Central reference 39\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 39\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=P300%2FCBP-associated%20factor%20%28PCAF%29%20inhibits%20the%20growth%20of%20hepatocellular%20carcinoma%20by%20promoting%20cell%20autophagy&amp;journal=Cell%20Death%20Dis.&amp;doi=10.1038%2Fcddis.2016.247&amp;volume=7&amp;pages=e2400-e2400&amp;publication_year=2016&amp;author=Jia%2CY-L\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"40.\"><p class=\"c-article-references__text\" id=\"ref-CR40\">Frasca, A. et al. MECP2 mutations affect ciliogenesis: a novel perspective for Rett syndrome and related disorders. <i>EMBO Mol. Med.</i> <b>12</b>, e10270 (2020).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.15252/emmm.201910270\" data-track-action=\"article reference\" href=\"https://doi.org/10.15252%2Femmm.201910270\" aria-label=\"Article reference 40\" data-doi=\"10.15252/emmm.201910270\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXoslygs7c%3D\" aria-label=\"CAS reference 40\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32383329\" aria-label=\"PubMed reference 40\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7278541\" aria-label=\"PubMed Central reference 40\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 40\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=MECP2%20mutations%20affect%20ciliogenesis%3A%20a%20novel%20perspective%20for%20Rett%20syndrome%20and%20related%20disorders&amp;journal=EMBO%20Mol.%20Med.&amp;doi=10.15252%2Femmm.201910270&amp;volume=12&amp;publication_year=2020&amp;author=Frasca%2CA\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"41.\"><p class=\"c-article-references__text\" id=\"ref-CR41\">Fleck, J.S., Jansen, S.M.J., Wollny, D. et al. Inferring and perturbing cell fate regulomes in human brain organoids. <i>Nature</i> <a href=\"https://doi.org/10.1038/s41586-022-05279-8\" data-track=\"click\" data-track-action=\"external reference\" data-track-label=\"10.1038/s41586-022-05279-8\">https://doi.org/10.1038/s41586-022-05279-8</a> (2022).</p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"42.\"><p class=\"c-article-references__text\" id=\"ref-CR42\">Becht, E. et al. Dimensionality reduction for visualizing single-cell data using UMAP. <i>Nat. Biotechnol.</i> <b>37</b>, 38–44 (2019).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nbt.4314\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnbt.4314\" aria-label=\"Article reference 42\" data-doi=\"10.1038/nbt.4314\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1cXitl2isbrN\" aria-label=\"CAS reference 42\">CAS</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 42\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Dimensionality%20reduction%20for%20visualizing%20single-cell%20data%20using%20UMAP&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.4314&amp;volume=37&amp;pages=38-44&amp;publication_year=2019&amp;author=Becht%2CE\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"43.\"><p class=\"c-article-references__text\" id=\"ref-CR43\">Lange, M. et al. CellRank for directed single-cell fate mapping. <i>Nat. Methods</i> <b>19</b>, 159–170 (2022).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41592-021-01346-6\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41592-021-01346-6\" aria-label=\"Article reference 43\" data-doi=\"10.1038/s41592-021-01346-6\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1Kqt7g%3D\" aria-label=\"CAS reference 43\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35027767\" aria-label=\"PubMed reference 43\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8828480\" aria-label=\"PubMed Central reference 43\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 43\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=CellRank%20for%20directed%20single-cell%20fate%20mapping&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-021-01346-6&amp;volume=19&amp;pages=159-170&amp;publication_year=2022&amp;author=Lange%2CM\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"44.\"><p class=\"c-article-references__text\" id=\"ref-CR44\">Petryniak, M. A., Potter, G. B., Rowitch, D. H. &amp; Rubenstein, J. L. R. Dlx1 and Dlx2 control neuronal versus oligodendroglial cell fate acquisition in the developing forebrain. <i>Neuron</i> <b>55</b>, 417–433 (2007).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.neuron.2007.06.036\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.neuron.2007.06.036\" aria-label=\"Article reference 44\" data-doi=\"10.1016/j.neuron.2007.06.036\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD2sXpsVGltrk%3D\" aria-label=\"CAS reference 44\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17678855\" aria-label=\"PubMed reference 44\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2039927\" aria-label=\"PubMed Central reference 44\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 44\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Dlx1%20and%20Dlx2%20control%20neuronal%20versus%20oligodendroglial%20cell%20fate%20acquisition%20in%20the%20developing%20forebrain&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2007.06.036&amp;volume=55&amp;pages=417-433&amp;publication_year=2007&amp;author=Petryniak%2CMA&amp;author=Potter%2CGB&amp;author=Rowitch%2CDH&amp;author=Rubenstein%2CJLR\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"45.\"><p class=\"c-article-references__text\" id=\"ref-CR45\">Sun, Y. et al. Phosphorylation state of Olig2 regulates proliferation of neural progenitors. <i>Neuron</i> <b>69</b>, 906–917 (2011).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.neuron.2011.02.005\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.neuron.2011.02.005\" aria-label=\"Article reference 45\" data-doi=\"10.1016/j.neuron.2011.02.005\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3MXjtFeqsb8%3D\" aria-label=\"CAS reference 45\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21382551\" aria-label=\"PubMed reference 45\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3065213\" aria-label=\"PubMed Central reference 45\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 45\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Phosphorylation%20state%20of%20Olig2%20regulates%20proliferation%20of%20neural%20progenitors&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2011.02.005&amp;volume=69&amp;pages=906-917&amp;publication_year=2011&amp;author=Sun%2CY\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"46.\"><p class=\"c-article-references__text\" id=\"ref-CR46\">Moffat, J. J., Smith, A. L., Jung, E.-M., Ka, M. &amp; Kim, W.-Y. Neurobiology of ARID1B haploinsufficiency related to neurodevelopmental and psychiatric disorders. <i>Mol. Psychiatr.</i> <b>27</b>, 476–489 (2022).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41380-021-01060-x\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41380-021-01060-x\" aria-label=\"Article reference 46\" data-doi=\"10.1038/s41380-021-01060-x\">Article</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 46\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Neurobiology%20of%20ARID1B%20haploinsufficiency%20related%20to%20neurodevelopmental%20and%20psychiatric%20disorders&amp;journal=Mol.%20Psychiatr.&amp;doi=10.1038%2Fs41380-021-01060-x&amp;volume=27&amp;pages=476-489&amp;publication_year=2022&amp;author=Moffat%2CJJ&amp;author=Smith%2CAL&amp;author=Jung%2CE-M&amp;author=Ka%2CM&amp;author=Kim%2CW-Y\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"47.\"><p class=\"c-article-references__text\" id=\"ref-CR47\">Bagley, J. A., Reumann, D., Bian, S., Lévi-Strauss, J. &amp; Knoblich, J. A. Fused cerebral organoids model interactions between brain regions. <i>Nat. Methods</i> <b>14</b>, 743–751 (2017).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nmeth.4304\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnmeth.4304\" aria-label=\"Article reference 47\" data-doi=\"10.1038/nmeth.4304\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2sXnslKkt7s%3D\" aria-label=\"CAS reference 47\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28504681\" aria-label=\"PubMed reference 47\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5540177\" aria-label=\"PubMed Central reference 47\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 47\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Fused%20cerebral%20organoids%20model%20interactions%20between%20brain%20regions&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.4304&amp;volume=14&amp;pages=743-751&amp;publication_year=2017&amp;author=Bagley%2CJA&amp;author=Reumann%2CD&amp;author=Bian%2CS&amp;author=L%C3%A9vi-Strauss%2CJ&amp;author=Knoblich%2CJA\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"48.\"><p class=\"c-article-references__text\" id=\"ref-CR48\">Bajaj, S. et al. Neurotransmitter signaling regulates distinct phases of multimodal human interneuron migration. <i>EMBO J.</i> <b>40</b>, e108714 (2021).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.15252/embj.2021108714\" data-track-action=\"article reference\" href=\"https://doi.org/10.15252%2Fembj.2021108714\" aria-label=\"Article reference 48\" data-doi=\"10.15252/embj.2021108714\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1Ohsr%2FK\" aria-label=\"CAS reference 48\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34661293\" aria-label=\"PubMed reference 48\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8634123\" aria-label=\"PubMed Central reference 48\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 48\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Neurotransmitter%20signaling%20regulates%20distinct%20phases%20of%20multimodal%20human%20interneuron%20migration&amp;journal=EMBO%20J.&amp;doi=10.15252%2Fembj.2021108714&amp;volume=40&amp;publication_year=2021&amp;author=Bajaj%2CS\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"49.\"><p class=\"c-article-references__text\" id=\"ref-CR49\">Martínez-Cerdeño, V., Noctor, S. C. &amp; Kriegstein, A. R. The role of intermediate progenitor cells in the evolutionary expansion of the cerebral cortex. <i>Cereb. Cortex</i> <b>16</b>, i152–i161 (2006).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1093/cercor/bhk017\" data-track-action=\"article reference\" href=\"https://doi.org/10.1093%2Fcercor%2Fbhk017\" aria-label=\"Article reference 49\" data-doi=\"10.1093/cercor/bhk017\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16766701\" aria-label=\"PubMed reference 49\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 49\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20intermediate%20progenitor%20cells%20in%20the%20evolutionary%20expansion%20of%20the%20cerebral%20cortex&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhk017&amp;volume=16&amp;pages=i152-i161&amp;publication_year=2006&amp;author=Mart%C3%ADnez-Cerde%C3%B1o%2CV&amp;author=Noctor%2CSC&amp;author=Kriegstein%2CAR\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"50.\"><p class=\"c-article-references__text\" id=\"ref-CR50\">Pebworth, M.-P., Ross, J., Andrews, M., Bhaduri, A. &amp; Kriegstein, A. R. Human intermediate progenitor diversity during cortical development. <i>Proc. Natl Acad. Sci. USA</i> <b>118</b>, e2019415118 (2021).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1073/pnas.2019415118\" data-track-action=\"article reference\" href=\"https://doi.org/10.1073%2Fpnas.2019415118\" aria-label=\"Article reference 50\" data-doi=\"10.1073/pnas.2019415118\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVOisLzN\" aria-label=\"CAS reference 50\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34155100\" aria-label=\"PubMed reference 50\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8256045\" aria-label=\"PubMed Central reference 50\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 50\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Human%20intermediate%20progenitor%20diversity%20during%20cortical%20development&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2019415118&amp;volume=118&amp;publication_year=2021&amp;author=Pebworth%2CM-P&amp;author=Ross%2CJ&amp;author=Andrews%2CM&amp;author=Bhaduri%2CA&amp;author=Kriegstein%2CAR\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"51.\"><p class=\"c-article-references__text\" id=\"ref-CR51\">Kessaris, N. et al. Competing waves of oligodendrocytes in the forebrain and postnatal elimination of an embryonic lineage. <i>Nat. Neurosci.</i> <b>9</b>, 173–179 (2006).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nn1620\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnn1620\" aria-label=\"Article reference 51\" data-doi=\"10.1038/nn1620\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD28XotVegsQ%3D%3D\" aria-label=\"CAS reference 51\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16388308\" aria-label=\"PubMed reference 51\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 51\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Competing%20waves%20of%20oligodendrocytes%20in%20the%20forebrain%20and%20postnatal%20elimination%20of%20an%20embryonic%20lineage&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn1620&amp;volume=9&amp;pages=173-179&amp;publication_year=2006&amp;author=Kessaris%2CN\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"52.\"><p class=\"c-article-references__text\" id=\"ref-CR52\">Sokpor, G., Xie, Y., Rosenbusch, J. &amp; Tuoc, T. Chromatin remodeling BAF (SWI/SNF) complexes in neural development and disorders. <i>Front. Mol. Neurosci.</i> <b>10</b>, 243 (2017).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.3389/fnmol.2017.00243\" data-track-action=\"article reference\" href=\"https://doi.org/10.3389%2Ffnmol.2017.00243\" aria-label=\"Article reference 52\" data-doi=\"10.3389/fnmol.2017.00243\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28824374\" aria-label=\"PubMed reference 52\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5540894\" aria-label=\"PubMed Central reference 52\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 52\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Chromatin%20remodeling%20BAF%20%28SWI%2FSNF%29%20complexes%20in%20neural%20development%20and%20disorders&amp;journal=Front.%20Mol.%20Neurosci.&amp;doi=10.3389%2Ffnmol.2017.00243&amp;volume=10&amp;publication_year=2017&amp;author=Sokpor%2CG&amp;author=Xie%2CY&amp;author=Rosenbusch%2CJ&amp;author=Tuoc%2CT\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"53.\"><p class=\"c-article-references__text\" id=\"ref-CR53\">Villa, C. E. et al. CHD8 haploinsufficiency links autism to transient alterations in excitatory and inhibitory trajectories. <i>Cell Rep.</i> <b>39</b>, 110615 (2022).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.celrep.2022.110615\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.celrep.2022.110615\" aria-label=\"Article reference 53\" data-doi=\"10.1016/j.celrep.2022.110615\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB38XptVCitLg%3D\" aria-label=\"CAS reference 53\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35385734\" aria-label=\"PubMed reference 53\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 53\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=CHD8%20haploinsufficiency%20links%20autism%20to%20transient%20alterations%20in%20excitatory%20and%20inhibitory%20trajectories&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2022.110615&amp;volume=39&amp;publication_year=2022&amp;author=Villa%2CCE\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"54.\"><p class=\"c-article-references__text\" id=\"ref-CR54\">Michlits, G. et al. Multilayered VBC score predicts sgRNAs that efficiently generate loss-of-function alleles. <i>Nat. Methods</i> <b>17</b>, 708–716 (2020).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41592-020-0850-8\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41592-020-0850-8\" aria-label=\"Article reference 54\" data-doi=\"10.1038/s41592-020-0850-8\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFWltb%2FJ\" aria-label=\"CAS reference 54\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32514112\" aria-label=\"PubMed reference 54\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 54\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Multilayered%20VBC%20score%20predicts%20sgRNAs%20that%20efficiently%20generate%20loss-of-function%20alleles&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-020-0850-8&amp;volume=17&amp;pages=708-716&amp;publication_year=2020&amp;author=Michlits%2CG\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"55.\"><p class=\"c-article-references__text\" id=\"ref-CR55\">Sanjana, N. E., Shalem, O. &amp; Zhang, F. Improved vectors and genome-wide libraries for CRISPR screening. <i>Nat. Methods</i> <b>11</b>, 783–784 (2014).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nmeth.3047\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnmeth.3047\" aria-label=\"Article reference 55\" data-doi=\"10.1038/nmeth.3047\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2cXhslelsLjK\" aria-label=\"CAS reference 55\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25075903\" aria-label=\"PubMed reference 55\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4486245\" aria-label=\"PubMed Central reference 55\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 55\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Improved%20vectors%20and%20genome-wide%20libraries%20for%20CRISPR%20screening&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.3047&amp;volume=11&amp;pages=783-784&amp;publication_year=2014&amp;author=Sanjana%2CNE&amp;author=Shalem%2CO&amp;author=Zhang%2CF\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"56.\"><p class=\"c-article-references__text\" id=\"ref-CR56\">Omelina, E. S., Ivankin, A. V., Letiagina, A. E. &amp; Pindyurin, A. V. Optimized PCR conditions minimizing the formation of chimeric DNA molecules from MPRA plasmid libraries. <i>BMC Genom.</i> <b>20</b>, 536 (2019).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"noopener\" data-track-label=\"10.1186/s12864-019-5847-2\" data-track-action=\"article reference\" href=\"https://link.springer.com/doi/10.1186/s12864-019-5847-2\" aria-label=\"Article reference 56\" data-doi=\"10.1186/s12864-019-5847-2\">Article</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 56\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Optimized%20PCR%20conditions%20minimizing%20the%20formation%20of%20chimeric%20DNA%20molecules%20from%20MPRA%20plasmid%20libraries&amp;journal=BMC%20Genom.&amp;doi=10.1186%2Fs12864-019-5847-2&amp;volume=20&amp;publication_year=2019&amp;author=Omelina%2CES&amp;author=Ivankin%2CAV&amp;author=Letiagina%2CAE&amp;author=Pindyurin%2CAV\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"57.\"><p class=\"c-article-references__text\" id=\"ref-CR57\">Williams, R. et al. Amplification of complex gene libraries by emulsion PCR. <i>Nat. Methods</i> <b>3</b>, 545–550 (2006).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nmeth896\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnmeth896\" aria-label=\"Article reference 57\" data-doi=\"10.1038/nmeth896\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD28XmtVSntro%3D\" aria-label=\"CAS reference 57\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16791213\" aria-label=\"PubMed reference 57\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 57\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Amplification%20of%20complex%20gene%20libraries%20by%20emulsion%20PCR&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth896&amp;volume=3&amp;pages=545-550&amp;publication_year=2006&amp;author=Williams%2CR\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"58.\"><p class=\"c-article-references__text\" id=\"ref-CR58\">Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. <i>EMBnet J</i> <b>17</b>, 10–12 (2011).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.14806/ej.17.1.200\" data-track-action=\"article reference\" href=\"https://doi.org/10.14806%2Fej.17.1.200\" aria-label=\"Article reference 58\" data-doi=\"10.14806/ej.17.1.200\">Article</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 58\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;journal=EMBnet%20J&amp;doi=10.14806%2Fej.17.1.200&amp;volume=17&amp;pages=10-12&amp;publication_year=2011&amp;author=Martin%2CM\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"59.\"><p class=\"c-article-references__text\" id=\"ref-CR59\">Zheng, G. X. Y. et al. Massively parallel digital transcriptional profiling of single cells. <i>Nat. Commun.</i> <b>8</b>, 14049 (2017).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/ncomms14049\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fncomms14049\" aria-label=\"Article reference 59\" data-doi=\"10.1038/ncomms14049\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"ads reference\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017NatCo...814049Z\" aria-label=\"ADS reference 59\">ADS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1WlsLo%3D\" aria-label=\"CAS reference 59\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28091601\" aria-label=\"PubMed reference 59\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5241818\" aria-label=\"PubMed Central reference 59\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 59\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Massively%20parallel%20digital%20transcriptional%20profiling%20of%20single%20cells&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms14049&amp;volume=8&amp;publication_year=2017&amp;author=Zheng%2CGXY\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"60.\"><p class=\"c-article-references__text\" id=\"ref-CR60\">Hao, Y. et al. Integrated analysis of multimodal single-cell data. <i>Cell</i> <b>184</b>, 3573–3587.e29 (2021).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.cell.2021.04.048\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.cell.2021.04.048\" aria-label=\"Article reference 60\" data-doi=\"10.1016/j.cell.2021.04.048\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlSrtrvE\" aria-label=\"CAS reference 60\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34062119\" aria-label=\"PubMed reference 60\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8238499\" aria-label=\"PubMed Central reference 60\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 60\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Integrated%20analysis%20of%20multimodal%20single-cell%20data&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2021.04.048&amp;volume=184&amp;pages=3573-3587.e29&amp;publication_year=2021&amp;author=Hao%2CY\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"61.\"><p class=\"c-article-references__text\" id=\"ref-CR61\">Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. <i>Nat. Methods</i> <b>16</b>, 1289–1296 (2019).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41592-019-0619-0\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41592-019-0619-0\" aria-label=\"Article reference 61\" data-doi=\"10.1038/s41592-019-0619-0\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFOqsr7N\" aria-label=\"CAS reference 61\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31740819\" aria-label=\"PubMed reference 61\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6884693\" aria-label=\"PubMed Central reference 61\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 61\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Fast%2C%20sensitive%20and%20accurate%20integration%20of%20single-cell%20data%20with%20Harmony&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0619-0&amp;volume=16&amp;pages=1289-1296&amp;publication_year=2019&amp;author=Korsunsky%2CI\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"62.\"><p class=\"c-article-references__text\" id=\"ref-CR62\">Blondel, V. D., Guillaume, J.-L., Lambiotte, R. &amp; Lefebvre, E. Fast unfolding of communities in large networks. <i>J Statist. Mech. Theory Exp.</i> <b>2008</b>, P10008 (2008).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1088/1742-5468/2008/10/P10008\" data-track-action=\"article reference\" href=\"https://doi.org/10.1088%2F1742-5468%2F2008%2F10%2FP10008\" aria-label=\"Article reference 62\" data-doi=\"10.1088/1742-5468/2008/10/P10008\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"math reference\" href=\"http://www.emis.de/MATH-item?1459.91130\" aria-label=\"MATH reference 62\">MATH</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 62\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Fast%20unfolding%20of%20communities%20in%20large%20networks&amp;journal=J%20Statist.%20Mech.%20Theory%20Exp.&amp;doi=10.1088%2F1742-5468%2F2008%2F10%2FP10008&amp;volume=2008&amp;publication_year=2008&amp;author=Blondel%2CVD&amp;author=Guillaume%2CJ-L&amp;author=Lambiotte%2CR&amp;author=Lefebvre%2CE\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"63.\"><p class=\"c-article-references__text\" id=\"ref-CR63\">Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. <i>Nat. Biotechnol.</i> <b>34</b>, 525–527 (2016).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nbt.3519\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnbt.3519\" aria-label=\"Article reference 63\" data-doi=\"10.1038/nbt.3519\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28XlsVansL8%3D\" aria-label=\"CAS reference 63\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27043002\" aria-label=\"PubMed reference 63\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 63\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Near-optimal%20probabilistic%20RNA-seq%20quantification&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.3519&amp;volume=34&amp;pages=525-527&amp;publication_year=2016&amp;author=Bray%2CNL&amp;author=Pimentel%2CH&amp;author=Melsted%2CP&amp;author=Pachter%2CL\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"64.\"><p class=\"c-article-references__text\" id=\"ref-CR64\">Nikolova, M. T. et al. Fate and state transitions during human blood vessel organoid development. Preprint at <i>bioRxiv</i> <a href=\"https://doi.org/10.1101/2022.03.23.485329\" data-track=\"click\" data-track-action=\"external reference\" data-track-label=\"10.1101/2022.03.23.485329\">https://doi.org/10.1101/2022.03.23.485329</a> (2022).</p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"65.\"><p class=\"c-article-references__text\" id=\"ref-CR65\">Wickham, H. in <i>ggplot2: Elegant Graphics for Data Analysis</i>\\xa0147–168 (Springer, 2016).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"noopener\" data-track-label=\"10.1007/978-3-319-24277-4\" data-track-action=\"book reference\" href=\"https://link.springer.com/doi/10.1007/978-3-319-24277-4\" aria-label=\"Book reference 65\" data-doi=\"10.1007/978-3-319-24277-4\">Book</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"math reference\" href=\"http://www.emis.de/MATH-item?1397.62006\" aria-label=\"MATH reference 65\">MATH</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 65\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=ggplot2%3A%20Elegant%20Graphics%20for%20Data%20Analysis&amp;doi=10.1007%2F978-3-319-24277-4&amp;publication_year=2016&amp;author=Wickham%2C%2CH\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"66.\"><p class=\"c-article-references__text\" id=\"ref-CR66\">Wu, T. et al. clusterProfiler 4.0: a universal enrichment tool for interpreting omics data. <i>Innovation</i> <b>2</b>, 100141 (2021).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB38XitVemt7jI\" aria-label=\"CAS reference 66\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34557778\" aria-label=\"PubMed reference 66\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8454663\" aria-label=\"PubMed Central reference 66\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 66\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=clusterProfiler%204.0%3A%20a%20universal%20enrichment%20tool%20for%20interpreting%20omics%20data&amp;journal=Innovation&amp;volume=2&amp;publication_year=2021&amp;author=Wu%2CT\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"67.\"><p class=\"c-article-references__text\" id=\"ref-CR67\">Stuart, T., Srivastava, A., Madad, S., Lareau, C. A. &amp; Satija, R. Single-cell chromatin state analysis with Signac. <i>Nat. Methods</i> <b>18</b>, 1333–1341 (2021).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41592-021-01282-5\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41592-021-01282-5\" aria-label=\"Article reference 67\" data-doi=\"10.1038/s41592-021-01282-5\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlyksL3L\" aria-label=\"CAS reference 67\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34725479\" aria-label=\"PubMed reference 67\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9255697\" aria-label=\"PubMed Central reference 67\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 67\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20chromatin%20state%20analysis%20with%20Signac&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-021-01282-5&amp;volume=18&amp;pages=1333-1341&amp;publication_year=2021&amp;author=Stuart%2CT&amp;author=Srivastava%2CA&amp;author=Madad%2CS&amp;author=Lareau%2CCA&amp;author=Satija%2CR\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"68.\"><p class=\"c-article-references__text\" id=\"ref-CR68\">Feng, J., Liu, T., Qin, B., Zhang, Y. &amp; Liu, X. S. Identifying ChIP-seq enrichment using MACS. <i>Nat. Protoc.</i> <b>7</b>, 1728–1740 (2012).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nprot.2012.101\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnprot.2012.101\" aria-label=\"Article reference 68\" data-doi=\"10.1038/nprot.2012.101\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1OhsLzF\" aria-label=\"CAS reference 68\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22936215\" aria-label=\"PubMed reference 68\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 68\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Identifying%20ChIP-seq%20enrichment%20using%20MACS&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fnprot.2012.101&amp;volume=7&amp;pages=1728-1740&amp;publication_year=2012&amp;author=Feng%2CJ&amp;author=Liu%2CT&amp;author=Qin%2CB&amp;author=Zhang%2CY&amp;author=Liu%2CXS\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"69.\"><p class=\"c-article-references__text\" id=\"ref-CR69\">Siepel, A. et al. Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. <i>Genome Res.</i> <b>15</b>, 1034–1050 (2005).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1101/gr.3715005\" data-track-action=\"article reference\" href=\"https://doi.org/10.1101%2Fgr.3715005\" aria-label=\"Article reference 69\" data-doi=\"10.1101/gr.3715005\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD2MXotFaht7o%3D\" aria-label=\"CAS reference 69\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16024819\" aria-label=\"PubMed reference 69\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1182216\" aria-label=\"PubMed Central reference 69\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 69\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Evolutionarily%20conserved%20elements%20in%20vertebrate%2C%20insect%2C%20worm%2C%20and%20yeast%20genomes&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.3715005&amp;volume=15&amp;pages=1034-1050&amp;publication_year=2005&amp;author=Siepel%2CA\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"70.\"><p class=\"c-article-references__text\" id=\"ref-CR70\">Consortium, T. E. P. et al. Expanded encyclopaedias of DNA elements in the human and mouse genomes. <i>Nature</i> <b>583</b>, 699–710 (2020).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41586-020-2493-4\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41586-020-2493-4\" aria-label=\"Article reference 70\" data-doi=\"10.1038/s41586-020-2493-4\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"ads reference\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Natur.583..699E\" aria-label=\"ADS reference 70\">ADS</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 70\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Expanded%20encyclopaedias%20of%20DNA%20elements%20in%20the%20human%20and%20mouse%20genomes&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2493-4&amp;volume=583&amp;pages=699-710&amp;publication_year=2020&amp;author=Consortium%2CTEP\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"71.\"><p class=\"c-article-references__text\" id=\"ref-CR71\">McLean, C. Y. et al. GREAT improves functional interpretation of <i>cis</i>-regulatory regions. <i>Nat. Biotechnol.</i> <b>28</b>, 495–501 (2010).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/nbt.1630\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fnbt.1630\" aria-label=\"Article reference 71\" data-doi=\"10.1038/nbt.1630\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3cXlsVyhs7w%3D\" aria-label=\"CAS reference 71\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20436461\" aria-label=\"PubMed reference 71\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4840234\" aria-label=\"PubMed Central reference 71\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 71\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=GREAT%20improves%20functional%20interpretation%20of%20cis-regulatory%20regions&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.1630&amp;volume=28&amp;pages=495-501&amp;publication_year=2010&amp;author=McLean%2CCY\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"72.\"><p class=\"c-article-references__text\" id=\"ref-CR72\">Setty, M. et al. Characterization of cell fate probabilities in single-cell data with Palantir. <i>Nat. Biotechnol.</i> <b>37</b>, 451–460 (2019).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41587-019-0068-4\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41587-019-0068-4\" aria-label=\"Article reference 72\" data-doi=\"10.1038/s41587-019-0068-4\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1MXosV2qs74%3D\" aria-label=\"CAS reference 72\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30899105\" aria-label=\"PubMed reference 72\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7549125\" aria-label=\"PubMed Central reference 72\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 72\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Characterization%20of%20cell%20fate%20probabilities%20in%20single-cell%20data%20with%20Palantir&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-019-0068-4&amp;volume=37&amp;pages=451-460&amp;publication_year=2019&amp;author=Setty%2CM\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"73.\"><p class=\"c-article-references__text\" id=\"ref-CR73\">Reuter, B., Weber, M., Fackeldey, K., Röblitz, S. &amp; Garcia, M. E. Generalized Markov state modeling method for nonequilibrium biomolecular dynamics: exemplified on amyloid β conformational dynamics driven by an oscillating electric field. <i>J. Chem. Theory Comput.</i> <b>14</b>, 3579–3594 (2018).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1021/acs.jctc.8b00079\" data-track-action=\"article reference\" href=\"https://doi.org/10.1021%2Facs.jctc.8b00079\" aria-label=\"Article reference 73\" data-doi=\"10.1021/acs.jctc.8b00079\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVemtrrE\" aria-label=\"CAS reference 73\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29812922\" aria-label=\"PubMed reference 73\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 73\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Generalized%20Markov%20state%20modeling%20method%20for%20nonequilibrium%20biomolecular%20dynamics%3A%20exemplified%20on%20amyloid%20%CE%B2%20conformational%20dynamics%20driven%20by%20an%20oscillating%20electric%20field&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;doi=10.1021%2Facs.jctc.8b00079&amp;volume=14&amp;pages=3579-3594&amp;publication_year=2018&amp;author=Reuter%2CB&amp;author=Weber%2CM&amp;author=Fackeldey%2CK&amp;author=R%C3%B6blitz%2CS&amp;author=Garcia%2CME\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"74.\"><p class=\"c-article-references__text\" id=\"ref-CR74\">Velten, L. et al. Human haematopoietic stem cell lineage commitment is a continuous process. <i>Nat. Cell Biol.</i> <b>19</b>, 271–281 (2017).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/ncb3493\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fncb3493\" aria-label=\"Article reference 74\" data-doi=\"10.1038/ncb3493\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2sXltVWnsr0%3D\" aria-label=\"CAS reference 74\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28319093\" aria-label=\"PubMed reference 74\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5496982\" aria-label=\"PubMed Central reference 74\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 74\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Human%20haematopoietic%20stem%20cell%20lineage%20commitment%20is%20a%20continuous%20process&amp;journal=Nat.%20Cell%20Biol.&amp;doi=10.1038%2Fncb3493&amp;volume=19&amp;pages=271-281&amp;publication_year=2017&amp;author=Velten%2CL\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"75.\"><p class=\"c-article-references__text\" id=\"ref-CR75\">Agu, C. A. et al. Successful generation of human induced pluripotent stem cell lines from blood samples held at room temperature for up to 48 hr. <i>Stem Cell Rep.</i> <b>5</b>, 660–671 (2015).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.stemcr.2015.08.012\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.stemcr.2015.08.012\" aria-label=\"Article reference 75\" data-doi=\"10.1016/j.stemcr.2015.08.012\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFeju7bJ\" aria-label=\"CAS reference 75\">CAS</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 75\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Successful%20generation%20of%20human%20induced%20pluripotent%20stem%20cell%20lines%20from%20blood%20samples%20held%20at%20room%20temperature%20for%20up%20to%2048%20hr&amp;journal=Stem%20Cell%20Rep.&amp;doi=10.1016%2Fj.stemcr.2015.08.012&amp;volume=5&amp;pages=660-671&amp;publication_year=2015&amp;author=Agu%2CCA\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"76.\"><p class=\"c-article-references__text\" id=\"ref-CR76\">Jinek, M. et al. A programmable dual-RNA–guided DNA endonuclease in adaptive bacterial immunity. <i>Science</i> <b>337</b>, 816–821 (2012).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1126/science.1225829\" data-track-action=\"article reference\" href=\"https://doi.org/10.1126%2Fscience.1225829\" aria-label=\"Article reference 76\" data-doi=\"10.1126/science.1225829\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"ads reference\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012Sci...337..816J\" aria-label=\"ADS reference 76\">ADS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"cas reference\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFOqsb3L\" aria-label=\"CAS reference 76\">CAS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22745249\" aria-label=\"PubMed reference 76\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6286148\" aria-label=\"PubMed Central reference 76\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 76\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=A%20programmable%20dual-RNA%E2%80%93guided%20DNA%20endonuclease%20in%20adaptive%20bacterial%20immunity&amp;journal=Science&amp;doi=10.1126%2Fscience.1225829&amp;volume=337&amp;pages=816-821&amp;publication_year=2012&amp;author=Jinek%2CM\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"77.\"><p class=\"c-article-references__text\" id=\"ref-CR77\">Gholipour, A. et al. A normative spatiotemporal MRI atlas of the fetal brain for automatic segmentation and analysis of early brain growth. <i>Sci. Rep.</i> <b>7</b>, 476 (2017).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1038/s41598-017-00525-w\" data-track-action=\"article reference\" href=\"https://doi.org/10.1038%2Fs41598-017-00525-w\" aria-label=\"Article reference 77\" data-doi=\"10.1038/s41598-017-00525-w\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"ads reference\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017NatSR...7..476G\" aria-label=\"ADS reference 77\">ADS</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28352082\" aria-label=\"PubMed reference 77\">PubMed</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed central reference\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5428658\" aria-label=\"PubMed Central reference 77\">PubMed Central</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 77\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=A%20normative%20spatiotemporal%20MRI%20atlas%20of%20the%20fetal%20brain%20for%20automatic%20segmentation%20and%20analysis%20of%20early%20brain%20growth&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-017-00525-w&amp;volume=7&amp;publication_year=2017&amp;author=Gholipour%2CA\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"78.\"><p class=\"c-article-references__text\" id=\"ref-CR78\">Yushkevich, P. A. et al. User-guided 3D active contour segmentation of anatomical structures: significantly improved efficiency and reliability. <i>Neuroimage</i> <b>31</b>, 1116–1128 (2006).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.neuroimage.2006.01.015\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.neuroimage.2006.01.015\" aria-label=\"Article reference 78\" data-doi=\"10.1016/j.neuroimage.2006.01.015\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16545965\" aria-label=\"PubMed reference 78\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 78\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=User-guided%203D%20active%20contour%20segmentation%20of%20anatomical%20structures%3A%20significantly%20improved%20efficiency%20and%20reliability&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2006.01.015&amp;volume=31&amp;pages=1116-1128&amp;publication_year=2006&amp;author=Yushkevich%2CPA\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"79.\"><p class=\"c-article-references__text\" id=\"ref-CR79\">Ebner, M. et al. An automated framework for localization, segmentation and super-resolution reconstruction of fetal brain MRI. <i>Neuroimage</i> <b>206</b>, 116324 (2020).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1016/j.neuroimage.2019.116324\" data-track-action=\"article reference\" href=\"https://doi.org/10.1016%2Fj.neuroimage.2019.116324\" aria-label=\"Article reference 79\" data-doi=\"10.1016/j.neuroimage.2019.116324\">Article</a>\\xa0\\n    <a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"link\" data-track-action=\"pubmed reference\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31704293\" aria-label=\"PubMed reference 79\">PubMed</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 79\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=An%20automated%20framework%20for%20localization%2C%20segmentation%20and%20super-resolution%20reconstruction%20of%20fetal%20brain%20MRI&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2019.116324&amp;volume=206&amp;publication_year=2020&amp;author=Ebner%2CM\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"80.\"><p class=\"c-article-references__text\" id=\"ref-CR80\">Bayer, S. A. &amp; Altman, J. <i>The Human Brain During the Second Trimester</i> (1st ed.) <a href=\"https://doi.org/10.1201/9780203507483\" data-track=\"click\" data-track-action=\"external reference\" data-track-label=\"10.1201/9780203507483\">https://doi.org/10.1201/9780203507483</a> (CRC, 2005).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" rel=\"nofollow noopener\" data-track-label=\"10.1201/9780203507483\" data-track-action=\"book reference\" href=\"https://doi.org/10.1201%2F9780203507483\" aria-label=\"Book reference 80\" data-doi=\"10.1201/9780203507483\">Book</a>\\xa0\\n    <a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 80\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20Human%20Brain%20during%20the%20Second%20Trimester&amp;doi=10.1201%2F9780203507483&amp;publication_year=2005&amp;author=Bayer%2C%2CSA&amp;author=Altman%2C%2CJ\">\\n                    Google Scholar</a>\\xa0\\n                </p></li><li class=\"c-article-references__item js-c-reading-companion-references-item\" data-counter=\"81.\"><p class=\"c-article-references__text\" id=\"ref-CR81\">Bayer, S. A. &amp; Altman, J. <i>The Human Brain During the Third Trimester</i> (1st ed.) <a href=\"https://doi.org/10.1201/9780203494943\" data-track=\"click\" data-track-action=\"external reference\" data-track-label=\"10.1201/9780203494943\">https://doi.org/10.1201/9780203494943</a>\\xa0(CRC, 2003).</p><p class=\"c-article-references__links u-hide-print\"><a data-track=\"click\" data-track-action=\"google scholar reference\" data-track-label=\"link\" rel=\"nofollow noopener\" aria-label=\"Google Scholar reference 81\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Atlas%20of%20Human%20Central%20Nervous%20System%20Development&amp;publication_year=2003&amp;author=Bayer%2C%2CSA&amp;author=Altman%2C%2CJ\">\\n                    Google Scholar</a>\\xa0\\n                </p></li></ol><p class=\"c-article-references__download u-hide-print\"><a data-track=\"click\" data-track-action=\"download citation references\" data-track-label=\"link\" rel=\"nofollow\" href=\"https://citation-needed.springer.com/v2/references/10.1038/s41586-023-06473-y?format=refman&amp;flavour=references\">Download references<svg width=\"16\" height=\"16\" focusable=\"false\" role=\"img\" aria-hidden=\"true\" class=\"u-icon\"><use xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"#icon-eds-i-download-medium\"></use></svg></a></p></div></div></div></section></div><section data-title=\"Acknowledgements\"><div class=\"c-article-section\" id=\"Ack1-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Ack1\">Acknowledgements</h2><div class=\"c-article-section__content\" id=\"Ack1-content\"><p>We thank the patients and their families for participating in this study. We thank all members of the laboratory of J.A.K. for support and discussions; F. Bonnay, E. Chatzidaki, O. L. Eichmüller, R. Najm and J. Sidhaye for comments on the manuscript; M. Nezhyba and T. Lendl from the IMP/IMBA Biooptics Facility for technical support; A. Vogt, F. Drochter and T. Grentzinger from the VBCF NGS facility for single-cell RNA sequencing library preparation; the IMBA Stem Cell Core facility for generation of induced pluripotent stem cell lines; and\\xa0M. Balmana Esteban\\xa0and the Christoph Bock group (CeMM) for the help with single-cell multiomics sequencing. Work in the laboratory of J.A.K. is supported by SFARI (pilot award no. 724430); the Austrian Federal Ministry of Education, Science and Research; the Austrian Academy of Sciences; the City of Vienna; the Austrian Science Fund (FWF) Special Research Programme (grant no. F 7804-B) and two Stand-Alone grants (grant no. P 35680 and no. P 35369); and a European Research Council (ERC) Advanced Grant under the European Union’s Horizon 2020 programs (grant no. 695642 and no. 874769).\\xa0Work in the laboratory of B.T. is supported by the European Research Council (organomics and braintime (to B.T.)); the Chan Zuckerberg Initiative DAF, an advised fund of the Silicon Valley Community Foundation (grant no. CZF2019-002440); the Swiss National Science Foundation (grant no. 310030_192604); and the National Centre of Competence in Research Molecular Systems Engineering. A.V. is supported by an EMBO Fellowship (grant no. ALTF-1112-2019). J.S.F. was supported by a Boehringer Ingelheim Fonds PhD fellowship.\\xa0C.M.-C. was\\xa0supported by the\\xa0SCORPION Austrian Science Fund (FWF) DOC 72-B27.</p></div></div></section><section aria-labelledby=\"author-information\" data-title=\"Author information\"><div class=\"c-article-section\" id=\"author-information-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"author-information\">Author information</h2><div class=\"c-article-section__content\" id=\"author-information-content\"><span class=\"c-article-author-information__subtitle u-visually-hidden\" id=\"author-notes\">Author notes</span><ol class=\"c-article-author-information__list\"><li class=\"c-article-author-information__item\" id=\"na1\"><p>These authors contributed equally: Chong Li, Jonas Simon Fleck</p></li></ol><h3 class=\"c-article__sub-heading\" id=\"affiliations\">Authors and Affiliations</h3><ol class=\"c-article-author-affiliation__list\"><li id=\"Aff1\"><p class=\"c-article-author-affiliation__address\">Institute of Molecular Biotechnology of the Austrian Academy of Science (IMBA), Vienna, Austria</p><p class=\"c-article-author-affiliation__authors-list\">Chong Li,\\xa0Catarina Martins-Costa,\\xa0Thomas R. Burkard,\\xa0Jan Themann,\\xa0Angela Maria Peer,\\xa0Ábel Vertesy,\\xa0Jamie B. Littleboy,\\xa0Christopher Esk,\\xa0Ulrich Elling,\\xa0Nina S. Corsini\\xa0&amp;\\xa0Juergen A. Knoblich</p></li><li id=\"Aff2\"><p class=\"c-article-author-affiliation__address\">Department of Biosystems Science and Engineering, ETH Zürich, Basel, Switzerland</p><p class=\"c-article-author-affiliation__authors-list\">Jonas Simon Fleck\\xa0&amp;\\xa0Barbara Treutlein</p></li><li id=\"Aff3\"><p class=\"c-article-author-affiliation__address\">Department of Radiodiagnostics, Medical University of Vienna, Vienna, Austria</p><p class=\"c-article-author-affiliation__authors-list\">Marlene Stuempflen\\xa0&amp;\\xa0Gregor Kasprian</p></li><li id=\"Aff4\"><p class=\"c-article-author-affiliation__address\">Institute of Molecular Biology, University of Innsbruck, Innsbruck, Austria</p><p class=\"c-article-author-affiliation__authors-list\">Christopher Esk</p></li><li id=\"Aff5\"><p class=\"c-article-author-affiliation__address\">Department of Neurology, Medical University of Vienna, Vienna, Austria</p><p class=\"c-article-author-affiliation__authors-list\">Juergen A. Knoblich</p></li></ol><div class=\"u-js-hide u-hide-print\" data-test=\"author-info\"><span class=\"c-article__sub-heading\">Authors</span><ol class=\"c-article-authors-search u-list-reset\"><li id=\"auth-Chong-Li-Aff1\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Chong Li</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Chong%20Li\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chong%20Li\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chong%20Li%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Jonas_Simon-Fleck-Aff2\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Jonas Simon Fleck</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Jonas%20Simon%20Fleck\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jonas%20Simon%20Fleck\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jonas%20Simon%20Fleck%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Catarina-Martins_Costa-Aff1\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Catarina Martins-Costa</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Catarina%20Martins-Costa\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Catarina%20Martins-Costa\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Catarina%20Martins-Costa%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Thomas_R_-Burkard-Aff1\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Thomas R. Burkard</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Thomas%20R.%20Burkard\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Thomas%20R.%20Burkard\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Thomas%20R.%20Burkard%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Jan-Themann-Aff1\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Jan Themann</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Jan%20Themann\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jan%20Themann\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jan%20Themann%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Marlene-Stuempflen-Aff3\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Marlene Stuempflen</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Marlene%20Stuempflen\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Marlene%20Stuempflen\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Marlene%20Stuempflen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Angela_Maria-Peer-Aff1\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Angela Maria Peer</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Angela%20Maria%20Peer\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Angela%20Maria%20Peer\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Angela%20Maria%20Peer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-_bel-Vertesy-Aff1\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Ábel Vertesy</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=%C3%81bel%20Vertesy\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=%C3%81bel%20Vertesy\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22%C3%81bel%20Vertesy%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Jamie_B_-Littleboy-Aff1\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Jamie B. Littleboy</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Jamie%20B.%20Littleboy\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jamie%20B.%20Littleboy\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jamie%20B.%20Littleboy%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Christopher-Esk-Aff1-Aff4\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Christopher Esk</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Christopher%20Esk\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Christopher%20Esk\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Christopher%20Esk%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Ulrich-Elling-Aff1\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Ulrich Elling</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Ulrich%20Elling\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ulrich%20Elling\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ulrich%20Elling%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Gregor-Kasprian-Aff3\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Gregor Kasprian</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Gregor%20Kasprian\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Gregor%20Kasprian\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Gregor%20Kasprian%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Nina_S_-Corsini-Aff1\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Nina S. Corsini</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Nina%20S.%20Corsini\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nina%20S.%20Corsini\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nina%20S.%20Corsini%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Barbara-Treutlein-Aff2\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Barbara Treutlein</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Barbara%20Treutlein\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Barbara%20Treutlein\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Barbara%20Treutlein%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li><li id=\"auth-Juergen_A_-Knoblich-Aff1-Aff5\"><span class=\"c-article-authors-search__title u-h3 js-search-name\">Juergen A. Knoblich</span><div class=\"c-article-authors-search__list\"><div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"><a href=\"/search?author=Juergen%20A.%20Knoblich\" class=\"c-article-button\" data-track=\"click\" data-track-action=\"author link - publication\" data-track-label=\"link\" rel=\"nofollow\">View author publications</a></div><div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"><p class=\"search-in-title-js c-article-authors-search__text\">You can also search for this author in\\n                        <span class=\"c-article-identifiers\"><a class=\"c-article-identifiers__item\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Juergen%20A.%20Knoblich\" data-track=\"click\" data-track-action=\"author link - pubmed\" data-track-label=\"link\" rel=\"nofollow\">PubMed</a><span class=\"u-hide\">\\xa0</span><a class=\"c-article-identifiers__item\" href=\"http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Juergen%20A.%20Knoblich%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" data-track=\"click\" data-track-action=\"author link - scholar\" data-track-label=\"link\" rel=\"nofollow\">Google Scholar</a></span></p></div></div></li></ol></div><h3 class=\"c-article__sub-heading\" id=\"contributions\">Contributions</h3><p>C.L. and J.A.K. conceived the project and experimental design and secured the funding. C.L., J.S.F. and J.A.K. prepared the manuscript with input from all authors. C.L. performed all the experiments and analysed the data with the help of T.R.B., J.T., A.M.P., A.V., J.B.L. and C.E. J.S.F. performed the analysis of all single-cell RNA sequencing and multiome data under the supervision of B.T. M.S. and G.K. performed patient diagnosis and analysed\\xa0magnetic resonance imaging. C.M.-C. generated induced pluripotent stem cell lines under the supervision of N.S.C. U.E. provided sgRNA predication.</p><h3 class=\"c-article__sub-heading\" id=\"corresponding-author\">Corresponding authors</h3><p id=\"corresponding-author-list\">Correspondence to\\n                <a id=\"corresp-c1\" href=\"mailto:chong.li@imba.oeaw.ac.at\">Chong Li</a>, <a id=\"corresp-c2\" href=\"mailto:barbara.treutlein@bsse.ethz.ch\">Barbara Treutlein</a> or <a id=\"corresp-c3\" href=\"mailto:juergen.knoblich@imba.oeaw.ac.at\">Juergen A. Knoblich</a>.</p></div></div></section><section data-title=\"Ethics declarations\"><div class=\"c-article-section\" id=\"ethics-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"ethics\">Ethics declarations</h2><div class=\"c-article-section__content\" id=\"ethics-content\">\\n              \\n                <h3 class=\"c-article__sub-heading\" id=\"FPar4\">Competing interests</h3>\\n                <p>J.A.K. is on the supervisory and scientific advisory board of a:head bio AG (<a href=\"https://aheadbio.com\">https://aheadbio.com</a>) and is an inventor on several patents relating to cerebral organoids. The remaining authors declare no competing interests.</p>\\n              \\n            </div></div></section><section data-title=\"Peer review\"><div class=\"c-article-section\" id=\"peer-review-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"peer-review\">Peer review</h2><div class=\"c-article-section__content\" id=\"peer-review-content\">\\n              \\n              \\n                <h3 class=\"c-article__sub-heading\" id=\"FPar3\">Peer review information</h3>\\n                <p><i>Nature</i> thanks the anonymous reviewer(s) for their contribution to the peer review of this work. <a data-track=\"click\" data-track-label=\"link\" data-track-action=\"supplementary material anchor\" href=\"/articles/s41586-023-06473-y#MOESM10\">Peer reviewer reports are</a> available.</p>\\n              \\n            </div></div></section><section data-title=\"Additional information\"><div class=\"c-article-section\" id=\"additional-information-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"additional-information\">Additional information</h2><div class=\"c-article-section__content\" id=\"additional-information-content\"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title=\"Extended data figures and tables\"><div class=\"c-article-section\" id=\"Sec41-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec41\">Extended data figures and tables</h2><div class=\"c-article-section__content\" id=\"Sec41-content\"><div data-test=\"supplementary-info\"><div id=\"figshareContainer\" class=\"c-article-figshare-container\" data-test=\"figshare-container\"></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig5\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 1 a grna reporter assay to dete\" href=\"/articles/s41586-023-06473-y/figures/5\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig5_ESM.jpg\">Extended Data Fig. 1 A gRNA reporter assay to determine gRNA efficiency.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>, Diagram of the lentiviral construct delivering dual-sgRNA cassette and eCas9 under the spleen focus forming virus (SFFV) promotor. Lentivirus infected cells are labelled with GFP. Retroviral transduction is used to generate 3T3 cell lines. A pre-assembled array of gRNA-targeting sequences is fused with TagBFP. <b>b</b>, Flow cytometry graph shows a 3T3 reporter cell line is positive for both BFP and Tomato. After lentiviral infection, transduced cells (GFP+) and internal control cells (GFP-) are subjected to FACS-based analysis at 20 dpi. Efficient editing causing frameshift mutations lead to the loss of BFP fluorescence. <b>c</b>, Four dual-sgRNA examples show no (0–10% reduction), mild (10–45% reduction), medium (45–75% reduction) and strong editing efficiency (&gt; 75% reduction) at 20 dpi, respectively, based on median fluorescence intensity. <b>d</b>, 98 pairs of gRNAs were tested and categorized into four groups. <b>e</b>, Editing efficiency of selected paired gRNAs from 36 ASD genes.</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig6\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 2 perturbations of asd genes ef\" href=\"/articles/s41586-023-06473-y/figures/6\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig6_ESM.jpg\">Extended Data Fig. 2 Perturbations of ASD genes efficiently decrease protein expression levels.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>, Confocal images showing IHC analyzes of protein expression levels in organoids with individual genes perturbed (<i>ADNP</i>, <i>ARID1B</i>, <i>PHF3</i>, <i>SMARCC2</i>, <i>KMT2C</i>). Note that both wild-type (dTomato-) and perturbed cells (dTomato+) present in each organoid. Arrowheads and dotted lines indicate perturbed cells or ventricular zones, respectively. <b>b</b>, Violin plots (all data points and median values; each data point represents a single cell; cells were measured from 4 organoids) showing quantification of protein expression levels, as measured by fluorescent intensity. Bar plot (mean\\u2009±\\u2009S.D., n\\u2009=\\u20093 areas from 3 individual organoid) showing quantification of protein expression levels from eCas9-uninduced and induced VZ area. Two-sided student’s t-test. *<i>P</i>\\u2009&lt;\\u20090.05, **<i>P</i>\\u2009&lt;\\u20090.01. Scale bar, 100\\u2009μm.</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig7\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 3 grna library, infection and i\" href=\"/articles/s41586-023-06473-y/figures/7\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig7_ESM.jpg\">Extended Data Fig. 3 gRNA library, infection and integration of CHOOSE lentivirus in hES\\xa0cells.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>, Plot showing cumulative fraction of uniquely barcoded gRNA cassettes from the CHOOSE ASD plasmid library. <b>b</b>, Plot showing cumulative fraction of uniquely barcoded gRNA cassettes separated by each target gene from the plasmid library. <b>c</b>, Infection rate of CHOOSE lentivirus on hES\\xa0cells determined by flow-cytometry. Infected cells are positive for GFP. <b>d</b>, Top diagram showing the duplication of the dual-sgRNA cassette after the lentiviral integration into the host genome. FW and RV are paired primers used to demonstrate the successful integration of the lentivirus. The primers amplify specific regions of the genomic DNA extracted from lentivirus-infected hES\\xa0cells, and the lentiviral plasmid. <b>e</b>, Representative gel electrophoresis analysis from 2 independent experiments showing 12\\u2009kb and 13\\u2009kb bands detected from gDNA and plasmid respectively.</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig8\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 4 clonal diversity of the choos\" href=\"/articles/s41586-023-06473-y/figures/8\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig8_ESM.jpg\">Extended Data Fig. 4 Clonal diversity of the CHOOSE organoids.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>,<b>b</b>, Reads distribution for each barcoded gRNA cassette from the plasmid library (<b>a</b>) and from genomic DNA (<b>b</b>) extracted from the hES\\xa0cells infected by lentivirus. <b>c</b>, Box plot (minimum, maximum and median values; all data points) showing clone numbers recovered from scRNAseq libraries. Each data point represents a scRNAseq library. n\\u2009=\\u200911 independent pools (libraries) of organoids as replicates, 2 batches. <b>d</b>, Bar plot showing total clone numbers for each perturbation recovered from 11 scRNAseq libraries. <b>e</b>, Violin plot showing clone size distribution for each perturbation recovered from 11 scRNAseq libraries. Each data point represents one clone. <b>f</b>, Violin plot showing the average size of total clones recovered for each perturbation. <b>g</b>, <b>h</b>, Box plots (minimum, maximum and median values; all data points) showing the number of clones recovered from bulk genomic DNA extracted from eCas9-induced and uninduced cells. Each data point represents a bulk genomic DNA sample (50,000–150,000 cells sorted from pooled organoids). n\\u2009=\\u20095 pools of organoids for eCas9-induced cells; n\\u2009=\\u200911 pools of organoids for eCas9-uninduced cells. UCB, unique clone barcode.</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig9\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 5 generation of cerebral organo\" href=\"/articles/s41586-023-06473-y/figures/9\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig9_ESM.jpg\">Extended Data Fig. 5 Generation of cerebral organoids using the CHOOSE system.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>, Schematic showing the protocol used for generating cerebral organoids. 4-OHT was added to induce eCas9 expression at day 5. A short 3-day CHIR treatment was applied from day 12–14. Culture medium was switched to Brainphys after day 60 for advanced neuronal maturation. Tissues are collected at day 120 and subjected to scRNA-seq assays. <b>b</b>, ES\\xa0cells were infected by lentivirus and GFP positive cells were collected to make EBs. Microscopic images show the overall morphology and GFP expression of cerebral organoids overtime. Top, bright-field; Bottom, GFP fluorescence. Scale bar, 100\\u2009μm. <b>c</b>. Immunohistochemical staining of cerebral organoids on day 24. GFP labels cells infected with lentivirus and Tomato is a reporter for eCas9 expression. SOX2 is a marker for progenitor cells. FOXG1 is a marker for tissues with telencephalon identity. <b>d</b>, FACS plots showing percentages of eCas9-induced cell population (GFP+/dTomato+) at day 30. Organoids are treated with different concentrations of 4-OHT at day 5. <b>e</b>, FACS plots showing percentages of eCas9-induced cell population at different developmental stages when organoids are treated with 0.3\\u2009ug/ml 4-OHT. <b>f</b>, Bar plot (mean\\u2009±\\u2009S.D.) showing average eCas9 induction rate at day 120 when organoids are treated with 0.3\\u2009ug/ml 4-OHT. Each dot indicates a pool of 3–7 organoids. n\\u2009=\\u20099 biologically independent pools of organoids, examined over 2 independent experimental batches. Data are presented as mean values ± SD.</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig10\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 6 cell type and grna compositio\" href=\"/articles/s41586-023-06473-y/figures/10\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig10_ESM.jpg\">Extended Data Fig. 6 Cell type and gRNA composition of scRNA-seq libraries.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>, Histograms showing UMI count and number of detected features in the QC-controlled scRNA-seq dataset for each batch of organoids. <b>b</b>, Bar plot showing numbers of recovered cells with assigned gRNAs for each library. <b>c</b>, Bar plots showing cell number and gRNA composition for each replicate split by batch. <b>d</b>, UMAP embedding of unperturbed cells (control gRNA and eCas9-uninduced) coloured by annotated cell type. <b>e</b>,<b>f</b>,<b>h</b>,<b>i</b>, UMAP embedding of the unperturbed cells and full CHOOSE dataset including induced and uninduced samples filtered for gRNA detection. Cells are coloured by perturbation status (<b>e</b>,<b>h</b>) and batch (<b>f</b>,<b>i</b>). <b>g</b>,<b>j</b>, Feature plots showing expression of marker genes on a UMAP embedding for unperturbed cells (<b>g</b>) and full CHOOSE dataset (<b>j</b>). <b>k</b>, UMAP embedding coloured by velocity pseudotime to show developmental directions. <b>l</b>,<b>m</b>, Heatmaps showing expression of perturbed target genes in cerebral organoid cell types (<b>l</b>) and cell types in the developing human brain (Braun et al.<sup><a data-track=\"click\" data-track-action=\"reference anchor\" data-track-label=\"link\" data-test=\"citation-ref\" aria-label=\"Reference 32\" title=\"Braun, E. et al. Comprehensive cell atlas of the first-trimester developing human brain. Preprint at bioRxiv &#xA;                  https://doi.org/10.1101/2022.10.24.513487&#xA;                  &#xA;                 (2022).\" href=\"/articles/s41586-023-06473-y#ref-CR32\" id=\"ref-link-section-d3800159e3472\">32</a></sup>) (<b>m</b>).</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig11\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 7 grna representation in ecas9-\" href=\"/articles/s41586-023-06473-y/figures/11\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig11_ESM.jpg\">Extended Data Fig. 7 gRNA representation in eCas9-induced and uninduced cell populations of the CHOOSE organoids.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>, Bar plots showing gRNA representation in scRNAseq libraries (pink circle; each library is an independent pool of organoids) and in bulk genomic DNA extracted from eCas9-induced and uninduced cell populations (blue circle; each sample is sorted from an independent pool of organoids) at 4 months. The numbers within each circle indicate batch numbers. Note batch 3 does not include non-targeting gRNA control and uses internal eCas9-uninduced cells as an alternative control. <b>b</b>, Bar plot showing gRNA representation in bulk genomic DNA extracted from eCas9-induced and uninduced cell populations sorted from organoid pools at different development stages.</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig12\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 8 perturbations of asd risk gen\" href=\"/articles/s41586-023-06473-y/figures/12\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig12_ESM.jpg\">Extended Data Fig. 8 Perturbations of ASD risk genes lead to cell proliferation or depletion.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>, Schematic showing organoids perturbed with individual genes are subjected to phenotypic characterizations through FACS and IHC analyzes. Note that wild-type (eCas9-) and perturbed (eCas9+) cells present in each organoid. <b>b</b>, <b>c</b>, Bar plots (mean\\u2009±\\u2009S.D., n\\u2009=\\u20093 (<b>b</b>) or 5 (<b>c</b>) biologically independent pools, each pool contains 3 organoids) showing perturbation-induced over-proliferation (<i>KMT2C</i> and <i>LEO1</i>) and depletion (<i>ADNP</i> and <i>WAC</i>). Over-proliferation and depletion were calculated using the number of induced cells divided by the number of un-induced cells sorted from the same pool of organoids, and compared to non-targeting gRNA control. One-way <i>ANOVA</i> post hoc Fisher’s LSD test. *<i>P</i>\\u2009&lt;\\u20090.05, **<i>P</i>\\u2009&lt;\\u20090.01. <b>d</b>, Confocal images showing representative images of organoids with individual genes perturbed. IHC analyzes were performed on 60-day-old organoids from at least 4 organoids over two independent batches. GFP labels cells expressing gRNA construct and dTomato indicates the induction of eCas9. Scale bar, 200\\u2009μm.</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig13\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 9 cell type compositional chang\" href=\"/articles/s41586-023-06473-y/figures/13\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig13_ESM.jpg\">Extended Data Fig. 9 Cell type compositional changes caused by ASD gene perturbations.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>, UMAP embeddings show single cell distribution for each perturbed gene. Local cell composition differences of perturbation versus control indicated by red (enrichment) or blue (depletion). <b>b</b>, UMAP embedding of the scRNA-seq dataset containing dorsal and ventral telencephalon trajectories.</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig14\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 10 differential abundance effec\" href=\"/articles/s41586-023-06473-y/figures/14\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig14_ESM.jpg\">Extended Data Fig. 10 Differential abundance effects of perturbations across independent pools of organoids.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p>Lolliplots showing effect size of differential abundance (log odds ratio) per cell type and target gene. Dot colour indicates batch, size indicates number of cells. A t-test was performed against a permuted background distribution to highlight highly consistent effects across replicates. Stars indicate statistical significance in the single cell-level test, diamonds indicate significance of a two-sided t-test across pools of organoids (<i>P</i>\\u2009&lt;\\u20090.05).</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig15\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 11 perturbations of asd risk ge\" href=\"/articles/s41586-023-06473-y/figures/15\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig15_ESM.jpg\">Extended Data Fig. 11 Perturbations of ASD risk genes lead to depletion of IPs.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>-<b>d</b>, Confocal images showing immunohistochemical staining of EOMES on 60-day-old organoids perturbed with individual genes. dTomato labels eCas9-induced cells. Yellow box labels radially organized structure with ventricular zone (VZ), subventricular zone (SVZ) and cortical plate (CP). Note the presence of EOMES+ cells in the SVZ. <b>e</b>, Bar plot (mean\\u2009±\\u2009S.E.M.; all data points; from left to right, n\\u2009=\\u200914, 14, 14, 12 areas collected from 4 organoids for each perturbation) showing quantification of percent EOMES+ cells per eCas9 positive column area. Individual perturbations are compared to non-targeting gRNA control. Two-sided student’s t-test. *<i>P</i>\\u2009&lt;\\u20090.05, **<i>P</i>\\u2009&lt;\\u20090.01. Scale bar, 100\\u2009μm.</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig16\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 12 perturbations of asd risk ge\" href=\"/articles/s41586-023-06473-y/figures/16\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig16_ESM.jpg\">Extended Data Fig. 12 Perturbations of ASD risk genes lead to enrichment of INPs.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>, Confocal images showing immunohistochemical staining of DLX2 on 60-day-old organoids perturbed with individual genes. dTomato labels eCas9-induced cells. <b>b</b>, Bar plot (mean\\u2009±\\u2009S.E.M.; all data points; from left to right, n\\u2009=\\u200916, 18, 16, 30 areas collected from 4 organoids for each perturbation) showing quantification of the number of DLX2+ cells per 100\\u2009μm<sup>2</sup> area. Areas containing multiple rosettes from each organoid were collected for quantification. Individual perturbations are compared to non-targeting gRNA control. Two-sided student’s t-test. *<i>P</i>\\u2009&lt;\\u20090.05, **<i>P</i>\\u2009&lt;\\u20090.01. Scale bar, 500\\u2009μm.</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig17\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 13 differential expression and \" href=\"/articles/s41586-023-06473-y/figures/17\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig17_ESM.jpg\">Extended Data Fig. 13 Differential expression and GO enrichment analysis of ASD genetic perturbations.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>, Scatter plot showing differential expression log fold change in dorsal and ventral telencephalon cells for LEO1 and KMT2C. Dot colour indicates the trajectory where the gene was differentially expressed. <b>b</b>, Bar plot showing perturbation-specific GO enrichment of DEGs in dorsal and ventral telencephalon cells. As a background gene set we used all genes expressed in &gt;5% of cells in the dataset.</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig18\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 14 quality control and grn infe\" href=\"/articles/s41586-023-06473-y/figures/18\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig18_ESM.jpg\">Extended Data Fig. 14 Quality control and GRN inference from the single-cell multiome dataset.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>, Histograms showing UMI count and number of detected features for RNA as well as fragment count and number of detected peaks for ATAC. <b>b</b>, Histogram showing distribution of transcription start site (TSS) enrichment scores across cells. <b>c</b>, Line plot showing TSS enrichment score at each position relative to the TSS. <b>d</b>, Histogram showing distribution of fragment length. <b>e</b>, UMAP embedding showing multiome data (black) integrated with the CHOOSE dataset. Multiome data were used in conjunction with the control cells from the CHOOSE dataset (grey) to infer the GRN with Pando. <b>f</b>, Feature plots showing expression of marker genes on the UMAP embedding. <b>g</b>, Genomic tracks showing accessible peaks in the proximity of GAD2 (ventral marker) and NEUROD6 (dorsal marker). <b>h</b>, Density scatter plot histograms showing the distributions of explained variance (x) and number of variables (y) in the fitted models for GRN construction. Dashed lines indicate the thresholds used for model selection. <b>i</b>, Boxplots showing the distribution of peaks (left) and TFs assigned per gene (middle, n\\u2009=\\u2009864), and number of genes assigned per TF (right, n\\u2009=\\u2009241) in the inferred GRN. The centre line represents the median, the box limits show the 25–75% interquartile range and the whiskers indicate 1.5× the interquartile range. <b>j</b>, Lolliplot showing the enrichment of ASD-associated genes from SFARI in inferred TF modules. Red colour indicates an FDR-corrected two-sided Fisher-test p-value of &lt;0.01. Dot size indicates the total number of genes in the module.</p></div></div><div class=\"c-article-supplementary__item js-c-reading-companion-figures-item\" data-test=\"supp-item\" id=\"Fig19\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"extended data fig. 15 an arid1b patient had enlarg\" href=\"/articles/s41586-023-06473-y/figures/19\" data-supp-info-image=\"//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_Fig19_ESM.jpg\">Extended Data Fig. 15 An ARID1B patient had enlarged ganglionic eminence.</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p><b>a</b>, Graph representation of <i>AR</i>, <i>PBX1</i>, <i>DLX6</i>, <i>OTX2</i>, and <i>OLIG1</i> TF modules, which are most strongly enriched in <i>ARID1B</i> DEG (Two-sided Fisher exact test p-value\\u2009&lt;\\u20090.01, top 4 odds ratio). The targets highlighted in yellow are SFARI genes, and the targets highlighted in black are TF modules enriched in SFARI genes. <b>b</b>, Top, sanger sequencing showing a duplication mutation (red arrow) identified in one of the alleles of patient 1. Bottom, sanger sequencing showing the mutation was repaired and this cell line is used as an isogenic control. Sequencing was done on fragments amplified from genomic DNA extracted from the two iPS\\xa0cell lines. <b>c</b>, SNP array genotyping of PBMCs and iPS\\xa0cells from patient 2 shows a microdeletion (dark green arrow) identified on chromosome 6. <b>d</b>-<b>f</b>, Bar plot showing the absolute GE (<b>d</b>), cortex (<b>e</b>) and total brain (<b>f</b>) volumes of control (mean\\u2009±\\u2009S.D.; n\\u2009=\\u20095, age macheted) and patient 2 at two gestational stages. <b>g</b>-<b>h</b>, Bar plots showing normalized GE to cortex volume (<b>g</b>) and total brain volume (<b>h</b>) of control (mean\\u2009±\\u2009S.D.; n\\u2009=\\u20095, age macheted) and patient 2 at two gestational stages.</p></div></div></div></div></div></section><section data-title=\"Supplementary information\"><div class=\"c-article-section\" id=\"Sec42-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec42\">Supplementary information</h2><div class=\"c-article-section__content\" id=\"Sec42-content\"><div data-test=\"supplementary-info\"><div class=\"c-article-supplementary__item\" data-test=\"supp-item\" id=\"MOESM1\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"supplementary table 1\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_MOESM1_ESM.xlsx\" data-supp-info-image=\"\">Supplementary Table 1</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p>Primers and sgRNA sequences.</p></div></div><div class=\"c-article-supplementary__item\" data-test=\"supp-item\" id=\"MOESM2\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"reporting summary\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_MOESM2_ESM.pdf\" data-supp-info-image=\"\">Reporting Summary</a></h3></div><div class=\"c-article-supplementary__item\" data-test=\"supp-item\" id=\"MOESM3\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"supplementary data 1\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_MOESM3_ESM.xlsx\" data-supp-info-image=\"\">Supplementary Data 1</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p>scRNAseq library cell numbers for each perturbation.</p></div></div><div class=\"c-article-supplementary__item\" data-test=\"supp-item\" id=\"MOESM4\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"supplementary data 2\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_MOESM4_ESM.txt\" data-supp-info-image=\"\">Supplementary Data 2</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p>CHOOSE differential gene expression in dorsal and ventral lineages.</p></div></div><div class=\"c-article-supplementary__item\" data-test=\"supp-item\" id=\"MOESM5\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"supplementary data 3\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_MOESM5_ESM.txt\" data-supp-info-image=\"\">Supplementary Data 3</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p>GO enrichment of TOP-DEG.</p></div></div><div class=\"c-article-supplementary__item\" data-test=\"supp-item\" id=\"MOESM6\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"supplementary data 4\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_MOESM6_ESM.txt\" data-supp-info-image=\"\">Supplementary Data 4</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p>Perturbation-specific GO enrichment of dorsal and ventral DEGs.</p></div></div><div class=\"c-article-supplementary__item\" data-test=\"supp-item\" id=\"MOESM7\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"supplementary data 5\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_MOESM7_ESM.txt\" data-supp-info-image=\"\">Supplementary Data 5</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p>TF modules and GRN inference.</p></div></div><div class=\"c-article-supplementary__item\" data-test=\"supp-item\" id=\"MOESM8\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"supplementary data 6\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_MOESM8_ESM.txt\" data-supp-info-image=\"\">Supplementary Data 6</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p>SFARI gene enriched TF modules.</p></div></div><div class=\"c-article-supplementary__item\" data-test=\"supp-item\" id=\"MOESM9\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"supplementary data 7\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_MOESM9_ESM.txt\" data-supp-info-image=\"\">Supplementary Data 7</a></h3><div class=\"c-article-supplementary__description\" data-component=\"thumbnail-container\"><p>CHOOSE DEG-enriched TF modules.</p></div></div><div class=\"c-article-supplementary__item\" data-test=\"supp-item\" id=\"MOESM10\"><h3 class=\"c-article-supplementary__title u-h3\"><a class=\"print-link\" data-track=\"click\" data-track-action=\"view supplementary info\" data-test=\"supp-info-link\" data-track-label=\"peer review file\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-023-06473-y/MediaObjects/41586_2023_6473_MOESM10_ESM.pdf\" data-supp-info-image=\"\">Peer Review File</a></h3></div></div></div></div></section><section data-title=\"Rights and permissions\"><div class=\"c-article-section\" id=\"rightslink-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"rightslink\">Rights and permissions</h2><div class=\"c-article-section__content\" id=\"rightslink-content\">\\n                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href=\"http://creativecommons.org/licenses/by/4.0/\" rel=\"license\">http://creativecommons.org/licenses/by/4.0/</a>.</p>\\n              <p class=\"c-article-rights\"><a data-track=\"click\" data-track-action=\"view rights and permissions\" data-track-label=\"link\" href=\"https://s100.copyright.com/AppDispatchServlet?title=Single-cell%20brain%20organoid%20screening%20identifies%20developmental%20defects%20in%20autism&amp;author=Chong%20Li%20et%20al&amp;contentID=10.1038%2Fs41586-023-06473-y&amp;copyright=The%20Author%28s%29&amp;publication=0028-0836&amp;publicationDate=2023-09-13&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY\">Reprints and permissions</a></p></div></div></section><section aria-labelledby=\"article-info\" data-title=\"About this article\"><div class=\"c-article-section\" id=\"article-info-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"article-info\">About this article</h2><div class=\"c-article-section__content\" id=\"article-info-content\"><div class=\"c-bibliographic-information\"><div class=\"u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border\"><a data-crossmark=\"10.1038/s41586-023-06473-y\" target=\"_blank\" rel=\"noopener\" href=\"https://crossmark.crossref.org/dialog/?doi=10.1038/s41586-023-06473-y\" data-track=\"click\" data-track-action=\"Click Crossmark\" data-track-label=\"link\" data-test=\"crossmark\"><img loading=\"lazy\" width=\"57\" height=\"81\" alt=\"Check for updates. Verify currency and authenticity via CrossMark\" src=\"data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>\"></a></div><div class=\"c-bibliographic-information__column\"><h3 class=\"c-article__sub-heading\" id=\"citeas\">Cite this article</h3><p class=\"c-bibliographic-information__citation\">Li, C., Fleck, J.S., Martins-Costa, C. <i>et al.</i> Single-cell brain organoid screening identifies developmental defects in autism.\\n                    <i>Nature</i> <b>621</b>, 373–380 (2023). https://doi.org/10.1038/s41586-023-06473-y</p><p class=\"c-bibliographic-information__download-citation u-hide-print\"><a data-test=\"citation-link\" data-track=\"click\" data-track-action=\"download article citation\" data-track-label=\"link\" data-track-external=\"\" rel=\"nofollow\" href=\"https://citation-needed.springer.com/v2/references/10.1038/s41586-023-06473-y?format=refman&amp;flavour=citation\">Download citation<svg width=\"16\" height=\"16\" focusable=\"false\" role=\"img\" aria-hidden=\"true\" class=\"u-icon\"><use xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"#icon-eds-i-download-medium\"></use></svg></a></p><ul class=\"c-bibliographic-information__list\" data-test=\"publication-history\"><li class=\"c-bibliographic-information__list-item\"><p>Received<span class=\"u-hide\">: </span><span class=\"c-bibliographic-information__value\"><time datetime=\"2022-04-25\">25 April 2022</time></span></p></li><li class=\"c-bibliographic-information__list-item\"><p>Accepted<span class=\"u-hide\">: </span><span class=\"c-bibliographic-information__value\"><time datetime=\"2023-07-21\">21 July 2023</time></span></p></li><li class=\"c-bibliographic-information__list-item\"><p>Published<span class=\"u-hide\">: </span><span class=\"c-bibliographic-information__value\"><time datetime=\"2023-09-13\">13 September 2023</time></span></p></li><li class=\"c-bibliographic-information__list-item\"><p>Issue Date<span class=\"u-hide\">: </span><span class=\"c-bibliographic-information__value\"><time datetime=\"2023-09-14\">14 September 2023</time></span></p></li><li class=\"c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width\"><p><abbr title=\"Digital Object Identifier\">DOI</abbr><span class=\"u-hide\">: </span><span class=\"c-bibliographic-information__value\">https://doi.org/10.1038/s41586-023-06473-y</span></p></li></ul><div data-component=\"share-box\"><div class=\"c-article-share-box u-display-none\" hidden=\"\"><h3 class=\"c-article__sub-heading\">Share this article</h3><p class=\"c-article-share-box__description\">Anyone you share the following link with will be able to read this content:</p><button class=\"js-get-share-url c-article-share-box__button\" type=\"button\" id=\"get-share-url\" data-track=\"click\" data-track-label=\"button\" data-track-external=\"\" data-track-action=\"get shareable link\">Get shareable link</button><div class=\"js-no-share-url-container u-display-none\" hidden=\"\"><p class=\"js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info\">Sorry, a shareable link is not currently available for this article.</p></div><div class=\"js-share-url-container u-display-none\" hidden=\"\"><p class=\"js-share-url c-article-share-box__only-read-input\" id=\"share-url\" data-track=\"click\" data-track-label=\"button\" data-track-action=\"select share url\"></p><button class=\"js-copy-share-url c-article-share-box__button--link-like\" type=\"button\" id=\"copy-share-url\" data-track=\"click\" data-track-label=\"button\" data-track-action=\"copy share url\" data-track-external=\"\">Copy to clipboard</button></div><p class=\"js-c-article-share-box__additional-info c-article-share-box__additional-info\">\\n                            Provided by the Springer Nature SharedIt content-sharing initiative\\n                        </p></div></div><div data-component=\"article-info-list\"></div></div></div></div></div></section>\\n            </div>\\n\\n            \\n        <section>\\n            <div class=\"c-article-section js-article-section\" id=\"further-reading-section\">\\n                <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"further-reading\">This article is cited by</h2>\\n                <div class=\"c-article-section__content js-collapsible-section\" id=\"further-reading-content\">\\n                    <ul class=\"c-article-further-reading__list\" id=\"further-reading-list\">\\n                        \\n                            <li class=\"c-article-further-reading__item js-ref-item\">\\n                            \\n                                <h3 class=\"c-article-further-reading__title\">\\n                                    <a class=\"print-link\" data-track=\"click\" data-track-action=\"view further reading article\"\\n                                       data-track-label=\"link:Humanized brain organoids-on-chip integrated with sensors for screening neuronal activity and neurotoxicity\" href=\"https://doi.org/10.1007/s00604-023-06165-4\">\\n                                        Humanized brain organoids-on-chip integrated with sensors for screening neuronal activity and neurotoxicity\\n                                    </a>\\n                                </h3>\\n                            \\n                                \\n                                    <ul data-test=\"author-list\" class=\"c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto\">\\n                                        <li>Pelin Saglam-Metiner</li><li>Ender Yildirim</li><li>Ozlem Yesil-Celiktas</li>\\n                                    </ul>\\n                                \\n                                <p class=\"c-article-further-reading__journal-title\"><i>Microchimica Acta</i> (2024)</p>\\n                            </li>\\n                        \\n                            <li class=\"c-article-further-reading__item js-ref-item\">\\n                            \\n                                <h3 class=\"c-article-further-reading__title\">\\n                                    <a class=\"print-link\" data-track=\"click\" data-track-action=\"view further reading article\"\\n                                       data-track-label=\"link:Curation of causal interactions mediated by genes associated with autism accelerates the understanding of gene-phenotype relationships underlying neurodevelopmental disorders\" href=\"https://doi.org/10.1038/s41380-023-02317-3\">\\n                                        Curation of causal interactions mediated by genes associated with autism accelerates the understanding of gene-phenotype relationships underlying neurodevelopmental disorders\\n                                    </a>\\n                                </h3>\\n                            \\n                                \\n                                    <ul data-test=\"author-list\" class=\"c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto\">\\n                                        <li>Marta Iannuccelli</li><li>Alessandro Vitriolo</li><li>Livia Perfetto</li>\\n                                    </ul>\\n                                \\n                                <p class=\"c-article-further-reading__journal-title\"><i>Molecular Psychiatry</i> (2023)</p>\\n                            </li>\\n                        \\n                    </ul>\\n                </div>\\n            </div>\\n        </section>\\n    \\n\\n            \\n                <section data-title=\"Comments\"><div class=\"c-article-section\" id=\"article-comments-section\"><h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"article-comments\">Comments</h2><div class=\"c-article-section__content\" id=\"article-comments-content\"><p>By submitting a comment you agree to abide by our <a href=\"/info/tandc.html\">Terms</a> and <a href=\"/info/community-guidelines.html\">Community Guidelines</a>. If you find something abusive or that does not comply with our terms or guidelines please flag it as inappropriate.</p></div></div></section>\\n                <div id=\"inject-comments\">\\n                    <div class=\"placeholder\" data-replace=\"true\" data-disqus-placeholder=\"/platform/disqus?doi=10.1038/s41586-023-06473-y #article-comments-container\">\\n                    </div>\\n                </div>\\n            \\n        </div>\\n</article>\\n</main>\\n\\n<aside class=\"c-article-extras u-hide-print\" aria-label=\"Article navigation\" data-component-reading-companion data-container-type=\"reading-companion\" data-track-component=\"reading companion\">\\n    <div class=\"js-context-bar-sticky-point-desktop\">\\n        \\n\\n        \\n            \\n                \\n    \\n        <div class=\"c-pdf-download u-clear-both js-pdf-download\">\\n            <a href=\"/articles/s41586-023-06473-y.pdf\" class=\"u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link\" data-article-pdf=\"true\" data-readcube-pdf-url=\"true\" data-test=\"download-pdf\" data-draft-ignore=\"true\" data-track=\"click\" data-track-action=\"download pdf\" data-track-label=\"link\" data-track-external download>\\n                <span class=\"c-pdf-download__text\">Download PDF</span>\\n                <svg aria-hidden=\"true\" focusable=\"false\" width=\"16\" height=\"16\" class=\"u-icon\"><use xlink:href=\"#icon-download\"/></svg>\\n            </a>\\n        </div>\\n    \\n\\n            \\n        \\n    </div>\\n\\n    \\n        \\n    \\n\\n    \\n    \\n        <div class=\"c-article-associated-content__container\">\\n            <section>\\n                <h2 class=\"c-article-associated-content__title u-mb-24\">Associated content</h2>\\n                \\n                    \\n                    \\n                        <div class=\"u-full-height u-mb-24\">\\n                            \\n    <article class=\"u-full-height c-card c-card--flush\">\\n        <div class=\"c-card__layout u-full-height\">\\n            <div class=\"c-card__body\">\\n                <h3 class=\"c-card__title\">\\n                    <a href=\"https://www.nature.com/articles/s41586-022-05279-8\"\\n                       class=\"c-card__link u-link-inherit\"\\n                       data-track=\"click\"\\n                       data-track-action=\"view article\"\\n                       data-track-category=\"associated content\"\\n                       \\n                       data-track-label=\"research\">Inferring and perturbing cell fate regulomes in human brain organoids</a>\\n                </h3>\\n                \\n<ul data-test=\"author-list\" class=\"c-author-list c-author-list--compact c-author-list--truncated\">\\n    <li>Jonas Simon Fleck</li><li>Sophie Martina Johanna Jansen</li><li>Barbara Treutlein</li>\\n</ul>\\n\\n                \\n    <div class=\"c-card__section c-meta\">\\n        \\n            <span class=\"c-meta__item\">Nature</span>\\n        \\n        <span class=\"c-meta__item\" data-test=\"article.type\"><span class=\"c-meta__type\">Article</span></span>\\n        \\n            <span class=\"c-meta__item\" data-test=\"open-access\"><span class=\"u-color-open-access\">Open Access</span></span>\\n        \\n        \\n            <time class=\"c-meta__item\" datetime=\"2022-10-05\">05 Oct 2022</time>\\n        \\n    </div>\\n\\n            </div>\\n        </div>\\n    </article>\\n\\n\\n                        </div>\\n                    \\n                \\n            </section>\\n        </div>\\n        <script>\\n            window.dataLayer = window.dataLayer || [];\\n            window.dataLayer[0] = window.dataLayer[0] || {};\\n            window.dataLayer[0].content = window.dataLayer[0].content || {};\\n            window.dataLayer[0].content.associatedContentTypes = \"research\";\\n        </script>\\n    \\n\\n    \\n\\n    <div class=\"c-reading-companion\">\\n        <div class=\"c-reading-companion__sticky\" data-component=\"reading-companion-sticky\" data-test=\"reading-companion-sticky\">\\n            <div class=\"c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active\" id=\"tabpanel-sections\">\\n                <div class=\"u-lazy-ad-wrapper u-mt-16 u-hide\" data-component-mpu>\\n                    <div class=\"c-ad c-ad--300x250\">\\n                        <div class=\"c-ad__inner\">\\n                            <p class=\"c-ad__label\">Advertisement</p>\\n                            \\n    <div id=\"div-gpt-ad-right-2\"\\n         class=\"div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide\"\\n         data-ad-type=\"right\"\\n         data-test=\"right-ad\"\\n         data-pa11y-ignore\\n         data-gpt\\n         data-gpt-unitpath=\"/285/nature.com/article\"\\n         data-gpt-sizes=\"300x250\"\\n         data-gpt-targeting=\"type=article;pos=right;artid=s41586-023-06473-y;doi=10.1038/s41586-023-06473-y;techmeta=100,107,13,14,31,38,39,44,45,47,51,63;subjmeta=136,2320,2571,378,532,61,631;kwrd=Development+of+the+nervous+system,Stem-cell+biotechnology,Stem+cells\">\\n        <noscript>\\n            <a href=\"//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature.com/article&amp;sz=300x250&amp;c=-2040194831&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41586-023-06473-y%26doi%3D10.1038/s41586-023-06473-y%26techmeta%3D100,107,13,14,31,38,39,44,45,47,51,63%26subjmeta%3D136,2320,2571,378,532,61,631%26kwrd%3DDevelopment+of+the+nervous+system,Stem-cell+biotechnology,Stem+cells\">\\n                <img data-test=\"gpt-advert-fallback-img\"\\n                     src=\"//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature.com/article&amp;sz=300x250&amp;c=-2040194831&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41586-023-06473-y%26doi%3D10.1038/s41586-023-06473-y%26techmeta%3D100,107,13,14,31,38,39,44,45,47,51,63%26subjmeta%3D136,2320,2571,378,532,61,631%26kwrd%3DDevelopment+of+the+nervous+system,Stem-cell+biotechnology,Stem+cells\"\\n                     alt=\"Advertisement\"\\n                     width=\"300\"\\n                     height=\"250\"></a>\\n        </noscript>\\n    </div>\\n\\n                        </div>\\n                    </div>\\n                </div>\\n            </div>\\n            <div class=\"c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width\" id=\"tabpanel-figures\"></div>\\n            <div class=\"c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width\" id=\"tabpanel-references\"></div>\\n        </div>\\n    </div>\\n</aside>\\n</div>\\n\\n\\n    \\n        <nav class=\"c-header__dropdown\" aria-labelledby=\"Explore-content\" data-test=\"Explore-content\" id=\"explore\" data-track-component=\"nature-150-split-header\">\\n            <div class=\"c-header__container\">\\n                <h2 id=\"Explore-content\" class=\"c-header__heading c-header__heading--js-hide\">Explore content</h2>\\n                <ul class=\"c-header__list c-header__list--js-stack\">\\n                    \\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/research-articles\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"research articles\"\\n                                   data-track-label=\"link\"\\n                                   data-test=\"explore-nav-item\">\\n                                    Research articles\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/news\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"news\"\\n                                   data-track-label=\"link\"\\n                                   data-test=\"explore-nav-item\">\\n                                    News\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/opinion\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"opinion\"\\n                                   data-track-label=\"link\"\\n                                   data-test=\"explore-nav-item\">\\n                                    Opinion\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/research-analysis\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"research analysis\"\\n                                   data-track-label=\"link\"\\n                                   data-test=\"explore-nav-item\">\\n                                    Research Analysis\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/careers\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"careers\"\\n                                   data-track-label=\"link\"\\n                                   data-test=\"explore-nav-item\">\\n                                    Careers\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/books-culture\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"books &amp; culture\"\\n                                   data-track-label=\"link\"\\n                                   data-test=\"explore-nav-item\">\\n                                    Books &amp; Culture\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/podcasts\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"podcasts\"\\n                                   data-track-label=\"link\"\\n                                   data-test=\"explore-nav-item\">\\n                                    Podcasts\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/videos\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"videos\"\\n                                   data-track-label=\"link\"\\n                                   data-test=\"explore-nav-item\">\\n                                    Videos\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/current-issue\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"current issue\"\\n                                   data-track-label=\"link\"\\n                                   data-test=\"explore-nav-item\">\\n                                    Current issue\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/browse-issues\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"browse issues\"\\n                                   data-track-label=\"link\"\\n                                   data-test=\"explore-nav-item\">\\n                                    Browse issues\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/collections\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"collections\"\\n                                   data-track-label=\"link\"\\n                                   data-test=\"explore-nav-item\">\\n                                    Collections\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/browse-subjects\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"subjects\"\\n                                   data-track-label=\"link\"\\n                                   data-test=\"explore-nav-item\">\\n                                    Subjects\\n                                </a>\\n                            </li>\\n                        \\n                    \\n                </ul>\\n                <ul class=\"c-header__list c-header__list--js-stack\">\\n                    \\n                        <li class=\"c-header__item\">\\n                            <a class=\"c-header__link\"\\n                               href=\"https://www.facebook.com/Nature\"\\n                               data-track=\"click\"\\n                               data-track-action=\"facebook\"\\n                               data-track-label=\"link\">Follow us on Facebook\\n                            </a>\\n                        </li>\\n                    \\n                    \\n                        <li class=\"c-header__item\">\\n                            <a class=\"c-header__link\"\\n                               href=\"https://twitter.com/nature\"\\n                               data-track=\"click\"\\n                               data-track-action=\"twitter\"\\n                               data-track-label=\"link\">Follow us on Twitter\\n                            </a>\\n                        </li>\\n                    \\n                    \\n                    \\n                        <li class=\"c-header__item c-header__item--hide-lg\">\\n                            <a class=\"c-header__link\"\\n                               href=\"https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;1\"\\n                               rel=\"nofollow\"\\n                               data-track=\"click\"\\n                               data-track-action=\"Sign up for alerts\"\\n                               data-track-external\\n                               data-track-label=\"link (mobile dropdown)\">Sign up for alerts<svg role=\"img\" aria-hidden=\"true\" focusable=\"false\" height=\"18\" viewBox=\"0 0 18 18\" width=\"18\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z\" fill=\"#fff\"/></svg>\\n                            </a>\\n                        </li>\\n                    \\n                    \\n                        <li class=\"c-header__item c-header__item--hide-lg\">\\n                            <a class=\"c-header__link\"\\n                               href=\"https://www.nature.com/nature.rss\"\\n                               data-track=\"click\"\\n                               data-track-action=\"rss feed\"\\n                               data-track-label=\"link\">\\n                                <span>RSS feed</span>\\n                            </a>\\n                        </li>\\n                    \\n                </ul>\\n            </div>\\n        </nav>\\n    \\n    \\n        \\n            <nav class=\"c-header__dropdown\" aria-labelledby=\"About-the-journal\" id=\"about-the-journal\" data-test=\"about-the-journal\" data-track-component=\"nature-150-split-header\">\\n                <div class=\"c-header__container\">\\n                    <h2 id=\"About-the-journal\" class=\"c-header__heading c-header__heading--js-hide\">About the journal</h2>\\n                    <ul class=\"c-header__list c-header__list--js-stack\">\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/journal-staff\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"journal staff\"\\n                                   data-track-label=\"link\">\\n                                    Journal Staff\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/editors\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"about the editors\"\\n                                   data-track-label=\"link\">\\n                                    About the Editors\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/journal-information\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"journal information\"\\n                                   data-track-label=\"link\">\\n                                    Journal Information\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/our-publishing-models\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"our publishing models\"\\n                                   data-track-label=\"link\">\\n                                    Our publishing models\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/editorial-values-statement\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"editorial values statement\"\\n                                   data-track-label=\"link\">\\n                                    Editorial Values Statement\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/journal-impact\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"journal metrics\"\\n                                   data-track-label=\"link\">\\n                                    Journal Metrics\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/awards\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"awards\"\\n                                   data-track-label=\"link\">\\n                                    Awards\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/contact\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"contact\"\\n                                   data-track-label=\"link\">\\n                                    Contact\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/editorial-policies\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"editorial policies\"\\n                                   data-track-label=\"link\">\\n                                    Editorial policies\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/history-of-nature\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"history of nature\"\\n                                   data-track-label=\"link\">\\n                                    History of Nature\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/send-a-news-tip\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"send a news tip\"\\n                                   data-track-label=\"link\">\\n                                    Send a news tip\\n                                </a>\\n                            </li>\\n                        \\n                    </ul>\\n                </div>\\n            </nav>\\n        \\n\\n        \\n            <nav class=\"c-header__dropdown\" aria-labelledby=\"Publish-with-us-label\" id=\"publish-with-us\" data-test=\"publish-with-us\" data-track-component=\"nature-150-split-header\">\\n                <div class=\"c-header__container\">\\n                    <h2 id=\"Publish-with-us-label\" class=\"c-header__heading c-header__heading--js-hide\">Publish with us</h2>\\n                    <ul class=\"c-header__list c-header__list--js-stack\">\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/for-authors\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"for authors\"\\n                                   data-track-label=\"link\">\\n                                    For Authors\\n                                </a>\\n                            </li>\\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"/nature/for-referees\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"for referees\"\\n                                   data-track-label=\"link\">\\n                                    For Referees\\n                                </a>\\n                            </li>\\n                        \\n                        \\n                            <li class=\"c-header__item\">\\n                                <a class=\"c-header__link\" data-test=\"nature-author-services\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"manuscript author services\"\\n                                   data-track-label=\"link manuscript author services\"\\n                                   href=\"https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022\">\\n                                    Language editing services\\n                                </a>\\n                            </li>\\n                        \\n                        \\n                            <li class=\"c-header__item c-header__item--keyline\">\\n                                <a class=\"c-header__link\"\\n                                   href=\"https://mts-nature.nature.com/\"\\n                                   data-track=\"click\"\\n                                   data-track-action=\"submit manuscript\"\\n                                   data-track-label=\"link (publish with us dropdown menu)\"\\n                                   data-track-external>Submit manuscript<svg role=\"img\" aria-hidden=\"true\" focusable=\"false\" height=\"18\" viewBox=\"0 0 18 18\" width=\"18\" xmlns=\"http://www.w3.org/2000/svg\"><path d=\"m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z\" fill=\"#fff\"/></svg>\\n                                </a>\\n                            </li>\\n                        \\n                    </ul>\\n                </div>\\n            </nav>\\n        \\n    \\n\\n\\n<div id=\"search-menu\" class=\"c-header__dropdown c-header__dropdown--full-width\" data-track-component=\"nature-150-split-header\">\\n    <div class=\"c-header__container\">\\n        <h2 class=\"c-header__visually-hidden\">Search</h2>\\n        <form class=\"c-header__search-form\" action=\"/search\" method=\"get\" role=\"search\" autocomplete=\"off\" data-test=\"inline-search\">\\n            <label class=\"c-header__heading\" for=\"keywords\">Search articles by subject, keyword or author</label>\\n            <div class=\"c-header__search-layout c-header__search-layout--max-width\">\\n                <div>\\n                    <input type=\"text\" required=\"\" class=\"c-header__input\" id=\"keywords\" name=\"q\" value=\"\">\\n                </div>\\n                <div class=\"c-header__search-layout\">\\n                    <div>\\n                        <label for=\"results-from\" class=\"c-header__visually-hidden\">Show results from</label>\\n                        <select id=\"results-from\" name=\"journal\" class=\"c-header__select\">\\n                            \\n                                \\n                                    <option value=\"\" selected>All journals</option>\\n                                    <option value=\"nature\">This journal</option>\\n                                \\n                            \\n                        </select>\\n                    </div>\\n                    <div>\\n                        <button type=\"submit\" class=\"c-header__search-button\">Search</button>\\n                    </div>\\n                </div>\\n\\n            </div>\\n        </form>\\n\\n        <div class=\"c-header__flush\">\\n            <a class=\"c-header__link\" href=\"/search/advanced\"\\n               data-track=\"click\" data-track-action=\"advanced search\" data-track-label=\"link\">\\n                Advanced search\\n            </a>\\n        </div>\\n\\n        <h3 class=\"c-header__heading c-header__heading--keyline\">Quick links</h3>\\n        <ul class=\"c-header__list\">\\n            <li><a class=\"c-header__link\" href=\"/subjects\" data-track=\"click\" data-track-action=\"explore articles by subject\" data-track-label=\"link\">Explore articles by subject</a></li>\\n            <li><a class=\"c-header__link\" href=\"/naturecareers\" data-track=\"click\" data-track-action=\"find a job\" data-track-label=\"link\">Find a job</a></li>\\n            <li><a class=\"c-header__link\" href=\"/authors/index.html\" data-track=\"click\" data-track-action=\"guide to authors\" data-track-label=\"link\">Guide to authors</a></li>\\n            <li><a class=\"c-header__link\" href=\"/authors/editorial_policies/\" data-track=\"click\" data-track-action=\"editorial policies\" data-track-label=\"link\">Editorial policies</a></li>\\n        </ul>\\n    </div>\\n</div>\\n\\n<footer class=\"composite-layer\" itemscope itemtype=\"http://schema.org/Periodical\">\\n        <meta itemprop=\"publisher\" content=\"Springer Nature\">\\n        \\n\\n        <div class=\"u-mt-16 u-mb-16\">\\n    <div class=\"u-container\">\\n        <div class=\"u-display-flex u-flex-wrap u-justify-content-space-between\">\\n            \\n\\n            <p class=\"c-meta u-ma-0 u-flex-shrink\">\\n                <span class=\"c-meta__item\">\\n                    Nature (<i>Nature</i>)\\n                </span>\\n                \\n    \\n    <span class=\"c-meta__item\">\\n        <abbr title=\"International Standard Serial Number\">ISSN</abbr> <span itemprop=\"onlineIssn\">1476-4687</span> (online)\\n    </span>\\n    \\n\\n\\n                \\n    \\n    <span class=\"c-meta__item\">\\n        <abbr title=\"International Standard Serial Number\">ISSN</abbr> <span itemprop=\"printIssn\">0028-0836</span> (print)\\n    </span>\\n    \\n\\n            </p>\\n        </div>\\n    </div>\\n</div>\\n\\n    <div class=\"c-footer\">\\n        <div class=\"u-hide-print\" data-track-component=\"footer\">\\n    <h2 class=\"u-visually-hidden\">nature.com sitemap</h2>\\n    <div class=\"c-footer__container\">\\n        <div class=\"c-footer__grid c-footer__group--separator\">\\n            <div class=\"c-footer__group\">\\n                <h3 class=\"c-footer__heading u-mt-0\">About Nature Portfolio</h3>\\n                <ul class=\"c-footer__list\">\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://www.nature.com/npg_/company_info/index.html\"\\n                                                  data-track=\"click\" data-track-action=\"about us\"\\n                                                  data-track-label=\"link\">About us</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://www.nature.com/npg_/press_room/press_releases.html\"\\n                                                  data-track=\"click\" data-track-action=\"press releases\"\\n                                                  data-track-label=\"link\">Press releases</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://press.nature.com/\"\\n                                                  data-track=\"click\" data-track-action=\"press office\"\\n                                                  data-track-label=\"link\">Press office</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://support.nature.com/support/home\"\\n                                                  data-track=\"click\" data-track-action=\"contact us\"\\n                                                  data-track-label=\"link\">Contact us</a></li>\\n                </ul>\\n            </div>\\n\\n            <div class=\"c-footer__group\">\\n                <h3 class=\"c-footer__heading u-mt-0\">Discover content</h3>\\n                <ul class=\"c-footer__list\">\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/siteindex\"\\n                                                  data-track=\"click\" data-track-action=\"journals a-z\"\\n                                                  data-track-label=\"link\">Journals A-Z</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/subjects\"\\n                                                  data-track=\"click\" data-track-action=\"article by subject\"\\n                                                  data-track-label=\"link\">Articles by subject</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/protocolexchange/\"\\n                                                  data-track=\"click\" data-track-action=\"protocol exchange\"\\n                                                  data-track-label=\"link\">Protocol Exchange</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.natureindex.com/\"\\n                                                  data-track=\"click\" data-track-action=\"nature index\"\\n                                                  data-track-label=\"link\">Nature Index</a></li>\\n                </ul>\\n            </div>\\n\\n            <div class=\"c-footer__group\">\\n                <h3 class=\"c-footer__heading u-mt-0\">Publishing policies</h3>\\n                <ul class=\"c-footer__list\">\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://www.nature.com/authors/editorial_policies\"\\n                                                  data-track=\"click\" data-track-action=\"Nature portfolio policies\"\\n                                                  data-track-label=\"link\">Nature portfolio policies</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://www.nature.com/nature-research/open-access\"\\n                                                  data-track=\"click\" data-track-action=\"open access\"\\n                                                  data-track-label=\"link\">Open access</a></li>\\n                </ul>\\n            </div>\\n\\n            <div class=\"c-footer__group\">\\n                <h3 class=\"c-footer__heading u-mt-0\">Author &amp; Researcher services</h3>\\n                <ul class=\"c-footer__list\">\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/reprints\"\\n                                                  data-track=\"click\" data-track-action=\"reprints and permissions\"\\n                                                  data-track-label=\"link\">Reprints &amp; permissions</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://www.springernature.com/gp/authors/research-data\"\\n                                                  data-track=\"click\" data-track-action=\"data research service\"\\n                                                  data-track-label=\"link\">Research data</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://authorservices.springernature.com/language-editing/\"\\n                                                  data-track=\"click\" data-track-action=\"language editing\"\\n                                                  data-track-label=\"link\">Language editing</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://authorservices.springernature.com/scientific-editing/\"\\n                                                  data-track=\"click\" data-track-action=\"scientific editing\"\\n                                                  data-track-label=\"link\">Scientific editing</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://masterclasses.nature.com/\"\\n                                                  data-track=\"click\" data-track-action=\"nature masterclasses\"\\n                                                  data-track-label=\"link\">Nature Masterclasses</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://solutions.springernature.com/\"\\n                                                  data-track=\"click\" data-track-action=\"research solutions\"\\n                                                  data-track-label=\"link\">Research Solutions</a></li>\\n                </ul>\\n            </div>\\n\\n            <div class=\"c-footer__group\">\\n                <h3 class=\"c-footer__heading u-mt-0\">Libraries &amp; institutions</h3>\\n                <ul class=\"c-footer__list\">\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://www.springernature.com/gp/librarians/tools-services\"\\n                                                  data-track=\"click\" data-track-action=\"librarian service and tools\"\\n                                                  data-track-label=\"link\">Librarian service &amp; tools</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://www.springernature.com/gp/librarians/manage-your-account/librarianportal\"\\n                                                  data-track=\"click\" data-track-action=\"librarian portal\"\\n                                                  data-track-label=\"link\">Librarian portal</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://www.nature.com/openresearch/about-open-access/information-for-institutions\"\\n                                                  data-track=\"click\" data-track-action=\"open research\"\\n                                                  data-track-label=\"link\">Open research</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://www.springernature.com/gp/librarians/recommend-to-your-library\"\\n                                                  data-track=\"click\" data-track-action=\"Recommend to library\"\\n                                                  data-track-label=\"link\">Recommend to library</a></li>\\n                </ul>\\n            </div>\\n\\n            <div class=\"c-footer__group\">\\n                <h3 class=\"c-footer__heading u-mt-0\">Advertising &amp; partnerships</h3>\\n                <ul class=\"c-footer__list\">\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://partnerships.nature.com/product/digital-advertising/\"\\n                                                  data-track=\"click\" data-track-action=\"advertising\"\\n                                                  data-track-label=\"link\">Advertising</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://partnerships.nature.com/\"\\n                                                  data-track=\"click\" data-track-action=\"partnerships and services\"\\n                                                  data-track-label=\"link\">Partnerships &amp; Services</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://partnerships.nature.com/media-kits/\" data-track=\"click\"\\n                                                  data-track-action=\"media kits\" data-track-label=\"link\">Media kits</a>\\n                    </li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                                  href=\"https://partnerships.nature.com/product/branded-content-native-advertising/\"\\n                                                  data-track-action=\"branded content\" data-track-label=\"link\">Branded\\n                        content</a></li>\\n                </ul>\\n            </div>\\n\\n            <div class=\"c-footer__group\">\\n                <h3 class=\"c-footer__heading u-mt-0\">Professional development</h3>\\n                <ul class=\"c-footer__list\">\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/naturecareers/\"\\n                                                  data-track=\"click\" data-track-action=\"nature careers\"\\n                                                  data-track-label=\"link\">Nature Careers</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://conferences.nature.com\"\\n                                                  data-track=\"click\" data-track-action=\"nature conferences\"\\n                                                  data-track-label=\"link\">Nature<span class=\"u-visually-hidden\"> </span>\\n                        Conferences</a></li>\\n                </ul>\\n            </div>\\n\\n            <div class=\"c-footer__group\">\\n                <h3 class=\"c-footer__heading u-mt-0\">Regional websites</h3>\\n                <ul class=\"c-footer__list\">\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/natafrica\"\\n                                                  data-track=\"click\" data-track-action=\"nature africa\"\\n                                                  data-track-label=\"link\">Nature Africa</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"http://www.naturechina.com\"\\n                                                  data-track=\"click\" data-track-action=\"nature china\"\\n                                                  data-track-label=\"link\">Nature China</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/nindia\"\\n                                                  data-track=\"click\" data-track-action=\"nature india\"\\n                                                  data-track-label=\"link\">Nature India</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/natitaly\"\\n                                                  data-track=\"click\" data-track-action=\"nature Italy\"\\n                                                  data-track-label=\"link\">Nature Italy</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.natureasia.com/ja-jp\"\\n                                                  data-track=\"click\" data-track-action=\"nature japan\"\\n                                                  data-track-label=\"link\">Nature Japan</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.natureasia.com/ko-kr\"\\n                                                  data-track=\"click\" data-track-action=\"nature korea\"\\n                                                  data-track-label=\"link\">Nature Korea</a></li>\\n                    <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/nmiddleeast\"\\n                                                  data-track=\"click\" data-track-action=\"nature middle east\"\\n                                                  data-track-label=\"link\">Nature Middle East</a></li>\\n                </ul>\\n            </div>\\n\\n        </div>\\n    </div>\\n    <div class=\"c-footer__container\">\\n        <ul class=\"c-footer__links\">\\n            <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/info/privacy\"\\n                                          data-track=\"click\" data-track-action=\"privacy policy\" data-track-label=\"link\">Privacy\\n                Policy</a></li>\\n            <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/info/cookies\"\\n                                          data-track=\"click\" data-track-action=\"use of cookies\" data-track-label=\"link\">Use\\n                of cookies</a></li>\\n            <li class=\"c-footer__item\">\\n                <button class=\"optanon-toggle-display c-footer__link\" onclick=\"javascript:;\"\\n                        data-cc-action=\"preferences\" data-track=\"click\" data-track-action=\"manage cookies\"\\n                        data-track-label=\"link\">Your privacy choices/Manage cookies\\n                </button>\\n            </li>\\n            <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/info/legal-notice\"\\n                                          data-track=\"click\" data-track-action=\"legal notice\" data-track-label=\"link\">Legal\\n                notice</a></li>\\n            <li class=\"c-footer__item\"><a class=\"c-footer__link\"\\n                                          href=\"https://www.nature.com/info/accessibility-statement\" data-track=\"click\"\\n                                          data-track-action=\"accessibility statement\" data-track-label=\"link\">Accessibility\\n                statement</a></li>\\n            <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.nature.com/info/terms-and-conditions\"\\n                                          data-track=\"click\" data-track-action=\"terms and conditions\"\\n                                          data-track-label=\"link\">Terms &amp; Conditions</a></li>\\n            <li class=\"c-footer__item\"><a class=\"c-footer__link\" href=\"https://www.springernature.com/ccpa\"\\n                                          data-track=\"click\" data-track-action=\"california privacy statement\"\\n                                          data-track-label=\"link\">Your US state privacy rights</a></li>\\n            \\n        </ul>\\n    </div>\\n</div>\\n\\n\\n        <div class=\"c-footer__container\">\\n    <a href=\"https://www.springernature.com/\" class=\"c-footer__link\">\\n        <img src=\"/static/images/logos/sn-logo-white-ea63208b81.svg\" alt=\"Springer Nature\" loading=\"lazy\" width=\"200\" height=\"20\"/>\\n    </a>\\n    <p class=\"c-footer__legal\" data-test=\"copyright\">&copy; 2024 Springer Nature Limited</p>\\n</div>\\n\\n    </div>\\n    <div class=\"u-visually-hidden\" aria-hidden=\"true\">\\n    \\n    <?xml version=\"1.0\" encoding=\"UTF-8\"?><!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\" \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\"><svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\"><defs><path id=\"a\" d=\"M0 .74h56.72v55.24H0z\"/></defs><symbol id=\"icon-access\" viewBox=\"0 0 18 18\"><path d=\"m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-account\" viewBox=\"0 0 18 18\"><path d=\"m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-alert\" viewBox=\"0 0 18 18\"><path d=\"m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-arrow-broad\" viewBox=\"0 0 16 16\"><path d=\"m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z\" fill-rule=\"evenodd\" transform=\"matrix(-1 0 0 -1 14 15)\"/></symbol><symbol id=\"icon-arrow-down\" viewBox=\"0 0 16 16\"><path d=\"m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-arrow-left\" viewBox=\"0 0 16 16\"><path d=\"m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-arrow-right\" viewBox=\"0 0 16 16\"><path d=\"m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-arrow-sub\" viewBox=\"0 0 16 16\"><path d=\"m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-arrow-up\" viewBox=\"0 0 16 16\"><path d=\"m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-article\" viewBox=\"0 0 18 18\"><path d=\"m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-audio\" viewBox=\"0 0 18 18\"><path d=\"m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-block\" viewBox=\"0 0 24 24\"><path d=\"m0 0h24v24h-24z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-book\" viewBox=\"0 0 18 18\"><path d=\"m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-broad\" viewBox=\"0 0 24 24\"><path d=\"m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z\" fill-rule=\"evenodd\" transform=\"matrix(-1 0 0 -1 20.182742 24.805206)\"/></symbol><symbol id=\"icon-calendar\" viewBox=\"0 0 18 18\"><path d=\"m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-cart\" viewBox=\"0 0 18 18\"><path d=\"m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z\"/></symbol><symbol id=\"icon-chevron-less\" viewBox=\"0 0 10 10\"><path d=\"m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z\" fill-rule=\"evenodd\" transform=\"matrix(0 -1 -1 0 9 9)\"/></symbol><symbol id=\"icon-chevron-more\" viewBox=\"0 0 10 10\"><path d=\"m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z\" fill-rule=\"evenodd\" transform=\"matrix(0 1 -1 0 11 1)\"/></symbol><symbol id=\"icon-chevron-right\" viewBox=\"0 0 10 10\"><path d=\"m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z\" fill-rule=\"evenodd\" transform=\"matrix(0 -1 1 0 0 10)\"/></symbol><symbol id=\"icon-circle-fill\" viewBox=\"0 0 16 16\"><path d=\"m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-circle\" viewBox=\"0 0 16 16\"><path d=\"m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-citation\" viewBox=\"0 0 18 18\"><path d=\"m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-close\" viewBox=\"0 0 16 16\"><path d=\"m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-collections\" viewBox=\"0 0 18 18\"><path d=\"m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-compare\" viewBox=\"0 0 18 18\"><path d=\"m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-download-file\" viewBox=\"0 0 18 18\"><path d=\"m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-download\" viewBox=\"0 0 16 16\"><path d=\"m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-editors\" viewBox=\"0 0 18 18\"><path d=\"m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-email\" viewBox=\"0 0 18 18\"><path d=\"m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-error\" viewBox=\"0 0 18 18\"><path d=\"m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-ethics\" viewBox=\"0 0 18 18\"><path d=\"m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-expand\"><path d=\"M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-explore\" viewBox=\"0 0 18 18\"><path d=\"m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-filter\" viewBox=\"0 0 16 16\"><path d=\"m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z\"/></symbol><symbol id=\"icon-home\" viewBox=\"0 0 18 18\"><path d=\"m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-image\" viewBox=\"0 0 18 18\"><path d=\"m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-info\" viewBox=\"0 0 18 18\"><path d=\"m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-institution\" viewBox=\"0 0 18 18\"><path d=\"m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-location\" viewBox=\"0 0 18 18\"><path d=\"m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-minus\" viewBox=\"0 0 16 16\"><path d=\"m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-newsletter\" viewBox=\"0 0 18 18\"><path d=\"m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-orcid\" viewBox=\"0 0 18 18\"><path d=\"m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-plus\" viewBox=\"0 0 16 16\"><path d=\"m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-print\" viewBox=\"0 0 18 18\"><path d=\"m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-search\" viewBox=\"0 0 22 22\"><path d=\"M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-social-facebook\" viewBox=\"0 0 24 24\"><path d=\"m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-social-twitter\" viewBox=\"0 0 24 24\"><path d=\"m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-social-youtube\" viewBox=\"0 0 24 24\"><path d=\"m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-subject-medicine\" viewBox=\"0 0 18 18\"><path d=\"m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-success\" viewBox=\"0 0 18 18\"><path d=\"m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-table\" viewBox=\"0 0 18 18\"><path d=\"m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-tick-circle\" viewBox=\"0 0 24 24\"><path d=\"m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-tick\" viewBox=\"0 0 16 16\"><path d=\"m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-update\" viewBox=\"0 0 18 18\"><path d=\"m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-upload\" viewBox=\"0 0 18 18\"><path d=\"m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-video\" viewBox=\"0 0 18 18\"><path d=\"m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-warning\" viewBox=\"0 0 18 18\"><path d=\"m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-checklist-banner\" viewBox=\"0 0 56.69 56.69\"><path style=\"fill:none\" d=\"M0 0h56.69v56.69H0z\"/><clipPath id=\"b\"><use xlink:href=\"#a\" style=\"overflow:visible\"/></clipPath><path d=\"M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81\" style=\"clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round\"/><path d=\"M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5\" style=\"clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round\"/></symbol><symbol id=\"icon-chevron-down\" viewBox=\"0 0 16 16\"><path d=\"m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z\" fill-rule=\"evenodd\" transform=\"matrix(0 1 -1 0 11 1)\"/></symbol><symbol id=\"icon-eds-i-arrow-right-medium\" viewBox=\"0 0 24 24\"><path d=\"m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z\"/></symbol><symbol id=\"icon-eds-i-chevron-down-medium\" viewBox=\"0 0 16 16\"><path d=\"m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-eds-i-chevron-down-small\" viewBox=\"0 0 16 16\"><path d=\"M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z\"/></symbol><symbol id=\"icon-eds-i-chevron-right-medium\" viewBox=\"0 0 10 10\"><path d=\"m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z\" fill-rule=\"evenodd\" transform=\"matrix(0 -1 1 0 0 10)\"/></symbol><symbol id=\"icon-eds-i-chevron-right-small\" viewBox=\"0 0 10 10\"><path d=\"m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z\" fill-rule=\"evenodd\" transform=\"matrix(0 -1 1 0 0 10)\"/></symbol><symbol id=\"icon-eds-i-chevron-up-medium\" viewBox=\"0 0 16 16\"><path d=\"m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-eds-i-close-medium\" viewBox=\"0 0 16 16\"><path d=\"m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-eds-i-download-medium\" viewBox=\"0 0 16 16\"><path d=\"m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-eds-i-info-filled-medium\" viewBox=\"0 0 18 18\"><path d=\"m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-eds-i-mail-medium\" viewBox=\"0 0 24 24\"><path d=\"m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z\"/></symbol><symbol id=\"icon-eds-i-menu-medium\" viewBox=\"0 0 24 24\"><path d=\"M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z\"/></symbol><symbol id=\"icon-eds-i-search-medium\" viewBox=\"0 0 24 24\"><path d=\"M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z\"/></symbol><symbol id=\"icon-eds-i-user-single-medium\" viewBox=\"0 0 24 24\"><path d=\"M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z\"/></symbol><symbol id=\"icon-eds-i-warning-filled-medium\" viewBox=\"0 0 18 18\"><path d=\"m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-expand-image\" viewBox=\"0 0 18 18\"><path d=\"m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z\" fill-rule=\"evenodd\"/></symbol><symbol id=\"icon-github\" viewBox=\"0 0 100 100\"><path fill-rule=\"evenodd\" clip-rule=\"evenodd\" d=\"M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z\"/></symbol><symbol id=\"icon-springer-arrow-left\"><path d=\"M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z\"/></symbol><symbol id=\"icon-springer-arrow-right\"><path d=\"M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z\"/></symbol><symbol id=\"icon-submit-open\" viewBox=\"0 0 16 17\"><path d=\"M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z\" fill-rule=\"nonzero\"/></symbol></svg>\\n</div>\\n</footer>\\n\\n\\n\\n\\n    \\n\\n    \\n\\n<div class=\"c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif c-site-messages--nature-briefing-transres\"\\ndata-component-id=\"nature-briefing-banner\"\\ndata-component-expirydays=\"30\"\\ndata-component-trigger-scroll-percentage=\"15\"\\ndata-track=\"in-view\"\\ndata-track-action=\"in-view\"\\ndata-track-category=\"nature briefing\"\\ndata-track-label=\"Briefing banner visible: TranslationalResearch\">\\n\\n    \\n    <div class=\"c-site-messages__banner-large\">\\n\\n        \\n<div class=\"c-site-messages__close-container\">\\n    <button class=\"c-site-messages__close\"\\n        data-track=\"click\"\\n        data-track-category=\"nature briefing\"\\n        data-track-label=\"Briefing banner dismiss: TranslationalResearch\">\\n        <svg width=\"25px\" height=\"25px\" focusable=\"false\" aria-hidden=\"true\" viewBox=\"0 0 25 25\" version=\"1.1\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\\n            <title>Close banner</title>\\n            <defs></defs>\\n            <g stroke=\"none\" stroke-width=\"1\" fill=\"none\" fill-rule=\"evenodd\">\\n                <rect opacity=\"0\" x=\"0\" y=\"0\" width=\"25\" height=\"25\"></rect>\\n                <path d=\"M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z\" fill=\"#ffffff\"></path>\\n            </g>\\n        </svg>\\n        <span class=\"visually-hidden\">Close</span>\\n    </button>\\n</div>\\n\\n\\n        <div class=\"c-site-messages__form-container\">\\n\\n            <div class=\"grid grid-12 last\">\\n                <div class=\"grid grid-4\">\\n                    <img alt=\"Nature Briefing: Translational Research\" src=\"/static/images/logos/nature-briefing-logo-transres-white-1245a3c374.svg\" width=\"213\" height=\"40\">\\n                    <p class=\"c-site-messages--nature-briefing__strapline extra-tight-line-height\">Sign up for the <em>Nature Briefing: Translational Research</em> newsletter — top stories in biotechnology, drug discovery and pharma.</p>\\n                </div>\\n                <div class=\"grid grid-8 last\">\\n                    <form action=\"https://www.nature.com/briefing/translational_research\" method=\"post\" data-location=\"banner\" data-track=\"submit||nature_briefing_sign_up\" data-track-action=\"transmit-form\" data-track-category=\"nature briefing\" data-track-label=\"Briefing banner submit: TranslationalResearch\">\\n                        <input id=\"briefing-banner-signup-form-input-track-originReferralPoint\" type=\"hidden\" name=\"track_originReferralPoint\" value=\"TransResBriefingBanner\">\\n                        <input id=\"briefing-banner-signup-form-input-track-formType\" type=\"hidden\" name=\"track_formType\" value=\"DirectEmailBanner\">\\n\\n                        <input type=\"hidden\" value=\"false\" name=\"gdpr_tick\" id=\"gdpr_tick\">\\n                        <input type=\"hidden\" value=\"false\" name=\"marketing\" id=\"marketing\">\\n                        <input type=\"hidden\" value=\"false\" name=\"marketing_tick\" id=\"marketing_tick\">\\n                        <input type=\"hidden\" value=\"TransResBriefingBanner\" name=\"brieferEntryPoint\" id=\"brieferEntryPoint\">\\n\\n                        <label class=\"nature-briefing-banner__email-label\" for=\"emailAddress\">Email address</label>\\n\\n                        <div class=\"nature-briefing-banner__email-wrapper\">\\n                            <input class=\"nature-briefing-banner__email-input box-sizing text14\" type=\"email\" id=\"emailAddress\" name=\"emailAddress\" value=\"\" placeholder=\"e.g. jo.smith@university.ac.uk\" required data-test-element=\"briefing-emailbanner-email-input\">\\n                            \\n                            <input type=\"hidden\" value=\"true\" name=\"N:translational_research\" id=\"defaultNewsletter\">\\n                            <button type=\"submit\" class=\"nature-briefing-banner__submit-button box-sizing text14\" data-test-element=\"briefing-emailbanner-signup-button\">Sign up</button>\\n                        </div>\\n\\n                        <div class=\"nature-briefing-banner__checkbox-wrapper grid grid-12 last\">\\n                            <input class=\"nature-briefing-banner__checkbox-checkbox\" id=\"gdpr-briefing-banner-checkbox\" type=\"checkbox\" name=\"gdpr\" value=\"true\" data-test-element=\"briefing-emailbanner-gdpr-checkbox\" required>\\n                            <label class=\"nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height\" for=\"gdpr-briefing-banner-checkbox\">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href=\"https://www.nature.com/info/privacy\">Privacy Policy</a>.</label>\\n                        </div>\\n                    </form>\\n                </div>\\n            </div>\\n\\n        </div>\\n\\n    </div>\\n\\n    \\n    <div class=\"c-site-messages__banner-small\">\\n\\n        \\n<div class=\"c-site-messages__close-container\">\\n    <button class=\"c-site-messages__close\"\\n        data-track=\"click\"\\n        data-track-category=\"nature briefing\"\\n        data-track-label=\"Briefing banner dismiss: TranslationalResearch\">\\n        <svg width=\"25px\" height=\"25px\" focusable=\"false\" aria-hidden=\"true\" viewBox=\"0 0 25 25\" version=\"1.1\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\\n            <title>Close banner</title>\\n            <defs></defs>\\n            <g stroke=\"none\" stroke-width=\"1\" fill=\"none\" fill-rule=\"evenodd\">\\n                <rect opacity=\"0\" x=\"0\" y=\"0\" width=\"25\" height=\"25\"></rect>\\n                <path d=\"M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z\" fill=\"#ffffff\"></path>\\n            </g>\\n        </svg>\\n        <span class=\"visually-hidden\">Close</span>\\n    </button>\\n</div>\\n\\n\\n        <div class=\"c-site-messages__content text14\">\\n            <span class=\"c-site-messages--nature-briefing__strapline strong\">Get what matters in translational research, free to your inbox weekly.</span>\\n            <a class=\"nature-briefing__link text14 sans-serif\"\\n                data-track=\"click\"\\n                data-track-category=\"nature briefing\"\\n                data-track-label=\"Small-screen banner CTA to site\"\\n                data-test-element=\"briefing-banner-link\"\\n                target=\"_blank\"\\n                rel=\"noreferrer noopener\"\\n                href=\"/briefing/translational-research/?brieferEntryPoint=TransResBriefingBanner\">Sign up for Nature Briefing: Translational Research\\n            </a>\\n        </div>\\n\\n    </div>\\n\\n</div>\\n\\n\\n\\n\\n\\n\\n<noscript>\\n    <img hidden src=\"https://verify.nature.com/verify/nature.png\" width=\"0\" height=\"0\" style=\"display: none\" alt=\"\">\\n</noscript>\\n\\n\\n\\n\\n<script src=\"//content.readcube.com/ping?doi=10.1038/s41586-023-06473-y&amp;format=js&amp;last_modified=2023-11-14\" async></script>\\n<img src=\"/w547jrmq/article/s41586-023-06473-y\" width=\"1\" height=\"1\" alt=\"\" class=\"u-visually-hidden\">\\n</body>\\n</html>'}\n"
     ]
    }
   ],
   "source": [
    "# import xmltodict\n",
    "# import requests\n",
    "from langchain_community.vectorstores import FAISS\n",
    "from langchain_community.embeddings import OpenAIEmbeddings\n",
    "from langchain.text_splitter import RecursiveCharacterTextSplitter\n",
    "from langchain_community.document_loaders.sitemap import SitemapLoader\n",
    "from langchain.embeddings import HuggingFaceEmbeddings\n",
    "from sentence_transformers import SentenceTransformer, util\n",
    "import pickle\n",
    "import nest_asyncio\n",
    "from langchain.chains.llm import LLMChain\n",
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "nest_asyncio.apply()\n",
    "\n",
    "# Function for getting the text data and title from a website url, returns as dict\n",
    "def get_data(url):\n",
    "    r = requests.get(url)\n",
    "    all_text = r.text\n",
    "    title = all_text[all_text.find('<title>') + 7 : all_text.find('</title>')]\n",
    "    return {title : all_text}\n",
    "\n",
    "\n",
    "test = get_data('https://doi.org/10.1038/s41586-023-06473-y')\n",
    "\n",
    "print(test)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def save_content(link_list):\n",
    "\n",
    "        # Remove the first 835 lines\n",
    "        lines = text.splitlines()\n",
    "        cleaned_text = \"\\n\".join(lines[835:])\n",
    "\n",
    "        # Get the first 3 words in the cleaned text\n",
    "        words = cleaned_text.split()[:3]\n",
    "        file_name_prefix = \"_\".join(words)\n",
    "\n",
    "        # Replace special characters and spaces with an underscore\n",
    "        # file_name_prefix = re.sub(r\"[^a-zA-Z0-9]+\", \"_\", file_name_prefix)\n",
    "\n",
    "        # Get the current working directory\n",
    "        current_dir = os.getcwd()\n",
    "\n",
    "        # Move up one level to the parent directory\n",
    "        parent_dir = os.path.dirname(current_dir)\n",
    "\n",
    "        # Set the path to the data folder\n",
    "        data_folder = os.path.join(parent_dir, \"data/langchain_doc\")\n",
    "\n",
    "        # Create the data folder if it doesn't exist\n",
    "        if not os.path.exists(data_folder):\n",
    "            os.makedirs(data_folder)\n",
    "\n",
    "        # Set the path to the output file\n",
    "        output_file = os.path.join(data_folder, f\"{i}_{file_name_prefix}.txt\")\n",
    "\n",
    "        # Save the cleaned content to the output file\n",
    "        with open(output_file, \"w\") as f:\n",
    "            f.write(cleaned_text)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_openai import OpenAIEmbeddings\n",
    "\n",
    "loader = UnstructuredHTMLLoader(\"filename\")\n",
    "data = loader.load()\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "os.environ['OPENAI_API_KEY'] = getpass.getpass('OpenAI API Key:')\n",
    "\n",
    "# Load the document, split it into chunks, embed each chunk and load it into the vector store.\n",
    "raw_documents = TextLoader('../../../state_of_the_union.txt').load()\n",
    "text_splitter = CharacterTextSplitter(chunk_size=1000, chunk_overlap=0)\n",
    "documents = text_splitter.split_documents(raw_documents)\n",
    "db = FAISS.from_documents(documents, OpenAIEmbeddings())\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def load_vector_store():\n",
    "    # Indexing\n",
    "    ### Load vector store\n",
    "    with open(\"db/faiss_store.pkl\", \"rb\") as f:\n",
    "        vectordb = pickle.load(f)\n",
    "\n",
    "    return vectordb"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
